[22.08.2024, 10:11:15] RequestError: Error: connect EHOSTUNREACH 169.254.0.51:3274
    at new RequestError (C:\web\chat-bot-telegram\node_modules\request-promise-core\lib\errors.js:14:15)
    at plumbing.callback (C:\web\chat-bot-telegram\node_modules\request-promise-core\lib\plumbing.js:87:29)
    at Request.RP$callback [as _callback] (C:\web\chat-bot-telegram\node_modules\request-promise-core\lib\plumbing.js:46:31)
    at self.callback (C:\web\chat-bot-telegram\node_modules\@cypress\request\request.js:183:22)
    at Request.emit (node:events:518:28)
    at Request.onRequestError (C:\web\chat-bot-telegram\node_modules\@cypress\request\request.js:869:8)
    at ClientRequest.emit (node:events:518:28)
    at _destroy (node:_http_client:880:13)
    at onSocketNT (node:_http_client:900:5)
    at process.processTicksAndRejections (node:internal/process/task_queues:83:21)
[22.08.2024, 10:11:16] RequestError: Error: connect EHOSTUNREACH 169.254.0.51:3274
    at new RequestError (C:\web\chat-bot-telegram\node_modules\request-promise-core\lib\errors.js:14:15)
    at plumbing.callback (C:\web\chat-bot-telegram\node_modules\request-promise-core\lib\plumbing.js:87:29)
    at Request.RP$callback [as _callback] (C:\web\chat-bot-telegram\node_modules\request-promise-core\lib\plumbing.js:46:31)
    at self.callback (C:\web\chat-bot-telegram\node_modules\@cypress\request\request.js:183:22)
    at Request.emit (node:events:518:28)
    at Request.onRequestError (C:\web\chat-bot-telegram\node_modules\@cypress\request\request.js:869:8)
    at ClientRequest.emit (node:events:518:28)
    at _destroy (node:_http_client:880:13)
    at onSocketNT (node:_http_client:900:5)
    at process.processTicksAndRejections (node:internal/process/task_queues:83:21)
[22.08.2024, 10:11:16] RequestError: Error: connect EHOSTUNREACH 169.254.0.51:3274
    at new RequestError (C:\web\chat-bot-telegram\node_modules\request-promise-core\lib\errors.js:14:15)
    at plumbing.callback (C:\web\chat-bot-telegram\node_modules\request-promise-core\lib\plumbing.js:87:29)
    at Request.RP$callback [as _callback] (C:\web\chat-bot-telegram\node_modules\request-promise-core\lib\plumbing.js:46:31)
    at self.callback (C:\web\chat-bot-telegram\node_modules\@cypress\request\request.js:183:22)
    at Request.emit (node:events:518:28)
    at Request.onRequestError (C:\web\chat-bot-telegram\node_modules\@cypress\request\request.js:869:8)
    at ClientRequest.emit (node:events:518:28)
    at _destroy (node:_http_client:880:13)
    at onSocketNT (node:_http_client:900:5)
    at process.processTicksAndRejections (node:internal/process/task_queues:83:21)
[22.08.2024, 10:11:16] RequestError: Error: connect EHOSTUNREACH 169.254.0.51:3274
    at new RequestError (C:\web\chat-bot-telegram\node_modules\request-promise-core\lib\errors.js:14:15)
    at plumbing.callback (C:\web\chat-bot-telegram\node_modules\request-promise-core\lib\plumbing.js:87:29)
    at Request.RP$callback [as _callback] (C:\web\chat-bot-telegram\node_modules\request-promise-core\lib\plumbing.js:46:31)
    at self.callback (C:\web\chat-bot-telegram\node_modules\@cypress\request\request.js:183:22)
    at Request.emit (node:events:518:28)
    at Request.onRequestError (C:\web\chat-bot-telegram\node_modules\@cypress\request\request.js:869:8)
    at ClientRequest.emit (node:events:518:28)
    at _destroy (node:_http_client:880:13)
    at onSocketNT (node:_http_client:900:5)
    at process.processTicksAndRejections (node:internal/process/task_queues:83:21)
[22.08.2024, 10:11:17] RequestError: Error: connect EHOSTUNREACH 169.254.0.51:3274
    at new RequestError (C:\web\chat-bot-telegram\node_modules\request-promise-core\lib\errors.js:14:15)
    at plumbing.callback (C:\web\chat-bot-telegram\node_modules\request-promise-core\lib\plumbing.js:87:29)
    at Request.RP$callback [as _callback] (C:\web\chat-bot-telegram\node_modules\request-promise-core\lib\plumbing.js:46:31)
    at self.callback (C:\web\chat-bot-telegram\node_modules\@cypress\request\request.js:183:22)
    at Request.emit (node:events:518:28)
    at Request.onRequestError (C:\web\chat-bot-telegram\node_modules\@cypress\request\request.js:869:8)
    at ClientRequest.emit (node:events:518:28)
    at _destroy (node:_http_client:880:13)
    at onSocketNT (node:_http_client:900:5)
    at process.processTicksAndRejections (node:internal/process/task_queues:83:21)
[22.08.2024, 10:11:17] RequestError: Error: connect EHOSTUNREACH 169.254.0.51:3274
    at new RequestError (C:\web\chat-bot-telegram\node_modules\request-promise-core\lib\errors.js:14:15)
    at plumbing.callback (C:\web\chat-bot-telegram\node_modules\request-promise-core\lib\plumbing.js:87:29)
    at Request.RP$callback [as _callback] (C:\web\chat-bot-telegram\node_modules\request-promise-core\lib\plumbing.js:46:31)
    at self.callback (C:\web\chat-bot-telegram\node_modules\@cypress\request\request.js:183:22)
    at Request.emit (node:events:518:28)
    at Request.onRequestError (C:\web\chat-bot-telegram\node_modules\@cypress\request\request.js:869:8)
    at ClientRequest.emit (node:events:518:28)
    at _destroy (node:_http_client:880:13)
    at onSocketNT (node:_http_client:900:5)
    at process.processTicksAndRejections (node:internal/process/task_queues:83:21)
[22.08.2024, 10:11:17] RequestError: Error: connect EHOSTUNREACH 169.254.0.51:3274
    at new RequestError (C:\web\chat-bot-telegram\node_modules\request-promise-core\lib\errors.js:14:15)
    at plumbing.callback (C:\web\chat-bot-telegram\node_modules\request-promise-core\lib\plumbing.js:87:29)
    at Request.RP$callback [as _callback] (C:\web\chat-bot-telegram\node_modules\request-promise-core\lib\plumbing.js:46:31)
    at self.callback (C:\web\chat-bot-telegram\node_modules\@cypress\request\request.js:183:22)
    at Request.emit (node:events:518:28)
    at Request.onRequestError (C:\web\chat-bot-telegram\node_modules\@cypress\request\request.js:869:8)
    at ClientRequest.emit (node:events:518:28)
    at _destroy (node:_http_client:880:13)
    at onSocketNT (node:_http_client:900:5)
    at process.processTicksAndRejections (node:internal/process/task_queues:83:21)
[22.08.2024, 10:11:18] RequestError: Error: connect EHOSTUNREACH 169.254.0.51:3274
    at new RequestError (C:\web\chat-bot-telegram\node_modules\request-promise-core\lib\errors.js:14:15)
    at plumbing.callback (C:\web\chat-bot-telegram\node_modules\request-promise-core\lib\plumbing.js:87:29)
    at Request.RP$callback [as _callback] (C:\web\chat-bot-telegram\node_modules\request-promise-core\lib\plumbing.js:46:31)
    at self.callback (C:\web\chat-bot-telegram\node_modules\@cypress\request\request.js:183:22)
    at Request.emit (node:events:518:28)
    at Request.onRequestError (C:\web\chat-bot-telegram\node_modules\@cypress\request\request.js:869:8)
    at ClientRequest.emit (node:events:518:28)
    at _destroy (node:_http_client:880:13)
    at onSocketNT (node:_http_client:900:5)
    at process.processTicksAndRejections (node:internal/process/task_queues:83:21)
[22.08.2024, 10:11:18] RequestError: Error: connect EHOSTUNREACH 169.254.0.51:3274
    at new RequestError (C:\web\chat-bot-telegram\node_modules\request-promise-core\lib\errors.js:14:15)
    at plumbing.callback (C:\web\chat-bot-telegram\node_modules\request-promise-core\lib\plumbing.js:87:29)
    at Request.RP$callback [as _callback] (C:\web\chat-bot-telegram\node_modules\request-promise-core\lib\plumbing.js:46:31)
    at self.callback (C:\web\chat-bot-telegram\node_modules\@cypress\request\request.js:183:22)
    at Request.emit (node:events:518:28)
    at Request.onRequestError (C:\web\chat-bot-telegram\node_modules\@cypress\request\request.js:869:8)
    at ClientRequest.emit (node:events:518:28)
    at _destroy (node:_http_client:880:13)
    at onSocketNT (node:_http_client:900:5)
    at process.processTicksAndRejections (node:internal/process/task_queues:83:21)
[22.08.2024, 10:11:18] RequestError: Error: connect EHOSTUNREACH 169.254.0.51:3274
    at new RequestError (C:\web\chat-bot-telegram\node_modules\request-promise-core\lib\errors.js:14:15)
    at plumbing.callback (C:\web\chat-bot-telegram\node_modules\request-promise-core\lib\plumbing.js:87:29)
    at Request.RP$callback [as _callback] (C:\web\chat-bot-telegram\node_modules\request-promise-core\lib\plumbing.js:46:31)
    at self.callback (C:\web\chat-bot-telegram\node_modules\@cypress\request\request.js:183:22)
    at Request.emit (node:events:518:28)
    at Request.onRequestError (C:\web\chat-bot-telegram\node_modules\@cypress\request\request.js:869:8)
    at ClientRequest.emit (node:events:518:28)
    at _destroy (node:_http_client:880:13)
    at onSocketNT (node:_http_client:900:5)
    at process.processTicksAndRejections (node:internal/process/task_queues:83:21)
[22.08.2024, 10:11:19] RequestError: Error: connect EHOSTUNREACH 169.254.0.51:3274
    at new RequestError (C:\web\chat-bot-telegram\node_modules\request-promise-core\lib\errors.js:14:15)
    at plumbing.callback (C:\web\chat-bot-telegram\node_modules\request-promise-core\lib\plumbing.js:87:29)
    at Request.RP$callback [as _callback] (C:\web\chat-bot-telegram\node_modules\request-promise-core\lib\plumbing.js:46:31)
    at self.callback (C:\web\chat-bot-telegram\node_modules\@cypress\request\request.js:183:22)
    at Request.emit (node:events:518:28)
    at Request.onRequestError (C:\web\chat-bot-telegram\node_modules\@cypress\request\request.js:869:8)
    at ClientRequest.emit (node:events:518:28)
    at _destroy (node:_http_client:880:13)
    at onSocketNT (node:_http_client:900:5)
    at process.processTicksAndRejections (node:internal/process/task_queues:83:21)
[22.08.2024, 10:11:19] RequestError: Error: connect EHOSTUNREACH 169.254.0.51:3274
    at new RequestError (C:\web\chat-bot-telegram\node_modules\request-promise-core\lib\errors.js:14:15)
    at plumbing.callback (C:\web\chat-bot-telegram\node_modules\request-promise-core\lib\plumbing.js:87:29)
    at Request.RP$callback [as _callback] (C:\web\chat-bot-telegram\node_modules\request-promise-core\lib\plumbing.js:46:31)
    at self.callback (C:\web\chat-bot-telegram\node_modules\@cypress\request\request.js:183:22)
    at Request.emit (node:events:518:28)
    at Request.onRequestError (C:\web\chat-bot-telegram\node_modules\@cypress\request\request.js:869:8)
    at ClientRequest.emit (node:events:518:28)
    at _destroy (node:_http_client:880:13)
    at onSocketNT (node:_http_client:900:5)
    at process.processTicksAndRejections (node:internal/process/task_queues:83:21)
[22.08.2024, 10:11:19] RequestError: Error: connect EHOSTUNREACH 169.254.0.51:3274
    at new RequestError (C:\web\chat-bot-telegram\node_modules\request-promise-core\lib\errors.js:14:15)
    at plumbing.callback (C:\web\chat-bot-telegram\node_modules\request-promise-core\lib\plumbing.js:87:29)
    at Request.RP$callback [as _callback] (C:\web\chat-bot-telegram\node_modules\request-promise-core\lib\plumbing.js:46:31)
    at self.callback (C:\web\chat-bot-telegram\node_modules\@cypress\request\request.js:183:22)
    at Request.emit (node:events:518:28)
    at Request.onRequestError (C:\web\chat-bot-telegram\node_modules\@cypress\request\request.js:869:8)
    at ClientRequest.emit (node:events:518:28)
    at _destroy (node:_http_client:880:13)
    at onSocketNT (node:_http_client:900:5)
    at process.processTicksAndRejections (node:internal/process/task_queues:83:21)
[22.08.2024, 10:11:20] RequestError: Error: connect EHOSTUNREACH 169.254.0.51:3274
    at new RequestError (C:\web\chat-bot-telegram\node_modules\request-promise-core\lib\errors.js:14:15)
    at plumbing.callback (C:\web\chat-bot-telegram\node_modules\request-promise-core\lib\plumbing.js:87:29)
    at Request.RP$callback [as _callback] (C:\web\chat-bot-telegram\node_modules\request-promise-core\lib\plumbing.js:46:31)
    at self.callback (C:\web\chat-bot-telegram\node_modules\@cypress\request\request.js:183:22)
    at Request.emit (node:events:518:28)
    at Request.onRequestError (C:\web\chat-bot-telegram\node_modules\@cypress\request\request.js:869:8)
    at ClientRequest.emit (node:events:518:28)
    at _destroy (node:_http_client:880:13)
    at onSocketNT (node:_http_client:900:5)
    at process.processTicksAndRejections (node:internal/process/task_queues:83:21)
[22.08.2024, 10:11:20] RequestError: Error: connect EHOSTUNREACH 169.254.0.51:3274
    at new RequestError (C:\web\chat-bot-telegram\node_modules\request-promise-core\lib\errors.js:14:15)
    at plumbing.callback (C:\web\chat-bot-telegram\node_modules\request-promise-core\lib\plumbing.js:87:29)
    at Request.RP$callback [as _callback] (C:\web\chat-bot-telegram\node_modules\request-promise-core\lib\plumbing.js:46:31)
    at self.callback (C:\web\chat-bot-telegram\node_modules\@cypress\request\request.js:183:22)
    at Request.emit (node:events:518:28)
    at Request.onRequestError (C:\web\chat-bot-telegram\node_modules\@cypress\request\request.js:869:8)
    at ClientRequest.emit (node:events:518:28)
    at _destroy (node:_http_client:880:13)
    at onSocketNT (node:_http_client:900:5)
    at process.processTicksAndRejections (node:internal/process/task_queues:83:21)
[22.08.2024, 10:11:20] RequestError: Error: connect EHOSTUNREACH 169.254.0.51:3274
    at new RequestError (C:\web\chat-bot-telegram\node_modules\request-promise-core\lib\errors.js:14:15)
    at plumbing.callback (C:\web\chat-bot-telegram\node_modules\request-promise-core\lib\plumbing.js:87:29)
    at Request.RP$callback [as _callback] (C:\web\chat-bot-telegram\node_modules\request-promise-core\lib\plumbing.js:46:31)
    at self.callback (C:\web\chat-bot-telegram\node_modules\@cypress\request\request.js:183:22)
    at Request.emit (node:events:518:28)
    at Request.onRequestError (C:\web\chat-bot-telegram\node_modules\@cypress\request\request.js:869:8)
    at ClientRequest.emit (node:events:518:28)
    at _destroy (node:_http_client:880:13)
    at onSocketNT (node:_http_client:900:5)
    at process.processTicksAndRejections (node:internal/process/task_queues:83:21)
[22.08.2024, 10:11:20] RequestError: Error: connect EHOSTUNREACH 169.254.0.51:3274
    at new RequestError (C:\web\chat-bot-telegram\node_modules\request-promise-core\lib\errors.js:14:15)
    at plumbing.callback (C:\web\chat-bot-telegram\node_modules\request-promise-core\lib\plumbing.js:87:29)
    at Request.RP$callback [as _callback] (C:\web\chat-bot-telegram\node_modules\request-promise-core\lib\plumbing.js:46:31)
    at self.callback (C:\web\chat-bot-telegram\node_modules\@cypress\request\request.js:183:22)
    at Request.emit (node:events:518:28)
    at Request.onRequestError (C:\web\chat-bot-telegram\node_modules\@cypress\request\request.js:869:8)
    at ClientRequest.emit (node:events:518:28)
    at _destroy (node:_http_client:880:13)
    at onSocketNT (node:_http_client:900:5)
    at process.processTicksAndRejections (node:internal/process/task_queues:83:21)
[22.08.2024, 10:11:21] RequestError: Error: connect EHOSTUNREACH 169.254.0.51:3274
    at new RequestError (C:\web\chat-bot-telegram\node_modules\request-promise-core\lib\errors.js:14:15)
    at plumbing.callback (C:\web\chat-bot-telegram\node_modules\request-promise-core\lib\plumbing.js:87:29)
    at Request.RP$callback [as _callback] (C:\web\chat-bot-telegram\node_modules\request-promise-core\lib\plumbing.js:46:31)
    at self.callback (C:\web\chat-bot-telegram\node_modules\@cypress\request\request.js:183:22)
    at Request.emit (node:events:518:28)
    at Request.onRequestError (C:\web\chat-bot-telegram\node_modules\@cypress\request\request.js:869:8)
    at ClientRequest.emit (node:events:518:28)
    at _destroy (node:_http_client:880:13)
    at onSocketNT (node:_http_client:900:5)
    at process.processTicksAndRejections (node:internal/process/task_queues:83:21)
[22.08.2024, 10:28:48] Error: ETELEGRAM: 400 Bad Request: message is not modified: specified new message content and reply markup are exactly the same as a current content and reply markup of the message
    at C:\web\chat-bot-telegram\node_modules\node-telegram-bot-api\src\telegram.js:316:15
    at tryCatcher (C:\web\chat-bot-telegram\node_modules\bluebird\js\release\util.js:16:23)
    at Promise._settlePromiseFromHandler (C:\web\chat-bot-telegram\node_modules\bluebird\js\release\promise.js:547:31)
    at Promise._settlePromise (C:\web\chat-bot-telegram\node_modules\bluebird\js\release\promise.js:604:18)
    at Promise._settlePromise0 (C:\web\chat-bot-telegram\node_modules\bluebird\js\release\promise.js:649:10)
    at Promise._settlePromises (C:\web\chat-bot-telegram\node_modules\bluebird\js\release\promise.js:729:18)
    at _drainQueueStep (C:\web\chat-bot-telegram\node_modules\bluebird\js\release\async.js:93:12)
    at _drainQueue (C:\web\chat-bot-telegram\node_modules\bluebird\js\release\async.js:86:9)
    at Async._drainQueues (C:\web\chat-bot-telegram\node_modules\bluebird\js\release\async.js:102:5)
    at Async.drainQueues [as _onImmediate] (C:\web\chat-bot-telegram\node_modules\bluebird\js\release\async.js:15:14)
    at process.processImmediate (node:internal/timers:478:21)
[22.08.2024, 10:30:17] Error: ETELEGRAM: 400 Bad Request: message is not modified: specified new message content and reply markup are exactly the same as a current content and reply markup of the message
    at C:\web\chat-bot-telegram\node_modules\node-telegram-bot-api\src\telegram.js:316:15
    at tryCatcher (C:\web\chat-bot-telegram\node_modules\bluebird\js\release\util.js:16:23)
    at Promise._settlePromiseFromHandler (C:\web\chat-bot-telegram\node_modules\bluebird\js\release\promise.js:547:31)
    at Promise._settlePromise (C:\web\chat-bot-telegram\node_modules\bluebird\js\release\promise.js:604:18)
    at Promise._settlePromise0 (C:\web\chat-bot-telegram\node_modules\bluebird\js\release\promise.js:649:10)
    at Promise._settlePromises (C:\web\chat-bot-telegram\node_modules\bluebird\js\release\promise.js:729:18)
    at _drainQueueStep (C:\web\chat-bot-telegram\node_modules\bluebird\js\release\async.js:93:12)
    at _drainQueue (C:\web\chat-bot-telegram\node_modules\bluebird\js\release\async.js:86:9)
    at Async._drainQueues (C:\web\chat-bot-telegram\node_modules\bluebird\js\release\async.js:102:5)
    at Async.drainQueues [as _onImmediate] (C:\web\chat-bot-telegram\node_modules\bluebird\js\release\async.js:15:14)
    at process.processImmediate (node:internal/timers:478:21)
[22.08.2024, 10:30:32] Error: ETELEGRAM: 400 Bad Request: message is not modified: specified new message content and reply markup are exactly the same as a current content and reply markup of the message
    at C:\web\chat-bot-telegram\node_modules\node-telegram-bot-api\src\telegram.js:316:15
    at tryCatcher (C:\web\chat-bot-telegram\node_modules\bluebird\js\release\util.js:16:23)
    at Promise._settlePromiseFromHandler (C:\web\chat-bot-telegram\node_modules\bluebird\js\release\promise.js:547:31)
    at Promise._settlePromise (C:\web\chat-bot-telegram\node_modules\bluebird\js\release\promise.js:604:18)
    at Promise._settlePromise0 (C:\web\chat-bot-telegram\node_modules\bluebird\js\release\promise.js:649:10)
    at Promise._settlePromises (C:\web\chat-bot-telegram\node_modules\bluebird\js\release\promise.js:729:18)
    at _drainQueueStep (C:\web\chat-bot-telegram\node_modules\bluebird\js\release\async.js:93:12)
    at _drainQueue (C:\web\chat-bot-telegram\node_modules\bluebird\js\release\async.js:86:9)
    at Async._drainQueues (C:\web\chat-bot-telegram\node_modules\bluebird\js\release\async.js:102:5)
    at Async.drainQueues [as _onImmediate] (C:\web\chat-bot-telegram\node_modules\bluebird\js\release\async.js:15:14)
    at process.processImmediate (node:internal/timers:478:21)
[22.08.2024, 10:32:59] Error: ETELEGRAM: 400 Bad Request: message is not modified: specified new message content and reply markup are exactly the same as a current content and reply markup of the message
    at C:\web\chat-bot-telegram\node_modules\node-telegram-bot-api\src\telegram.js:316:15
    at tryCatcher (C:\web\chat-bot-telegram\node_modules\bluebird\js\release\util.js:16:23)
    at Promise._settlePromiseFromHandler (C:\web\chat-bot-telegram\node_modules\bluebird\js\release\promise.js:547:31)
    at Promise._settlePromise (C:\web\chat-bot-telegram\node_modules\bluebird\js\release\promise.js:604:18)
    at Promise._settlePromise0 (C:\web\chat-bot-telegram\node_modules\bluebird\js\release\promise.js:649:10)
    at Promise._settlePromises (C:\web\chat-bot-telegram\node_modules\bluebird\js\release\promise.js:729:18)
    at _drainQueueStep (C:\web\chat-bot-telegram\node_modules\bluebird\js\release\async.js:93:12)
    at _drainQueue (C:\web\chat-bot-telegram\node_modules\bluebird\js\release\async.js:86:9)
    at Async._drainQueues (C:\web\chat-bot-telegram\node_modules\bluebird\js\release\async.js:102:5)
    at Async.drainQueues [as _onImmediate] (C:\web\chat-bot-telegram\node_modules\bluebird\js\release\async.js:15:14)
    at process.processImmediate (node:internal/timers:478:21)
[22.08.2024, 10:33:06] Error: ETELEGRAM: 400 Bad Request: message is not modified: specified new message content and reply markup are exactly the same as a current content and reply markup of the message
    at C:\web\chat-bot-telegram\node_modules\node-telegram-bot-api\src\telegram.js:316:15
    at tryCatcher (C:\web\chat-bot-telegram\node_modules\bluebird\js\release\util.js:16:23)
    at Promise._settlePromiseFromHandler (C:\web\chat-bot-telegram\node_modules\bluebird\js\release\promise.js:547:31)
    at Promise._settlePromise (C:\web\chat-bot-telegram\node_modules\bluebird\js\release\promise.js:604:18)
    at Promise._settlePromise0 (C:\web\chat-bot-telegram\node_modules\bluebird\js\release\promise.js:649:10)
    at Promise._settlePromises (C:\web\chat-bot-telegram\node_modules\bluebird\js\release\promise.js:729:18)
    at _drainQueueStep (C:\web\chat-bot-telegram\node_modules\bluebird\js\release\async.js:93:12)
    at _drainQueue (C:\web\chat-bot-telegram\node_modules\bluebird\js\release\async.js:86:9)
    at Async._drainQueues (C:\web\chat-bot-telegram\node_modules\bluebird\js\release\async.js:102:5)
    at Async.drainQueues [as _onImmediate] (C:\web\chat-bot-telegram\node_modules\bluebird\js\release\async.js:15:14)
    at process.processImmediate (node:internal/timers:478:21)
[22.08.2024, 10:33:31] Error: ETELEGRAM: 400 Bad Request: message is not modified: specified new message content and reply markup are exactly the same as a current content and reply markup of the message
    at C:\web\chat-bot-telegram\node_modules\node-telegram-bot-api\src\telegram.js:316:15
    at tryCatcher (C:\web\chat-bot-telegram\node_modules\bluebird\js\release\util.js:16:23)
    at Promise._settlePromiseFromHandler (C:\web\chat-bot-telegram\node_modules\bluebird\js\release\promise.js:547:31)
    at Promise._settlePromise (C:\web\chat-bot-telegram\node_modules\bluebird\js\release\promise.js:604:18)
    at Promise._settlePromise0 (C:\web\chat-bot-telegram\node_modules\bluebird\js\release\promise.js:649:10)
    at Promise._settlePromises (C:\web\chat-bot-telegram\node_modules\bluebird\js\release\promise.js:729:18)
    at _drainQueueStep (C:\web\chat-bot-telegram\node_modules\bluebird\js\release\async.js:93:12)
    at _drainQueue (C:\web\chat-bot-telegram\node_modules\bluebird\js\release\async.js:86:9)
    at Async._drainQueues (C:\web\chat-bot-telegram\node_modules\bluebird\js\release\async.js:102:5)
    at Async.drainQueues [as _onImmediate] (C:\web\chat-bot-telegram\node_modules\bluebird\js\release\async.js:15:14)
    at process.processImmediate (node:internal/timers:478:21)
[22.08.2024, 10:49:14] RequestError: Error: connect ECONNREFUSED 169.254.0.51:3274
    at new RequestError (C:\web\chat-bot-telegram\node_modules\request-promise-core\lib\errors.js:14:15)
    at plumbing.callback (C:\web\chat-bot-telegram\node_modules\request-promise-core\lib\plumbing.js:87:29)
    at Request.RP$callback [as _callback] (C:\web\chat-bot-telegram\node_modules\request-promise-core\lib\plumbing.js:46:31)
    at self.callback (C:\web\chat-bot-telegram\node_modules\@cypress\request\request.js:183:22)
    at Request.emit (node:events:518:28)
    at Request.onRequestError (C:\web\chat-bot-telegram\node_modules\@cypress\request\request.js:869:8)
    at ClientRequest.emit (node:events:518:28)
    at _destroy (node:_http_client:880:13)
    at onSocketNT (node:_http_client:900:5)
    at process.processTicksAndRejections (node:internal/process/task_queues:83:21)
[22.08.2024, 10:49:16] RequestError: Error: connect ECONNREFUSED 169.254.0.51:3274
    at new RequestError (C:\web\chat-bot-telegram\node_modules\request-promise-core\lib\errors.js:14:15)
    at plumbing.callback (C:\web\chat-bot-telegram\node_modules\request-promise-core\lib\plumbing.js:87:29)
    at Request.RP$callback [as _callback] (C:\web\chat-bot-telegram\node_modules\request-promise-core\lib\plumbing.js:46:31)
    at self.callback (C:\web\chat-bot-telegram\node_modules\@cypress\request\request.js:183:22)
    at Request.emit (node:events:518:28)
    at Request.onRequestError (C:\web\chat-bot-telegram\node_modules\@cypress\request\request.js:869:8)
    at ClientRequest.emit (node:events:518:28)
    at _destroy (node:_http_client:880:13)
    at onSocketNT (node:_http_client:900:5)
    at process.processTicksAndRejections (node:internal/process/task_queues:83:21)
[22.08.2024, 10:49:18] RequestError: Error: connect ECONNREFUSED 169.254.0.51:3274
    at new RequestError (C:\web\chat-bot-telegram\node_modules\request-promise-core\lib\errors.js:14:15)
    at plumbing.callback (C:\web\chat-bot-telegram\node_modules\request-promise-core\lib\plumbing.js:87:29)
    at Request.RP$callback [as _callback] (C:\web\chat-bot-telegram\node_modules\request-promise-core\lib\plumbing.js:46:31)
    at self.callback (C:\web\chat-bot-telegram\node_modules\@cypress\request\request.js:183:22)
    at Request.emit (node:events:518:28)
    at Request.onRequestError (C:\web\chat-bot-telegram\node_modules\@cypress\request\request.js:869:8)
    at ClientRequest.emit (node:events:518:28)
    at _destroy (node:_http_client:880:13)
    at onSocketNT (node:_http_client:900:5)
    at process.processTicksAndRejections (node:internal/process/task_queues:83:21)
[22.08.2024, 10:49:21] RequestError: Error: connect ECONNREFUSED 169.254.0.51:3274
    at new RequestError (C:\web\chat-bot-telegram\node_modules\request-promise-core\lib\errors.js:14:15)
    at plumbing.callback (C:\web\chat-bot-telegram\node_modules\request-promise-core\lib\plumbing.js:87:29)
    at Request.RP$callback [as _callback] (C:\web\chat-bot-telegram\node_modules\request-promise-core\lib\plumbing.js:46:31)
    at self.callback (C:\web\chat-bot-telegram\node_modules\@cypress\request\request.js:183:22)
    at Request.emit (node:events:518:28)
    at Request.onRequestError (C:\web\chat-bot-telegram\node_modules\@cypress\request\request.js:869:8)
    at ClientRequest.emit (node:events:518:28)
    at _destroy (node:_http_client:880:13)
    at onSocketNT (node:_http_client:900:5)
    at process.processTicksAndRejections (node:internal/process/task_queues:83:21)
[22.08.2024, 10:49:23] RequestError: Error: connect ECONNREFUSED 169.254.0.51:3274
    at new RequestError (C:\web\chat-bot-telegram\node_modules\request-promise-core\lib\errors.js:14:15)
    at plumbing.callback (C:\web\chat-bot-telegram\node_modules\request-promise-core\lib\plumbing.js:87:29)
    at Request.RP$callback [as _callback] (C:\web\chat-bot-telegram\node_modules\request-promise-core\lib\plumbing.js:46:31)
    at self.callback (C:\web\chat-bot-telegram\node_modules\@cypress\request\request.js:183:22)
    at Request.emit (node:events:518:28)
    at Request.onRequestError (C:\web\chat-bot-telegram\node_modules\@cypress\request\request.js:869:8)
    at ClientRequest.emit (node:events:518:28)
    at _destroy (node:_http_client:880:13)
    at onSocketNT (node:_http_client:900:5)
    at process.processTicksAndRejections (node:internal/process/task_queues:83:21)
[22.08.2024, 10:49:26] RequestError: Error: connect ECONNREFUSED 169.254.0.51:3274
    at new RequestError (C:\web\chat-bot-telegram\node_modules\request-promise-core\lib\errors.js:14:15)
    at plumbing.callback (C:\web\chat-bot-telegram\node_modules\request-promise-core\lib\plumbing.js:87:29)
    at Request.RP$callback [as _callback] (C:\web\chat-bot-telegram\node_modules\request-promise-core\lib\plumbing.js:46:31)
    at self.callback (C:\web\chat-bot-telegram\node_modules\@cypress\request\request.js:183:22)
    at Request.emit (node:events:518:28)
    at Request.onRequestError (C:\web\chat-bot-telegram\node_modules\@cypress\request\request.js:869:8)
    at ClientRequest.emit (node:events:518:28)
    at _destroy (node:_http_client:880:13)
    at onSocketNT (node:_http_client:900:5)
    at process.processTicksAndRejections (node:internal/process/task_queues:83:21)
[22.08.2024, 10:49:28] RequestError: Error: connect ECONNREFUSED 169.254.0.51:3274
    at new RequestError (C:\web\chat-bot-telegram\node_modules\request-promise-core\lib\errors.js:14:15)
    at plumbing.callback (C:\web\chat-bot-telegram\node_modules\request-promise-core\lib\plumbing.js:87:29)
    at Request.RP$callback [as _callback] (C:\web\chat-bot-telegram\node_modules\request-promise-core\lib\plumbing.js:46:31)
    at self.callback (C:\web\chat-bot-telegram\node_modules\@cypress\request\request.js:183:22)
    at Request.emit (node:events:518:28)
    at Request.onRequestError (C:\web\chat-bot-telegram\node_modules\@cypress\request\request.js:869:8)
    at ClientRequest.emit (node:events:518:28)
    at _destroy (node:_http_client:880:13)
    at onSocketNT (node:_http_client:900:5)
    at process.processTicksAndRejections (node:internal/process/task_queues:83:21)
[22.08.2024, 11:19:45] RequestError: Error: connect ECONNREFUSED 169.254.0.51:3274
    at new RequestError (C:\web\chat-bot-telegram\node_modules\request-promise-core\lib\errors.js:14:15)
    at plumbing.callback (C:\web\chat-bot-telegram\node_modules\request-promise-core\lib\plumbing.js:87:29)
    at Request.RP$callback [as _callback] (C:\web\chat-bot-telegram\node_modules\request-promise-core\lib\plumbing.js:46:31)
    at self.callback (C:\web\chat-bot-telegram\node_modules\@cypress\request\request.js:183:22)
    at Request.emit (node:events:518:28)
    at Request.onRequestError (C:\web\chat-bot-telegram\node_modules\@cypress\request\request.js:869:8)
    at ClientRequest.emit (node:events:518:28)
    at _destroy (node:_http_client:880:13)
    at onSocketNT (node:_http_client:900:5)
    at process.processTicksAndRejections (node:internal/process/task_queues:83:21)
[22.08.2024, 11:23:22] Error: ETELEGRAM: 400 Bad Request: message to edit not found
    at C:\web\chat-bot-telegram\node_modules\node-telegram-bot-api\src\telegram.js:316:15
    at tryCatcher (C:\web\chat-bot-telegram\node_modules\bluebird\js\release\util.js:16:23)
    at Promise._settlePromiseFromHandler (C:\web\chat-bot-telegram\node_modules\bluebird\js\release\promise.js:547:31)
    at Promise._settlePromise (C:\web\chat-bot-telegram\node_modules\bluebird\js\release\promise.js:604:18)
    at Promise._settlePromise0 (C:\web\chat-bot-telegram\node_modules\bluebird\js\release\promise.js:649:10)
    at Promise._settlePromises (C:\web\chat-bot-telegram\node_modules\bluebird\js\release\promise.js:729:18)
    at _drainQueueStep (C:\web\chat-bot-telegram\node_modules\bluebird\js\release\async.js:93:12)
    at _drainQueue (C:\web\chat-bot-telegram\node_modules\bluebird\js\release\async.js:86:9)
    at Async._drainQueues (C:\web\chat-bot-telegram\node_modules\bluebird\js\release\async.js:102:5)
    at Async.drainQueues [as _onImmediate] (C:\web\chat-bot-telegram\node_modules\bluebird\js\release\async.js:15:14)
    at process.processImmediate (node:internal/timers:478:21)
[22.08.2024, 11:23:37] Error: ETELEGRAM: 400 Bad Request: message to edit not found
    at C:\web\chat-bot-telegram\node_modules\node-telegram-bot-api\src\telegram.js:316:15
    at tryCatcher (C:\web\chat-bot-telegram\node_modules\bluebird\js\release\util.js:16:23)
    at Promise._settlePromiseFromHandler (C:\web\chat-bot-telegram\node_modules\bluebird\js\release\promise.js:547:31)
    at Promise._settlePromise (C:\web\chat-bot-telegram\node_modules\bluebird\js\release\promise.js:604:18)
    at Promise._settlePromise0 (C:\web\chat-bot-telegram\node_modules\bluebird\js\release\promise.js:649:10)
    at Promise._settlePromises (C:\web\chat-bot-telegram\node_modules\bluebird\js\release\promise.js:729:18)
    at _drainQueueStep (C:\web\chat-bot-telegram\node_modules\bluebird\js\release\async.js:93:12)
    at _drainQueue (C:\web\chat-bot-telegram\node_modules\bluebird\js\release\async.js:86:9)
    at Async._drainQueues (C:\web\chat-bot-telegram\node_modules\bluebird\js\release\async.js:102:5)
    at Async.drainQueues [as _onImmediate] (C:\web\chat-bot-telegram\node_modules\bluebird\js\release\async.js:15:14)
    at process.processImmediate (node:internal/timers:478:21)
[22.08.2024, 11:43:27] RequestError: Error: Proxy connection ended before receiving CONNECT response
    at new RequestError (C:\web\chat-bot-telegram\node_modules\request-promise-core\lib\errors.js:14:15)
    at plumbing.callback (C:\web\chat-bot-telegram\node_modules\request-promise-core\lib\plumbing.js:87:29)
    at Request.RP$callback [as _callback] (C:\web\chat-bot-telegram\node_modules\request-promise-core\lib\plumbing.js:46:31)
    at self.callback (C:\web\chat-bot-telegram\node_modules\@cypress\request\request.js:183:22)
    at Request.emit (node:events:518:28)
    at Request.onRequestError (C:\web\chat-bot-telegram\node_modules\@cypress\request\request.js:869:8)
    at ClientRequest.emit (node:events:518:28)
    at _destroy (node:_http_client:880:13)
    at onSocketNT (node:_http_client:900:5)
    at process.processTicksAndRejections (node:internal/process/task_queues:83:21)
[22.08.2024, 11:44:22] Error: ETELEGRAM: 400 Bad Request: message to edit not found
    at C:\web\chat-bot-telegram\node_modules\node-telegram-bot-api\src\telegram.js:316:15
    at tryCatcher (C:\web\chat-bot-telegram\node_modules\bluebird\js\release\util.js:16:23)
    at Promise._settlePromiseFromHandler (C:\web\chat-bot-telegram\node_modules\bluebird\js\release\promise.js:547:31)
    at Promise._settlePromise (C:\web\chat-bot-telegram\node_modules\bluebird\js\release\promise.js:604:18)
    at Promise._settlePromise0 (C:\web\chat-bot-telegram\node_modules\bluebird\js\release\promise.js:649:10)
    at Promise._settlePromises (C:\web\chat-bot-telegram\node_modules\bluebird\js\release\promise.js:729:18)
    at _drainQueueStep (C:\web\chat-bot-telegram\node_modules\bluebird\js\release\async.js:93:12)
    at _drainQueue (C:\web\chat-bot-telegram\node_modules\bluebird\js\release\async.js:86:9)
    at Async._drainQueues (C:\web\chat-bot-telegram\node_modules\bluebird\js\release\async.js:102:5)
    at Async.drainQueues [as _onImmediate] (C:\web\chat-bot-telegram\node_modules\bluebird\js\release\async.js:15:14)
    at process.processImmediate (node:internal/timers:478:21)
[22.08.2024, 11:44:56] Error: ETELEGRAM: 400 Bad Request: message to edit not found
    at C:\web\chat-bot-telegram\node_modules\node-telegram-bot-api\src\telegram.js:316:15
    at tryCatcher (C:\web\chat-bot-telegram\node_modules\bluebird\js\release\util.js:16:23)
    at Promise._settlePromiseFromHandler (C:\web\chat-bot-telegram\node_modules\bluebird\js\release\promise.js:547:31)
    at Promise._settlePromise (C:\web\chat-bot-telegram\node_modules\bluebird\js\release\promise.js:604:18)
    at Promise._settlePromise0 (C:\web\chat-bot-telegram\node_modules\bluebird\js\release\promise.js:649:10)
    at Promise._settlePromises (C:\web\chat-bot-telegram\node_modules\bluebird\js\release\promise.js:729:18)
    at _drainQueueStep (C:\web\chat-bot-telegram\node_modules\bluebird\js\release\async.js:93:12)
    at _drainQueue (C:\web\chat-bot-telegram\node_modules\bluebird\js\release\async.js:86:9)
    at Async._drainQueues (C:\web\chat-bot-telegram\node_modules\bluebird\js\release\async.js:102:5)
    at Async.drainQueues [as _onImmediate] (C:\web\chat-bot-telegram\node_modules\bluebird\js\release\async.js:15:14)
    at process.processImmediate (node:internal/timers:478:21)
[22.08.2024, 11:45:00] Error: ETELEGRAM: 400 Bad Request: message to edit not found
    at C:\web\chat-bot-telegram\node_modules\node-telegram-bot-api\src\telegram.js:316:15
    at tryCatcher (C:\web\chat-bot-telegram\node_modules\bluebird\js\release\util.js:16:23)
    at Promise._settlePromiseFromHandler (C:\web\chat-bot-telegram\node_modules\bluebird\js\release\promise.js:547:31)
    at Promise._settlePromise (C:\web\chat-bot-telegram\node_modules\bluebird\js\release\promise.js:604:18)
    at Promise._settlePromise0 (C:\web\chat-bot-telegram\node_modules\bluebird\js\release\promise.js:649:10)
    at Promise._settlePromises (C:\web\chat-bot-telegram\node_modules\bluebird\js\release\promise.js:729:18)
    at _drainQueueStep (C:\web\chat-bot-telegram\node_modules\bluebird\js\release\async.js:93:12)
    at _drainQueue (C:\web\chat-bot-telegram\node_modules\bluebird\js\release\async.js:86:9)
    at Async._drainQueues (C:\web\chat-bot-telegram\node_modules\bluebird\js\release\async.js:102:5)
    at Async.drainQueues [as _onImmediate] (C:\web\chat-bot-telegram\node_modules\bluebird\js\release\async.js:15:14)
    at process.processImmediate (node:internal/timers:478:21)
[22.08.2024, 11:45:19] Error: ETELEGRAM: 400 Bad Request: message to edit not found
    at C:\web\chat-bot-telegram\node_modules\node-telegram-bot-api\src\telegram.js:316:15
    at tryCatcher (C:\web\chat-bot-telegram\node_modules\bluebird\js\release\util.js:16:23)
    at Promise._settlePromiseFromHandler (C:\web\chat-bot-telegram\node_modules\bluebird\js\release\promise.js:547:31)
    at Promise._settlePromise (C:\web\chat-bot-telegram\node_modules\bluebird\js\release\promise.js:604:18)
    at Promise._settlePromise0 (C:\web\chat-bot-telegram\node_modules\bluebird\js\release\promise.js:649:10)
    at Promise._settlePromises (C:\web\chat-bot-telegram\node_modules\bluebird\js\release\promise.js:729:18)
    at _drainQueueStep (C:\web\chat-bot-telegram\node_modules\bluebird\js\release\async.js:93:12)
    at _drainQueue (C:\web\chat-bot-telegram\node_modules\bluebird\js\release\async.js:86:9)
    at Async._drainQueues (C:\web\chat-bot-telegram\node_modules\bluebird\js\release\async.js:102:5)
    at Async.drainQueues [as _onImmediate] (C:\web\chat-bot-telegram\node_modules\bluebird\js\release\async.js:15:14)
    at process.processImmediate (node:internal/timers:478:21)
[22.08.2024, 11:45:59] Error: ETELEGRAM: 400 Bad Request: message to edit not found
    at C:\web\chat-bot-telegram\node_modules\node-telegram-bot-api\src\telegram.js:316:15
    at tryCatcher (C:\web\chat-bot-telegram\node_modules\bluebird\js\release\util.js:16:23)
    at Promise._settlePromiseFromHandler (C:\web\chat-bot-telegram\node_modules\bluebird\js\release\promise.js:547:31)
    at Promise._settlePromise (C:\web\chat-bot-telegram\node_modules\bluebird\js\release\promise.js:604:18)
    at Promise._settlePromise0 (C:\web\chat-bot-telegram\node_modules\bluebird\js\release\promise.js:649:10)
    at Promise._settlePromises (C:\web\chat-bot-telegram\node_modules\bluebird\js\release\promise.js:729:18)
    at _drainQueueStep (C:\web\chat-bot-telegram\node_modules\bluebird\js\release\async.js:93:12)
    at _drainQueue (C:\web\chat-bot-telegram\node_modules\bluebird\js\release\async.js:86:9)
    at Async._drainQueues (C:\web\chat-bot-telegram\node_modules\bluebird\js\release\async.js:102:5)
    at Async.drainQueues [as _onImmediate] (C:\web\chat-bot-telegram\node_modules\bluebird\js\release\async.js:15:14)
    at process.processImmediate (node:internal/timers:478:21)
[22.08.2024, 11:54:05] Error: ETELEGRAM: 400 Bad Request: message to edit not found
    at C:\web\chat-bot-telegram\node_modules\node-telegram-bot-api\src\telegram.js:316:15
    at tryCatcher (C:\web\chat-bot-telegram\node_modules\bluebird\js\release\util.js:16:23)
    at Promise._settlePromiseFromHandler (C:\web\chat-bot-telegram\node_modules\bluebird\js\release\promise.js:547:31)
    at Promise._settlePromise (C:\web\chat-bot-telegram\node_modules\bluebird\js\release\promise.js:604:18)
    at Promise._settlePromise0 (C:\web\chat-bot-telegram\node_modules\bluebird\js\release\promise.js:649:10)
    at Promise._settlePromises (C:\web\chat-bot-telegram\node_modules\bluebird\js\release\promise.js:729:18)
    at _drainQueueStep (C:\web\chat-bot-telegram\node_modules\bluebird\js\release\async.js:93:12)
    at _drainQueue (C:\web\chat-bot-telegram\node_modules\bluebird\js\release\async.js:86:9)
    at Async._drainQueues (C:\web\chat-bot-telegram\node_modules\bluebird\js\release\async.js:102:5)
    at Async.drainQueues [as _onImmediate] (C:\web\chat-bot-telegram\node_modules\bluebird\js\release\async.js:15:14)
    at process.processImmediate (node:internal/timers:478:21)
[22.08.2024, 11:54:13] Error: ETELEGRAM: 400 Bad Request: message to edit not found
    at C:\web\chat-bot-telegram\node_modules\node-telegram-bot-api\src\telegram.js:316:15
    at tryCatcher (C:\web\chat-bot-telegram\node_modules\bluebird\js\release\util.js:16:23)
    at Promise._settlePromiseFromHandler (C:\web\chat-bot-telegram\node_modules\bluebird\js\release\promise.js:547:31)
    at Promise._settlePromise (C:\web\chat-bot-telegram\node_modules\bluebird\js\release\promise.js:604:18)
    at Promise._settlePromise0 (C:\web\chat-bot-telegram\node_modules\bluebird\js\release\promise.js:649:10)
    at Promise._settlePromises (C:\web\chat-bot-telegram\node_modules\bluebird\js\release\promise.js:729:18)
    at _drainQueueStep (C:\web\chat-bot-telegram\node_modules\bluebird\js\release\async.js:93:12)
    at _drainQueue (C:\web\chat-bot-telegram\node_modules\bluebird\js\release\async.js:86:9)
    at Async._drainQueues (C:\web\chat-bot-telegram\node_modules\bluebird\js\release\async.js:102:5)
    at Async.drainQueues [as _onImmediate] (C:\web\chat-bot-telegram\node_modules\bluebird\js\release\async.js:15:14)
    at process.processImmediate (node:internal/timers:478:21)
[22.08.2024, 11:54:56] Error: ETELEGRAM: 400 Bad Request: there is no text in the message to edit
    at C:\web\chat-bot-telegram\node_modules\node-telegram-bot-api\src\telegram.js:316:15
    at tryCatcher (C:\web\chat-bot-telegram\node_modules\bluebird\js\release\util.js:16:23)
    at Promise._settlePromiseFromHandler (C:\web\chat-bot-telegram\node_modules\bluebird\js\release\promise.js:547:31)
    at Promise._settlePromise (C:\web\chat-bot-telegram\node_modules\bluebird\js\release\promise.js:604:18)
    at Promise._settlePromise0 (C:\web\chat-bot-telegram\node_modules\bluebird\js\release\promise.js:649:10)
    at Promise._settlePromises (C:\web\chat-bot-telegram\node_modules\bluebird\js\release\promise.js:729:18)
    at _drainQueueStep (C:\web\chat-bot-telegram\node_modules\bluebird\js\release\async.js:93:12)
    at _drainQueue (C:\web\chat-bot-telegram\node_modules\bluebird\js\release\async.js:86:9)
    at Async._drainQueues (C:\web\chat-bot-telegram\node_modules\bluebird\js\release\async.js:102:5)
    at Async.drainQueues [as _onImmediate] (C:\web\chat-bot-telegram\node_modules\bluebird\js\release\async.js:15:14)
    at process.processImmediate (node:internal/timers:478:21)
[22.08.2024, 11:55:02] Error: ETELEGRAM: 400 Bad Request: there is no text in the message to edit
    at C:\web\chat-bot-telegram\node_modules\node-telegram-bot-api\src\telegram.js:316:15
    at tryCatcher (C:\web\chat-bot-telegram\node_modules\bluebird\js\release\util.js:16:23)
    at Promise._settlePromiseFromHandler (C:\web\chat-bot-telegram\node_modules\bluebird\js\release\promise.js:547:31)
    at Promise._settlePromise (C:\web\chat-bot-telegram\node_modules\bluebird\js\release\promise.js:604:18)
    at Promise._settlePromise0 (C:\web\chat-bot-telegram\node_modules\bluebird\js\release\promise.js:649:10)
    at Promise._settlePromises (C:\web\chat-bot-telegram\node_modules\bluebird\js\release\promise.js:729:18)
    at _drainQueueStep (C:\web\chat-bot-telegram\node_modules\bluebird\js\release\async.js:93:12)
    at _drainQueue (C:\web\chat-bot-telegram\node_modules\bluebird\js\release\async.js:86:9)
    at Async._drainQueues (C:\web\chat-bot-telegram\node_modules\bluebird\js\release\async.js:102:5)
    at Async.drainQueues [as _onImmediate] (C:\web\chat-bot-telegram\node_modules\bluebird\js\release\async.js:15:14)
    at process.processImmediate (node:internal/timers:478:21)
[22.08.2024, 11:55:14] Error: ETELEGRAM: 400 Bad Request: there is no text in the message to edit
    at C:\web\chat-bot-telegram\node_modules\node-telegram-bot-api\src\telegram.js:316:15
    at tryCatcher (C:\web\chat-bot-telegram\node_modules\bluebird\js\release\util.js:16:23)
    at Promise._settlePromiseFromHandler (C:\web\chat-bot-telegram\node_modules\bluebird\js\release\promise.js:547:31)
    at Promise._settlePromise (C:\web\chat-bot-telegram\node_modules\bluebird\js\release\promise.js:604:18)
    at Promise._settlePromise0 (C:\web\chat-bot-telegram\node_modules\bluebird\js\release\promise.js:649:10)
    at Promise._settlePromises (C:\web\chat-bot-telegram\node_modules\bluebird\js\release\promise.js:729:18)
    at _drainQueueStep (C:\web\chat-bot-telegram\node_modules\bluebird\js\release\async.js:93:12)
    at _drainQueue (C:\web\chat-bot-telegram\node_modules\bluebird\js\release\async.js:86:9)
    at Async._drainQueues (C:\web\chat-bot-telegram\node_modules\bluebird\js\release\async.js:102:5)
    at Async.drainQueues [as _onImmediate] (C:\web\chat-bot-telegram\node_modules\bluebird\js\release\async.js:15:14)
    at process.processImmediate (node:internal/timers:478:21)
[22.08.2024, 11:55:26] Error: ETELEGRAM: 400 Bad Request: message to edit not found
    at C:\web\chat-bot-telegram\node_modules\node-telegram-bot-api\src\telegram.js:316:15
    at tryCatcher (C:\web\chat-bot-telegram\node_modules\bluebird\js\release\util.js:16:23)
    at Promise._settlePromiseFromHandler (C:\web\chat-bot-telegram\node_modules\bluebird\js\release\promise.js:547:31)
    at Promise._settlePromise (C:\web\chat-bot-telegram\node_modules\bluebird\js\release\promise.js:604:18)
    at Promise._settlePromise0 (C:\web\chat-bot-telegram\node_modules\bluebird\js\release\promise.js:649:10)
    at Promise._settlePromises (C:\web\chat-bot-telegram\node_modules\bluebird\js\release\promise.js:729:18)
    at _drainQueueStep (C:\web\chat-bot-telegram\node_modules\bluebird\js\release\async.js:93:12)
    at _drainQueue (C:\web\chat-bot-telegram\node_modules\bluebird\js\release\async.js:86:9)
    at Async._drainQueues (C:\web\chat-bot-telegram\node_modules\bluebird\js\release\async.js:102:5)
    at Async.drainQueues [as _onImmediate] (C:\web\chat-bot-telegram\node_modules\bluebird\js\release\async.js:15:14)
    at process.processImmediate (node:internal/timers:478:21)
[22.08.2024, 11:56:23] Error: ETELEGRAM: 400 Bad Request: there is no text in the message to edit
    at C:\web\chat-bot-telegram\node_modules\node-telegram-bot-api\src\telegram.js:316:15
    at tryCatcher (C:\web\chat-bot-telegram\node_modules\bluebird\js\release\util.js:16:23)
    at Promise._settlePromiseFromHandler (C:\web\chat-bot-telegram\node_modules\bluebird\js\release\promise.js:547:31)
    at Promise._settlePromise (C:\web\chat-bot-telegram\node_modules\bluebird\js\release\promise.js:604:18)
    at Promise._settlePromise0 (C:\web\chat-bot-telegram\node_modules\bluebird\js\release\promise.js:649:10)
    at Promise._settlePromises (C:\web\chat-bot-telegram\node_modules\bluebird\js\release\promise.js:729:18)
    at _drainQueueStep (C:\web\chat-bot-telegram\node_modules\bluebird\js\release\async.js:93:12)
    at _drainQueue (C:\web\chat-bot-telegram\node_modules\bluebird\js\release\async.js:86:9)
    at Async._drainQueues (C:\web\chat-bot-telegram\node_modules\bluebird\js\release\async.js:102:5)
    at Async.drainQueues [as _onImmediate] (C:\web\chat-bot-telegram\node_modules\bluebird\js\release\async.js:15:14)
    at process.processImmediate (node:internal/timers:478:21)
[22.08.2024, 12:09:08] Error: ETELEGRAM: 400 Bad Request: message to edit not found
    at C:\web\chat-bot-telegram\node_modules\node-telegram-bot-api\src\telegram.js:316:15
    at tryCatcher (C:\web\chat-bot-telegram\node_modules\bluebird\js\release\util.js:16:23)
    at Promise._settlePromiseFromHandler (C:\web\chat-bot-telegram\node_modules\bluebird\js\release\promise.js:547:31)
    at Promise._settlePromise (C:\web\chat-bot-telegram\node_modules\bluebird\js\release\promise.js:604:18)
    at Promise._settlePromise0 (C:\web\chat-bot-telegram\node_modules\bluebird\js\release\promise.js:649:10)
    at Promise._settlePromises (C:\web\chat-bot-telegram\node_modules\bluebird\js\release\promise.js:729:18)
    at _drainQueueStep (C:\web\chat-bot-telegram\node_modules\bluebird\js\release\async.js:93:12)
    at _drainQueue (C:\web\chat-bot-telegram\node_modules\bluebird\js\release\async.js:86:9)
    at Async._drainQueues (C:\web\chat-bot-telegram\node_modules\bluebird\js\release\async.js:102:5)
    at Async.drainQueues [as _onImmediate] (C:\web\chat-bot-telegram\node_modules\bluebird\js\release\async.js:15:14)
    at process.processImmediate (node:internal/timers:478:21)
[22.08.2024, 12:09:11] Error: ETELEGRAM: 400 Bad Request: message to edit not found
    at C:\web\chat-bot-telegram\node_modules\node-telegram-bot-api\src\telegram.js:316:15
    at tryCatcher (C:\web\chat-bot-telegram\node_modules\bluebird\js\release\util.js:16:23)
    at Promise._settlePromiseFromHandler (C:\web\chat-bot-telegram\node_modules\bluebird\js\release\promise.js:547:31)
    at Promise._settlePromise (C:\web\chat-bot-telegram\node_modules\bluebird\js\release\promise.js:604:18)
    at Promise._settlePromise0 (C:\web\chat-bot-telegram\node_modules\bluebird\js\release\promise.js:649:10)
    at Promise._settlePromises (C:\web\chat-bot-telegram\node_modules\bluebird\js\release\promise.js:729:18)
    at _drainQueueStep (C:\web\chat-bot-telegram\node_modules\bluebird\js\release\async.js:93:12)
    at _drainQueue (C:\web\chat-bot-telegram\node_modules\bluebird\js\release\async.js:86:9)
    at Async._drainQueues (C:\web\chat-bot-telegram\node_modules\bluebird\js\release\async.js:102:5)
    at Async.drainQueues [as _onImmediate] (C:\web\chat-bot-telegram\node_modules\bluebird\js\release\async.js:15:14)
    at process.processImmediate (node:internal/timers:478:21)
[22.08.2024, 12:09:25] Error: ETELEGRAM: 400 Bad Request: message is not modified: specified new message content and reply markup are exactly the same as a current content and reply markup of the message
    at C:\web\chat-bot-telegram\node_modules\node-telegram-bot-api\src\telegram.js:316:15
    at tryCatcher (C:\web\chat-bot-telegram\node_modules\bluebird\js\release\util.js:16:23)
    at Promise._settlePromiseFromHandler (C:\web\chat-bot-telegram\node_modules\bluebird\js\release\promise.js:547:31)
    at Promise._settlePromise (C:\web\chat-bot-telegram\node_modules\bluebird\js\release\promise.js:604:18)
    at Promise._settlePromise0 (C:\web\chat-bot-telegram\node_modules\bluebird\js\release\promise.js:649:10)
    at Promise._settlePromises (C:\web\chat-bot-telegram\node_modules\bluebird\js\release\promise.js:729:18)
    at _drainQueueStep (C:\web\chat-bot-telegram\node_modules\bluebird\js\release\async.js:93:12)
    at _drainQueue (C:\web\chat-bot-telegram\node_modules\bluebird\js\release\async.js:86:9)
    at Async._drainQueues (C:\web\chat-bot-telegram\node_modules\bluebird\js\release\async.js:102:5)
    at Async.drainQueues [as _onImmediate] (C:\web\chat-bot-telegram\node_modules\bluebird\js\release\async.js:15:14)
    at process.processImmediate (node:internal/timers:478:21)
[22.08.2024, 12:52:51] ReferenceError: msg is not defined
    at handleMessage (file:///C:/web/chat-bot-telegram/src/telegram-bot/buttonHandlers.js:24:3)
    at TelegramBot.<anonymous> (file:///C:/web/chat-bot-telegram/src/telegram-bot/telegramBot.js:18:7)
    at TelegramBot.emit (C:\web\chat-bot-telegram\node_modules\eventemitter3\index.js:182:35)
    at TelegramBot.processUpdate (C:\web\chat-bot-telegram\node_modules\node-telegram-bot-api\src\telegram.js:706:12)
    at C:\web\chat-bot-telegram\node_modules\node-telegram-bot-api\src\telegramPolling.js:110:22
    at Array.forEach (<anonymous>)
    at C:\web\chat-bot-telegram\node_modules\node-telegram-bot-api\src\telegramPolling.js:106:17
    at tryCatcher (C:\web\chat-bot-telegram\node_modules\bluebird\js\release\util.js:16:23)
    at Promise._settlePromiseFromHandler (C:\web\chat-bot-telegram\node_modules\bluebird\js\release\promise.js:547:31)
    at Promise._settlePromise (C:\web\chat-bot-telegram\node_modules\bluebird\js\release\promise.js:604:18)
    at Promise._settlePromise0 (C:\web\chat-bot-telegram\node_modules\bluebird\js\release\promise.js:649:10)
    at Promise._settlePromises (C:\web\chat-bot-telegram\node_modules\bluebird\js\release\promise.js:729:18)
    at _drainQueueStep (C:\web\chat-bot-telegram\node_modules\bluebird\js\release\async.js:93:12)
    at _drainQueue (C:\web\chat-bot-telegram\node_modules\bluebird\js\release\async.js:86:9)
    at Async._drainQueues (C:\web\chat-bot-telegram\node_modules\bluebird\js\release\async.js:102:5)
    at Async.drainQueues [as _onImmediate] (C:\web\chat-bot-telegram\node_modules\bluebird\js\release\async.js:15:14)
    at process.processImmediate (node:internal/timers:478:21)
[22.08.2024, 12:53:20] ReferenceError: msg is not defined
    at handleMessage (file:///C:/web/chat-bot-telegram/src/telegram-bot/buttonHandlers.js:24:3)
    at TelegramBot.<anonymous> (file:///C:/web/chat-bot-telegram/src/telegram-bot/telegramBot.js:18:7)
    at TelegramBot.emit (C:\web\chat-bot-telegram\node_modules\eventemitter3\index.js:182:35)
    at TelegramBot.processUpdate (C:\web\chat-bot-telegram\node_modules\node-telegram-bot-api\src\telegram.js:706:12)
    at C:\web\chat-bot-telegram\node_modules\node-telegram-bot-api\src\telegramPolling.js:110:22
    at Array.forEach (<anonymous>)
    at C:\web\chat-bot-telegram\node_modules\node-telegram-bot-api\src\telegramPolling.js:106:17
    at tryCatcher (C:\web\chat-bot-telegram\node_modules\bluebird\js\release\util.js:16:23)
    at Promise._settlePromiseFromHandler (C:\web\chat-bot-telegram\node_modules\bluebird\js\release\promise.js:547:31)
    at Promise._settlePromise (C:\web\chat-bot-telegram\node_modules\bluebird\js\release\promise.js:604:18)
    at Promise._settlePromise0 (C:\web\chat-bot-telegram\node_modules\bluebird\js\release\promise.js:649:10)
    at Promise._settlePromises (C:\web\chat-bot-telegram\node_modules\bluebird\js\release\promise.js:729:18)
    at _drainQueueStep (C:\web\chat-bot-telegram\node_modules\bluebird\js\release\async.js:93:12)
    at _drainQueue (C:\web\chat-bot-telegram\node_modules\bluebird\js\release\async.js:86:9)
    at Async._drainQueues (C:\web\chat-bot-telegram\node_modules\bluebird\js\release\async.js:102:5)
    at Async.drainQueues [as _onImmediate] (C:\web\chat-bot-telegram\node_modules\bluebird\js\release\async.js:15:14)
    at process.processImmediate (node:internal/timers:478:21)
[22.08.2024, 12:56:23] TypeError: Cannot read properties of undefined (reading 'id')
    at handleMessage (file:///C:/web/chat-bot-telegram/src/telegram-bot/buttonHandlers.js:24:27)
    at TelegramBot.<anonymous> (file:///C:/web/chat-bot-telegram/src/telegram-bot/telegramBot.js:18:7)
    at TelegramBot.emit (C:\web\chat-bot-telegram\node_modules\eventemitter3\index.js:182:35)
    at TelegramBot.processUpdate (C:\web\chat-bot-telegram\node_modules\node-telegram-bot-api\src\telegram.js:706:12)
    at C:\web\chat-bot-telegram\node_modules\node-telegram-bot-api\src\telegramPolling.js:110:22
    at Array.forEach (<anonymous>)
    at C:\web\chat-bot-telegram\node_modules\node-telegram-bot-api\src\telegramPolling.js:106:17
    at tryCatcher (C:\web\chat-bot-telegram\node_modules\bluebird\js\release\util.js:16:23)
    at Promise._settlePromiseFromHandler (C:\web\chat-bot-telegram\node_modules\bluebird\js\release\promise.js:547:31)
    at Promise._settlePromise (C:\web\chat-bot-telegram\node_modules\bluebird\js\release\promise.js:604:18)
    at Promise._settlePromise0 (C:\web\chat-bot-telegram\node_modules\bluebird\js\release\promise.js:649:10)
    at Promise._settlePromises (C:\web\chat-bot-telegram\node_modules\bluebird\js\release\promise.js:729:18)
    at _drainQueueStep (C:\web\chat-bot-telegram\node_modules\bluebird\js\release\async.js:93:12)
    at _drainQueue (C:\web\chat-bot-telegram\node_modules\bluebird\js\release\async.js:86:9)
    at Async._drainQueues (C:\web\chat-bot-telegram\node_modules\bluebird\js\release\async.js:102:5)
    at Async.drainQueues [as _onImmediate] (C:\web\chat-bot-telegram\node_modules\bluebird\js\release\async.js:15:14)
    at process.processImmediate (node:internal/timers:478:21)
[22.08.2024, 12:57:48] ReferenceError: msg is not defined
    at handleMessage (file:///C:/web/chat-bot-telegram/src/telegram-bot/buttonHandlers.js:24:3)
    at TelegramBot.<anonymous> (file:///C:/web/chat-bot-telegram/src/telegram-bot/telegramBot.js:18:7)
    at TelegramBot.emit (C:\web\chat-bot-telegram\node_modules\eventemitter3\index.js:182:35)
    at TelegramBot.processUpdate (C:\web\chat-bot-telegram\node_modules\node-telegram-bot-api\src\telegram.js:706:12)
    at C:\web\chat-bot-telegram\node_modules\node-telegram-bot-api\src\telegramPolling.js:110:22
    at Array.forEach (<anonymous>)
    at C:\web\chat-bot-telegram\node_modules\node-telegram-bot-api\src\telegramPolling.js:106:17
    at tryCatcher (C:\web\chat-bot-telegram\node_modules\bluebird\js\release\util.js:16:23)
    at Promise._settlePromiseFromHandler (C:\web\chat-bot-telegram\node_modules\bluebird\js\release\promise.js:547:31)
    at Promise._settlePromise (C:\web\chat-bot-telegram\node_modules\bluebird\js\release\promise.js:604:18)
    at Promise._settlePromise0 (C:\web\chat-bot-telegram\node_modules\bluebird\js\release\promise.js:649:10)
    at Promise._settlePromises (C:\web\chat-bot-telegram\node_modules\bluebird\js\release\promise.js:729:18)
    at _drainQueueStep (C:\web\chat-bot-telegram\node_modules\bluebird\js\release\async.js:93:12)
    at _drainQueue (C:\web\chat-bot-telegram\node_modules\bluebird\js\release\async.js:86:9)
    at Async._drainQueues (C:\web\chat-bot-telegram\node_modules\bluebird\js\release\async.js:102:5)
    at Async.drainQueues [as _onImmediate] (C:\web\chat-bot-telegram\node_modules\bluebird\js\release\async.js:15:14)
    at process.processImmediate (node:internal/timers:478:21)
[22.08.2024, 13:02:12] RequestError: Error: connect ECONNREFUSED 169.254.0.51:3274
    at new RequestError (C:\web\chat-bot-telegram\node_modules\request-promise-core\lib\errors.js:14:15)
    at plumbing.callback (C:\web\chat-bot-telegram\node_modules\request-promise-core\lib\plumbing.js:87:29)
    at Request.RP$callback [as _callback] (C:\web\chat-bot-telegram\node_modules\request-promise-core\lib\plumbing.js:46:31)
    at self.callback (C:\web\chat-bot-telegram\node_modules\@cypress\request\request.js:183:22)
    at Request.emit (node:events:518:28)
    at Request.onRequestError (C:\web\chat-bot-telegram\node_modules\@cypress\request\request.js:869:8)
    at ClientRequest.emit (node:events:518:28)
    at _destroy (node:_http_client:880:13)
    at onSocketNT (node:_http_client:900:5)
    at process.processTicksAndRejections (node:internal/process/task_queues:83:21)
[22.08.2024, 13:02:14] RequestError: Error: connect ECONNREFUSED 169.254.0.51:3274
    at new RequestError (C:\web\chat-bot-telegram\node_modules\request-promise-core\lib\errors.js:14:15)
    at plumbing.callback (C:\web\chat-bot-telegram\node_modules\request-promise-core\lib\plumbing.js:87:29)
    at Request.RP$callback [as _callback] (C:\web\chat-bot-telegram\node_modules\request-promise-core\lib\plumbing.js:46:31)
    at self.callback (C:\web\chat-bot-telegram\node_modules\@cypress\request\request.js:183:22)
    at Request.emit (node:events:518:28)
    at Request.onRequestError (C:\web\chat-bot-telegram\node_modules\@cypress\request\request.js:869:8)
    at ClientRequest.emit (node:events:518:28)
    at _destroy (node:_http_client:880:13)
    at onSocketNT (node:_http_client:900:5)
    at process.processTicksAndRejections (node:internal/process/task_queues:83:21)
[22.08.2024, 13:02:16] RequestError: Error: connect ECONNREFUSED 169.254.0.51:3274
    at new RequestError (C:\web\chat-bot-telegram\node_modules\request-promise-core\lib\errors.js:14:15)
    at plumbing.callback (C:\web\chat-bot-telegram\node_modules\request-promise-core\lib\plumbing.js:87:29)
    at Request.RP$callback [as _callback] (C:\web\chat-bot-telegram\node_modules\request-promise-core\lib\plumbing.js:46:31)
    at self.callback (C:\web\chat-bot-telegram\node_modules\@cypress\request\request.js:183:22)
    at Request.emit (node:events:518:28)
    at Request.onRequestError (C:\web\chat-bot-telegram\node_modules\@cypress\request\request.js:869:8)
    at ClientRequest.emit (node:events:518:28)
    at _destroy (node:_http_client:880:13)
    at onSocketNT (node:_http_client:900:5)
    at process.processTicksAndRejections (node:internal/process/task_queues:83:21)
[22.08.2024, 13:33:32] Error: ETELEGRAM: 400 Bad Request: message to edit not found
    at C:\web\chat-bot-telegram\node_modules\node-telegram-bot-api\src\telegram.js:316:15
    at tryCatcher (C:\web\chat-bot-telegram\node_modules\bluebird\js\release\util.js:16:23)
    at Promise._settlePromiseFromHandler (C:\web\chat-bot-telegram\node_modules\bluebird\js\release\promise.js:547:31)
    at Promise._settlePromise (C:\web\chat-bot-telegram\node_modules\bluebird\js\release\promise.js:604:18)
    at Promise._settlePromise0 (C:\web\chat-bot-telegram\node_modules\bluebird\js\release\promise.js:649:10)
    at Promise._settlePromises (C:\web\chat-bot-telegram\node_modules\bluebird\js\release\promise.js:729:18)
    at _drainQueueStep (C:\web\chat-bot-telegram\node_modules\bluebird\js\release\async.js:93:12)
    at _drainQueue (C:\web\chat-bot-telegram\node_modules\bluebird\js\release\async.js:86:9)
    at Async._drainQueues (C:\web\chat-bot-telegram\node_modules\bluebird\js\release\async.js:102:5)
    at Async.drainQueues [as _onImmediate] (C:\web\chat-bot-telegram\node_modules\bluebird\js\release\async.js:15:14)
    at process.processImmediate (node:internal/timers:478:21)
[22.08.2024, 13:33:56] Error: ETELEGRAM: 400 Bad Request: message to edit not found
    at C:\web\chat-bot-telegram\node_modules\node-telegram-bot-api\src\telegram.js:316:15
    at tryCatcher (C:\web\chat-bot-telegram\node_modules\bluebird\js\release\util.js:16:23)
    at Promise._settlePromiseFromHandler (C:\web\chat-bot-telegram\node_modules\bluebird\js\release\promise.js:547:31)
    at Promise._settlePromise (C:\web\chat-bot-telegram\node_modules\bluebird\js\release\promise.js:604:18)
    at Promise._settlePromise0 (C:\web\chat-bot-telegram\node_modules\bluebird\js\release\promise.js:649:10)
    at Promise._settlePromises (C:\web\chat-bot-telegram\node_modules\bluebird\js\release\promise.js:729:18)
    at _drainQueueStep (C:\web\chat-bot-telegram\node_modules\bluebird\js\release\async.js:93:12)
    at _drainQueue (C:\web\chat-bot-telegram\node_modules\bluebird\js\release\async.js:86:9)
    at Async._drainQueues (C:\web\chat-bot-telegram\node_modules\bluebird\js\release\async.js:102:5)
    at Async.drainQueues [as _onImmediate] (C:\web\chat-bot-telegram\node_modules\bluebird\js\release\async.js:15:14)
    at process.processImmediate (node:internal/timers:478:21)
[22.08.2024, 13:34:03] Error: ETELEGRAM: 400 Bad Request: message to edit not found
    at C:\web\chat-bot-telegram\node_modules\node-telegram-bot-api\src\telegram.js:316:15
    at tryCatcher (C:\web\chat-bot-telegram\node_modules\bluebird\js\release\util.js:16:23)
    at Promise._settlePromiseFromHandler (C:\web\chat-bot-telegram\node_modules\bluebird\js\release\promise.js:547:31)
    at Promise._settlePromise (C:\web\chat-bot-telegram\node_modules\bluebird\js\release\promise.js:604:18)
    at Promise._settlePromise0 (C:\web\chat-bot-telegram\node_modules\bluebird\js\release\promise.js:649:10)
    at Promise._settlePromises (C:\web\chat-bot-telegram\node_modules\bluebird\js\release\promise.js:729:18)
    at _drainQueueStep (C:\web\chat-bot-telegram\node_modules\bluebird\js\release\async.js:93:12)
    at _drainQueue (C:\web\chat-bot-telegram\node_modules\bluebird\js\release\async.js:86:9)
    at Async._drainQueues (C:\web\chat-bot-telegram\node_modules\bluebird\js\release\async.js:102:5)
    at Async.drainQueues [as _onImmediate] (C:\web\chat-bot-telegram\node_modules\bluebird\js\release\async.js:15:14)
    at process.processImmediate (node:internal/timers:478:21)
[22.08.2024, 13:36:08] Error: ETELEGRAM: 400 Bad Request: message to edit not found
    at C:\web\chat-bot-telegram\node_modules\node-telegram-bot-api\src\telegram.js:316:15
    at tryCatcher (C:\web\chat-bot-telegram\node_modules\bluebird\js\release\util.js:16:23)
    at Promise._settlePromiseFromHandler (C:\web\chat-bot-telegram\node_modules\bluebird\js\release\promise.js:547:31)
    at Promise._settlePromise (C:\web\chat-bot-telegram\node_modules\bluebird\js\release\promise.js:604:18)
    at Promise._settlePromise0 (C:\web\chat-bot-telegram\node_modules\bluebird\js\release\promise.js:649:10)
    at Promise._settlePromises (C:\web\chat-bot-telegram\node_modules\bluebird\js\release\promise.js:729:18)
    at _drainQueueStep (C:\web\chat-bot-telegram\node_modules\bluebird\js\release\async.js:93:12)
    at _drainQueue (C:\web\chat-bot-telegram\node_modules\bluebird\js\release\async.js:86:9)
    at Async._drainQueues (C:\web\chat-bot-telegram\node_modules\bluebird\js\release\async.js:102:5)
    at Async.drainQueues [as _onImmediate] (C:\web\chat-bot-telegram\node_modules\bluebird\js\release\async.js:15:14)
    at process.processImmediate (node:internal/timers:478:21)
[22.08.2024, 13:36:39] RequestError: Error: Proxy connection ended before receiving CONNECT response
    at new RequestError (C:\web\chat-bot-telegram\node_modules\request-promise-core\lib\errors.js:14:15)
    at plumbing.callback (C:\web\chat-bot-telegram\node_modules\request-promise-core\lib\plumbing.js:87:29)
    at Request.RP$callback [as _callback] (C:\web\chat-bot-telegram\node_modules\request-promise-core\lib\plumbing.js:46:31)
    at self.callback (C:\web\chat-bot-telegram\node_modules\@cypress\request\request.js:183:22)
    at Request.emit (node:events:518:28)
    at Request.onRequestError (C:\web\chat-bot-telegram\node_modules\@cypress\request\request.js:869:8)
    at ClientRequest.emit (node:events:518:28)
    at _destroy (node:_http_client:880:13)
    at onSocketNT (node:_http_client:900:5)
    at process.processTicksAndRejections (node:internal/process/task_queues:83:21)
[22.08.2024, 13:49:24] RequestError: Error: connect ECONNREFUSED 169.254.0.51:3274
    at new RequestError (C:\web\chat-bot-telegram\node_modules\request-promise-core\lib\errors.js:14:15)
    at plumbing.callback (C:\web\chat-bot-telegram\node_modules\request-promise-core\lib\plumbing.js:87:29)
    at Request.RP$callback [as _callback] (C:\web\chat-bot-telegram\node_modules\request-promise-core\lib\plumbing.js:46:31)
    at self.callback (C:\web\chat-bot-telegram\node_modules\@cypress\request\request.js:183:22)
    at Request.emit (node:events:518:28)
    at Request.onRequestError (C:\web\chat-bot-telegram\node_modules\@cypress\request\request.js:869:8)
    at ClientRequest.emit (node:events:518:28)
    at _destroy (node:_http_client:880:13)
    at onSocketNT (node:_http_client:900:5)
    at process.processTicksAndRejections (node:internal/process/task_queues:83:21)
[22.08.2024, 13:49:26] RequestError: Error: connect ECONNREFUSED 169.254.0.51:3274
    at new RequestError (C:\web\chat-bot-telegram\node_modules\request-promise-core\lib\errors.js:14:15)
    at plumbing.callback (C:\web\chat-bot-telegram\node_modules\request-promise-core\lib\plumbing.js:87:29)
    at Request.RP$callback [as _callback] (C:\web\chat-bot-telegram\node_modules\request-promise-core\lib\plumbing.js:46:31)
    at self.callback (C:\web\chat-bot-telegram\node_modules\@cypress\request\request.js:183:22)
    at Request.emit (node:events:518:28)
    at Request.onRequestError (C:\web\chat-bot-telegram\node_modules\@cypress\request\request.js:869:8)
    at ClientRequest.emit (node:events:518:28)
    at _destroy (node:_http_client:880:13)
    at onSocketNT (node:_http_client:900:5)
    at process.processTicksAndRejections (node:internal/process/task_queues:83:21)
[22.08.2024, 13:49:31] RequestError: Error: connect ECONNREFUSED 169.254.0.51:3274
    at new RequestError (C:\web\chat-bot-telegram\node_modules\request-promise-core\lib\errors.js:14:15)
    at plumbing.callback (C:\web\chat-bot-telegram\node_modules\request-promise-core\lib\plumbing.js:87:29)
    at Request.RP$callback [as _callback] (C:\web\chat-bot-telegram\node_modules\request-promise-core\lib\plumbing.js:46:31)
    at self.callback (C:\web\chat-bot-telegram\node_modules\@cypress\request\request.js:183:22)
    at Request.emit (node:events:518:28)
    at Request.onRequestError (C:\web\chat-bot-telegram\node_modules\@cypress\request\request.js:869:8)
    at ClientRequest.emit (node:events:518:28)
    at _destroy (node:_http_client:880:13)
    at onSocketNT (node:_http_client:900:5)
    at process.processTicksAndRejections (node:internal/process/task_queues:83:21)
[22.08.2024, 13:49:33] RequestError: Error: connect ECONNREFUSED 169.254.0.51:3274
    at new RequestError (C:\web\chat-bot-telegram\node_modules\request-promise-core\lib\errors.js:14:15)
    at plumbing.callback (C:\web\chat-bot-telegram\node_modules\request-promise-core\lib\plumbing.js:87:29)
    at Request.RP$callback [as _callback] (C:\web\chat-bot-telegram\node_modules\request-promise-core\lib\plumbing.js:46:31)
    at self.callback (C:\web\chat-bot-telegram\node_modules\@cypress\request\request.js:183:22)
    at Request.emit (node:events:518:28)
    at Request.onRequestError (C:\web\chat-bot-telegram\node_modules\@cypress\request\request.js:869:8)
    at ClientRequest.emit (node:events:518:28)
    at _destroy (node:_http_client:880:13)
    at onSocketNT (node:_http_client:900:5)
    at process.processTicksAndRejections (node:internal/process/task_queues:83:21)
[22.08.2024, 13:49:36] RequestError: Error: connect ECONNREFUSED 169.254.0.51:3274
    at new RequestError (C:\web\chat-bot-telegram\node_modules\request-promise-core\lib\errors.js:14:15)
    at plumbing.callback (C:\web\chat-bot-telegram\node_modules\request-promise-core\lib\plumbing.js:87:29)
    at Request.RP$callback [as _callback] (C:\web\chat-bot-telegram\node_modules\request-promise-core\lib\plumbing.js:46:31)
    at self.callback (C:\web\chat-bot-telegram\node_modules\@cypress\request\request.js:183:22)
    at Request.emit (node:events:518:28)
    at Request.onRequestError (C:\web\chat-bot-telegram\node_modules\@cypress\request\request.js:869:8)
    at ClientRequest.emit (node:events:518:28)
    at _destroy (node:_http_client:880:13)
    at onSocketNT (node:_http_client:900:5)
    at process.processTicksAndRejections (node:internal/process/task_queues:83:21)
[22.08.2024, 13:55:53] RequestError: Error: connect ECONNREFUSED 169.254.0.51:3274
    at new RequestError (C:\web\chat-bot-telegram\node_modules\request-promise-core\lib\errors.js:14:15)
    at plumbing.callback (C:\web\chat-bot-telegram\node_modules\request-promise-core\lib\plumbing.js:87:29)
    at Request.RP$callback [as _callback] (C:\web\chat-bot-telegram\node_modules\request-promise-core\lib\plumbing.js:46:31)
    at self.callback (C:\web\chat-bot-telegram\node_modules\@cypress\request\request.js:183:22)
    at Request.emit (node:events:518:28)
    at Request.onRequestError (C:\web\chat-bot-telegram\node_modules\@cypress\request\request.js:869:8)
    at ClientRequest.emit (node:events:518:28)
    at _destroy (node:_http_client:880:13)
    at onSocketNT (node:_http_client:900:5)
    at process.processTicksAndRejections (node:internal/process/task_queues:83:21)
[22.08.2024, 13:55:55] RequestError: Error: connect ECONNREFUSED 169.254.0.51:3274
    at new RequestError (C:\web\chat-bot-telegram\node_modules\request-promise-core\lib\errors.js:14:15)
    at plumbing.callback (C:\web\chat-bot-telegram\node_modules\request-promise-core\lib\plumbing.js:87:29)
    at Request.RP$callback [as _callback] (C:\web\chat-bot-telegram\node_modules\request-promise-core\lib\plumbing.js:46:31)
    at self.callback (C:\web\chat-bot-telegram\node_modules\@cypress\request\request.js:183:22)
    at Request.emit (node:events:518:28)
    at Request.onRequestError (C:\web\chat-bot-telegram\node_modules\@cypress\request\request.js:869:8)
    at ClientRequest.emit (node:events:518:28)
    at _destroy (node:_http_client:880:13)
    at onSocketNT (node:_http_client:900:5)
    at process.processTicksAndRejections (node:internal/process/task_queues:83:21)
[22.08.2024, 13:55:58] RequestError: Error: connect ECONNREFUSED 169.254.0.51:3274
    at new RequestError (C:\web\chat-bot-telegram\node_modules\request-promise-core\lib\errors.js:14:15)
    at plumbing.callback (C:\web\chat-bot-telegram\node_modules\request-promise-core\lib\plumbing.js:87:29)
    at Request.RP$callback [as _callback] (C:\web\chat-bot-telegram\node_modules\request-promise-core\lib\plumbing.js:46:31)
    at self.callback (C:\web\chat-bot-telegram\node_modules\@cypress\request\request.js:183:22)
    at Request.emit (node:events:518:28)
    at Request.onRequestError (C:\web\chat-bot-telegram\node_modules\@cypress\request\request.js:869:8)
    at ClientRequest.emit (node:events:518:28)
    at _destroy (node:_http_client:880:13)
    at onSocketNT (node:_http_client:900:5)
    at process.processTicksAndRejections (node:internal/process/task_queues:83:21)
[22.08.2024, 14:01:52] Error: ETELEGRAM: 400 Bad Request: message to edit not found
    at C:\web\chat-bot-telegram\node_modules\node-telegram-bot-api\src\telegram.js:316:15
    at tryCatcher (C:\web\chat-bot-telegram\node_modules\bluebird\js\release\util.js:16:23)
    at Promise._settlePromiseFromHandler (C:\web\chat-bot-telegram\node_modules\bluebird\js\release\promise.js:547:31)
    at Promise._settlePromise (C:\web\chat-bot-telegram\node_modules\bluebird\js\release\promise.js:604:18)
    at Promise._settlePromise0 (C:\web\chat-bot-telegram\node_modules\bluebird\js\release\promise.js:649:10)
    at Promise._settlePromises (C:\web\chat-bot-telegram\node_modules\bluebird\js\release\promise.js:729:18)
    at _drainQueueStep (C:\web\chat-bot-telegram\node_modules\bluebird\js\release\async.js:93:12)
    at _drainQueue (C:\web\chat-bot-telegram\node_modules\bluebird\js\release\async.js:86:9)
    at Async._drainQueues (C:\web\chat-bot-telegram\node_modules\bluebird\js\release\async.js:102:5)
    at Async.drainQueues [as _onImmediate] (C:\web\chat-bot-telegram\node_modules\bluebird\js\release\async.js:15:14)
    at process.processImmediate (node:internal/timers:478:21)
[22.08.2024, 14:13:59] RequestError: Error: connect ECONNREFUSED 169.254.0.51:3274
    at new RequestError (C:\web\chat-bot-telegram\node_modules\request-promise-core\lib\errors.js:14:15)
    at plumbing.callback (C:\web\chat-bot-telegram\node_modules\request-promise-core\lib\plumbing.js:87:29)
    at Request.RP$callback [as _callback] (C:\web\chat-bot-telegram\node_modules\request-promise-core\lib\plumbing.js:46:31)
    at self.callback (C:\web\chat-bot-telegram\node_modules\@cypress\request\request.js:183:22)
    at Request.emit (node:events:518:28)
    at Request.onRequestError (C:\web\chat-bot-telegram\node_modules\@cypress\request\request.js:869:8)
    at ClientRequest.emit (node:events:518:28)
    at _destroy (node:_http_client:880:13)
    at onSocketNT (node:_http_client:900:5)
    at process.processTicksAndRejections (node:internal/process/task_queues:83:21)
[22.08.2024, 14:14:02] RequestError: Error: connect ECONNREFUSED 169.254.0.51:3274
    at new RequestError (C:\web\chat-bot-telegram\node_modules\request-promise-core\lib\errors.js:14:15)
    at plumbing.callback (C:\web\chat-bot-telegram\node_modules\request-promise-core\lib\plumbing.js:87:29)
    at Request.RP$callback [as _callback] (C:\web\chat-bot-telegram\node_modules\request-promise-core\lib\plumbing.js:46:31)
    at self.callback (C:\web\chat-bot-telegram\node_modules\@cypress\request\request.js:183:22)
    at Request.emit (node:events:518:28)
    at Request.onRequestError (C:\web\chat-bot-telegram\node_modules\@cypress\request\request.js:869:8)
    at ClientRequest.emit (node:events:518:28)
    at _destroy (node:_http_client:880:13)
    at onSocketNT (node:_http_client:900:5)
    at process.processTicksAndRejections (node:internal/process/task_queues:83:21)
[22.08.2024, 14:14:04] RequestError: Error: connect ECONNREFUSED 169.254.0.51:3274
    at new RequestError (C:\web\chat-bot-telegram\node_modules\request-promise-core\lib\errors.js:14:15)
    at plumbing.callback (C:\web\chat-bot-telegram\node_modules\request-promise-core\lib\plumbing.js:87:29)
    at Request.RP$callback [as _callback] (C:\web\chat-bot-telegram\node_modules\request-promise-core\lib\plumbing.js:46:31)
    at self.callback (C:\web\chat-bot-telegram\node_modules\@cypress\request\request.js:183:22)
    at Request.emit (node:events:518:28)
    at Request.onRequestError (C:\web\chat-bot-telegram\node_modules\@cypress\request\request.js:869:8)
    at ClientRequest.emit (node:events:518:28)
    at _destroy (node:_http_client:880:13)
    at onSocketNT (node:_http_client:900:5)
    at process.processTicksAndRejections (node:internal/process/task_queues:83:21)
[22.08.2024, 14:14:06] RequestError: Error: connect ECONNREFUSED 169.254.0.51:3274
    at new RequestError (C:\web\chat-bot-telegram\node_modules\request-promise-core\lib\errors.js:14:15)
    at plumbing.callback (C:\web\chat-bot-telegram\node_modules\request-promise-core\lib\plumbing.js:87:29)
    at Request.RP$callback [as _callback] (C:\web\chat-bot-telegram\node_modules\request-promise-core\lib\plumbing.js:46:31)
    at self.callback (C:\web\chat-bot-telegram\node_modules\@cypress\request\request.js:183:22)
    at Request.emit (node:events:518:28)
    at Request.onRequestError (C:\web\chat-bot-telegram\node_modules\@cypress\request\request.js:869:8)
    at ClientRequest.emit (node:events:518:28)
    at _destroy (node:_http_client:880:13)
    at onSocketNT (node:_http_client:900:5)
    at process.processTicksAndRejections (node:internal/process/task_queues:83:21)
[22.08.2024, 14:14:09] RequestError: Error: connect ECONNREFUSED 169.254.0.51:3274
    at new RequestError (C:\web\chat-bot-telegram\node_modules\request-promise-core\lib\errors.js:14:15)
    at plumbing.callback (C:\web\chat-bot-telegram\node_modules\request-promise-core\lib\plumbing.js:87:29)
    at Request.RP$callback [as _callback] (C:\web\chat-bot-telegram\node_modules\request-promise-core\lib\plumbing.js:46:31)
    at self.callback (C:\web\chat-bot-telegram\node_modules\@cypress\request\request.js:183:22)
    at Request.emit (node:events:518:28)
    at Request.onRequestError (C:\web\chat-bot-telegram\node_modules\@cypress\request\request.js:869:8)
    at ClientRequest.emit (node:events:518:28)
    at _destroy (node:_http_client:880:13)
    at onSocketNT (node:_http_client:900:5)
    at process.processTicksAndRejections (node:internal/process/task_queues:83:21)
[22.08.2024, 14:14:11] RequestError: Error: connect ECONNREFUSED 169.254.0.51:3274
    at new RequestError (C:\web\chat-bot-telegram\node_modules\request-promise-core\lib\errors.js:14:15)
    at plumbing.callback (C:\web\chat-bot-telegram\node_modules\request-promise-core\lib\plumbing.js:87:29)
    at Request.RP$callback [as _callback] (C:\web\chat-bot-telegram\node_modules\request-promise-core\lib\plumbing.js:46:31)
    at self.callback (C:\web\chat-bot-telegram\node_modules\@cypress\request\request.js:183:22)
    at Request.emit (node:events:518:28)
    at Request.onRequestError (C:\web\chat-bot-telegram\node_modules\@cypress\request\request.js:869:8)
    at ClientRequest.emit (node:events:518:28)
    at _destroy (node:_http_client:880:13)
    at onSocketNT (node:_http_client:900:5)
    at process.processTicksAndRejections (node:internal/process/task_queues:83:21)
[22.08.2024, 14:55:56] RequestError: Error: connect ECONNREFUSED 169.254.0.51:3274
    at new RequestError (C:\web\chat-bot-telegram\node_modules\request-promise-core\lib\errors.js:14:15)
    at plumbing.callback (C:\web\chat-bot-telegram\node_modules\request-promise-core\lib\plumbing.js:87:29)
    at Request.RP$callback [as _callback] (C:\web\chat-bot-telegram\node_modules\request-promise-core\lib\plumbing.js:46:31)
    at self.callback (C:\web\chat-bot-telegram\node_modules\@cypress\request\request.js:183:22)
    at Request.emit (node:events:518:28)
    at Request.onRequestError (C:\web\chat-bot-telegram\node_modules\@cypress\request\request.js:869:8)
    at ClientRequest.emit (node:events:518:28)
    at _destroy (node:_http_client:880:13)
    at onSocketNT (node:_http_client:900:5)
    at process.processTicksAndRejections (node:internal/process/task_queues:83:21)
[22.08.2024, 14:55:56] RequestError: Error: connect ECONNREFUSED 169.254.0.51:3274
    at new RequestError (C:\web\chat-bot-telegram\node_modules\request-promise-core\lib\errors.js:14:15)
    at plumbing.callback (C:\web\chat-bot-telegram\node_modules\request-promise-core\lib\plumbing.js:87:29)
    at Request.RP$callback [as _callback] (C:\web\chat-bot-telegram\node_modules\request-promise-core\lib\plumbing.js:46:31)
    at self.callback (C:\web\chat-bot-telegram\node_modules\@cypress\request\request.js:183:22)
    at Request.emit (node:events:518:28)
    at Request.onRequestError (C:\web\chat-bot-telegram\node_modules\@cypress\request\request.js:869:8)
    at ClientRequest.emit (node:events:518:28)
    at _destroy (node:_http_client:880:13)
    at onSocketNT (node:_http_client:900:5)
    at process.processTicksAndRejections (node:internal/process/task_queues:83:21)
[22.08.2024, 14:55:59] RequestError: Error: connect ECONNREFUSED 169.254.0.51:3274
    at new RequestError (C:\web\chat-bot-telegram\node_modules\request-promise-core\lib\errors.js:14:15)
    at plumbing.callback (C:\web\chat-bot-telegram\node_modules\request-promise-core\lib\plumbing.js:87:29)
    at Request.RP$callback [as _callback] (C:\web\chat-bot-telegram\node_modules\request-promise-core\lib\plumbing.js:46:31)
    at self.callback (C:\web\chat-bot-telegram\node_modules\@cypress\request\request.js:183:22)
    at Request.emit (node:events:518:28)
    at Request.onRequestError (C:\web\chat-bot-telegram\node_modules\@cypress\request\request.js:869:8)
    at ClientRequest.emit (node:events:518:28)
    at _destroy (node:_http_client:880:13)
    at onSocketNT (node:_http_client:900:5)
    at process.processTicksAndRejections (node:internal/process/task_queues:83:21)
[22.08.2024, 14:56:01] RequestError: Error: connect ECONNREFUSED 169.254.0.51:3274
    at new RequestError (C:\web\chat-bot-telegram\node_modules\request-promise-core\lib\errors.js:14:15)
    at plumbing.callback (C:\web\chat-bot-telegram\node_modules\request-promise-core\lib\plumbing.js:87:29)
    at Request.RP$callback [as _callback] (C:\web\chat-bot-telegram\node_modules\request-promise-core\lib\plumbing.js:46:31)
    at self.callback (C:\web\chat-bot-telegram\node_modules\@cypress\request\request.js:183:22)
    at Request.emit (node:events:518:28)
    at Request.onRequestError (C:\web\chat-bot-telegram\node_modules\@cypress\request\request.js:869:8)
    at ClientRequest.emit (node:events:518:28)
    at _destroy (node:_http_client:880:13)
    at onSocketNT (node:_http_client:900:5)
    at process.processTicksAndRejections (node:internal/process/task_queues:83:21)
[22.08.2024, 14:56:04] RequestError: Error: connect ECONNREFUSED 169.254.0.51:3274
    at new RequestError (C:\web\chat-bot-telegram\node_modules\request-promise-core\lib\errors.js:14:15)
    at plumbing.callback (C:\web\chat-bot-telegram\node_modules\request-promise-core\lib\plumbing.js:87:29)
    at Request.RP$callback [as _callback] (C:\web\chat-bot-telegram\node_modules\request-promise-core\lib\plumbing.js:46:31)
    at self.callback (C:\web\chat-bot-telegram\node_modules\@cypress\request\request.js:183:22)
    at Request.emit (node:events:518:28)
    at Request.onRequestError (C:\web\chat-bot-telegram\node_modules\@cypress\request\request.js:869:8)
    at ClientRequest.emit (node:events:518:28)
    at _destroy (node:_http_client:880:13)
    at onSocketNT (node:_http_client:900:5)
    at process.processTicksAndRejections (node:internal/process/task_queues:83:21)
[22.08.2024, 14:57:18] RequestError: Error: connect ECONNREFUSED 169.254.0.51:3274
    at new RequestError (C:\web\chat-bot-telegram\node_modules\request-promise-core\lib\errors.js:14:15)
    at plumbing.callback (C:\web\chat-bot-telegram\node_modules\request-promise-core\lib\plumbing.js:87:29)
    at Request.RP$callback [as _callback] (C:\web\chat-bot-telegram\node_modules\request-promise-core\lib\plumbing.js:46:31)
    at self.callback (C:\web\chat-bot-telegram\node_modules\@cypress\request\request.js:183:22)
    at Request.emit (node:events:518:28)
    at Request.onRequestError (C:\web\chat-bot-telegram\node_modules\@cypress\request\request.js:869:8)
    at ClientRequest.emit (node:events:518:28)
    at _destroy (node:_http_client:880:13)
    at onSocketNT (node:_http_client:900:5)
    at process.processTicksAndRejections (node:internal/process/task_queues:83:21)
[22.08.2024, 15:19:09] ReferenceError: mongoose is not defined
    at convertOldTimestamps (file:///C:/web/chat-bot-telegram/src/routes/updateValues.js:59:5)
    at file:///C:/web/chat-bot-telegram/src/routes/updateValues.js:63:1
    at ModuleJob.run (node:internal/modules/esm/module_job:222:25)
    at async ModuleLoader.import (node:internal/modules/esm/loader:323:24)
    at async loadESM (node:internal/process/esm_loader:28:7)
    at async handleMainPromise (node:internal/modules/run_main:113:12)
[22.08.2024, 15:24:01] Error: ETELEGRAM: 400 Bad Request: query is too old and response timeout expired or query ID is invalid
    at C:\web\chat-bot-telegram\node_modules\node-telegram-bot-api\src\telegram.js:316:15
    at tryCatcher (C:\web\chat-bot-telegram\node_modules\bluebird\js\release\util.js:16:23)
    at Promise._settlePromiseFromHandler (C:\web\chat-bot-telegram\node_modules\bluebird\js\release\promise.js:547:31)
    at Promise._settlePromise (C:\web\chat-bot-telegram\node_modules\bluebird\js\release\promise.js:604:18)
    at Promise._settlePromise0 (C:\web\chat-bot-telegram\node_modules\bluebird\js\release\promise.js:649:10)
    at Promise._settlePromises (C:\web\chat-bot-telegram\node_modules\bluebird\js\release\promise.js:729:18)
    at _drainQueueStep (C:\web\chat-bot-telegram\node_modules\bluebird\js\release\async.js:93:12)
    at _drainQueue (C:\web\chat-bot-telegram\node_modules\bluebird\js\release\async.js:86:9)
    at Async._drainQueues (C:\web\chat-bot-telegram\node_modules\bluebird\js\release\async.js:102:5)
    at Async.drainQueues [as _onImmediate] (C:\web\chat-bot-telegram\node_modules\bluebird\js\release\async.js:15:14)
    at process.processImmediate (node:internal/timers:478:21)
[22.08.2024, 15:24:01] Error: ETELEGRAM: 400 Bad Request: query is too old and response timeout expired or query ID is invalid
    at C:\web\chat-bot-telegram\node_modules\node-telegram-bot-api\src\telegram.js:316:15
    at tryCatcher (C:\web\chat-bot-telegram\node_modules\bluebird\js\release\util.js:16:23)
    at Promise._settlePromiseFromHandler (C:\web\chat-bot-telegram\node_modules\bluebird\js\release\promise.js:547:31)
    at Promise._settlePromise (C:\web\chat-bot-telegram\node_modules\bluebird\js\release\promise.js:604:18)
    at Promise._settlePromise0 (C:\web\chat-bot-telegram\node_modules\bluebird\js\release\promise.js:649:10)
    at Promise._settlePromises (C:\web\chat-bot-telegram\node_modules\bluebird\js\release\promise.js:729:18)
    at _drainQueueStep (C:\web\chat-bot-telegram\node_modules\bluebird\js\release\async.js:93:12)
    at _drainQueue (C:\web\chat-bot-telegram\node_modules\bluebird\js\release\async.js:86:9)
    at Async._drainQueues (C:\web\chat-bot-telegram\node_modules\bluebird\js\release\async.js:102:5)
    at Async.drainQueues [as _onImmediate] (C:\web\chat-bot-telegram\node_modules\bluebird\js\release\async.js:15:14)
    at process.processImmediate (node:internal/timers:478:21)
[22.08.2024, 15:42:33] Error: ETELEGRAM: 400 Bad Request: query is too old and response timeout expired or query ID is invalid
    at C:\web\chat-bot-telegram\node_modules\node-telegram-bot-api\src\telegram.js:316:15
    at tryCatcher (C:\web\chat-bot-telegram\node_modules\bluebird\js\release\util.js:16:23)
    at Promise._settlePromiseFromHandler (C:\web\chat-bot-telegram\node_modules\bluebird\js\release\promise.js:547:31)
    at Promise._settlePromise (C:\web\chat-bot-telegram\node_modules\bluebird\js\release\promise.js:604:18)
    at Promise._settlePromise0 (C:\web\chat-bot-telegram\node_modules\bluebird\js\release\promise.js:649:10)
    at Promise._settlePromises (C:\web\chat-bot-telegram\node_modules\bluebird\js\release\promise.js:729:18)
    at _drainQueueStep (C:\web\chat-bot-telegram\node_modules\bluebird\js\release\async.js:93:12)
    at _drainQueue (C:\web\chat-bot-telegram\node_modules\bluebird\js\release\async.js:86:9)
    at Async._drainQueues (C:\web\chat-bot-telegram\node_modules\bluebird\js\release\async.js:102:5)
    at Async.drainQueues [as _onImmediate] (C:\web\chat-bot-telegram\node_modules\bluebird\js\release\async.js:15:14)
    at process.processImmediate (node:internal/timers:478:21)
[22.08.2024, 15:48:35] RequestError: Error: connect ETIMEDOUT 169.254.0.51:3274
    at new RequestError (C:\web\chat-bot-telegram\node_modules\request-promise-core\lib\errors.js:14:15)
    at plumbing.callback (C:\web\chat-bot-telegram\node_modules\request-promise-core\lib\plumbing.js:87:29)
    at Request.RP$callback [as _callback] (C:\web\chat-bot-telegram\node_modules\request-promise-core\lib\plumbing.js:46:31)
    at self.callback (C:\web\chat-bot-telegram\node_modules\@cypress\request\request.js:183:22)
    at Request.emit (node:events:518:28)
    at Request.onRequestError (C:\web\chat-bot-telegram\node_modules\@cypress\request\request.js:869:8)
    at ClientRequest.emit (node:events:518:28)
    at _destroy (node:_http_client:880:13)
    at onSocketNT (node:_http_client:900:5)
    at process.processTicksAndRejections (node:internal/process/task_queues:83:21)
[22.08.2024, 16:07:16] RequestError: Error: connect ECONNREFUSED 169.254.0.51:3274
    at new RequestError (C:\web\chat-bot-telegram\node_modules\request-promise-core\lib\errors.js:14:15)
    at plumbing.callback (C:\web\chat-bot-telegram\node_modules\request-promise-core\lib\plumbing.js:87:29)
    at Request.RP$callback [as _callback] (C:\web\chat-bot-telegram\node_modules\request-promise-core\lib\plumbing.js:46:31)
    at self.callback (C:\web\chat-bot-telegram\node_modules\@cypress\request\request.js:183:22)
    at Request.emit (node:events:518:28)
    at Request.onRequestError (C:\web\chat-bot-telegram\node_modules\@cypress\request\request.js:869:8)
    at ClientRequest.emit (node:events:518:28)
    at _destroy (node:_http_client:880:13)
    at onSocketNT (node:_http_client:900:5)
    at process.processTicksAndRejections (node:internal/process/task_queues:83:21)
[22.08.2024, 16:07:16] RequestError: Error: connect ECONNREFUSED 169.254.0.51:3274
    at new RequestError (C:\web\chat-bot-telegram\node_modules\request-promise-core\lib\errors.js:14:15)
    at plumbing.callback (C:\web\chat-bot-telegram\node_modules\request-promise-core\lib\plumbing.js:87:29)
    at Request.RP$callback [as _callback] (C:\web\chat-bot-telegram\node_modules\request-promise-core\lib\plumbing.js:46:31)
    at self.callback (C:\web\chat-bot-telegram\node_modules\@cypress\request\request.js:183:22)
    at Request.emit (node:events:518:28)
    at Request.onRequestError (C:\web\chat-bot-telegram\node_modules\@cypress\request\request.js:869:8)
    at ClientRequest.emit (node:events:518:28)
    at _destroy (node:_http_client:880:13)
    at onSocketNT (node:_http_client:900:5)
    at process.processTicksAndRejections (node:internal/process/task_queues:83:21)
[22.08.2024, 16:07:19] RequestError: Error: connect ECONNREFUSED 169.254.0.51:3274
    at new RequestError (C:\web\chat-bot-telegram\node_modules\request-promise-core\lib\errors.js:14:15)
    at plumbing.callback (C:\web\chat-bot-telegram\node_modules\request-promise-core\lib\plumbing.js:87:29)
    at Request.RP$callback [as _callback] (C:\web\chat-bot-telegram\node_modules\request-promise-core\lib\plumbing.js:46:31)
    at self.callback (C:\web\chat-bot-telegram\node_modules\@cypress\request\request.js:183:22)
    at Request.emit (node:events:518:28)
    at Request.onRequestError (C:\web\chat-bot-telegram\node_modules\@cypress\request\request.js:869:8)
    at ClientRequest.emit (node:events:518:28)
    at _destroy (node:_http_client:880:13)
    at onSocketNT (node:_http_client:900:5)
    at process.processTicksAndRejections (node:internal/process/task_queues:83:21)
[22.08.2024, 16:07:21] RequestError: Error: connect ECONNREFUSED 169.254.0.51:3274
    at new RequestError (C:\web\chat-bot-telegram\node_modules\request-promise-core\lib\errors.js:14:15)
    at plumbing.callback (C:\web\chat-bot-telegram\node_modules\request-promise-core\lib\plumbing.js:87:29)
    at Request.RP$callback [as _callback] (C:\web\chat-bot-telegram\node_modules\request-promise-core\lib\plumbing.js:46:31)
    at self.callback (C:\web\chat-bot-telegram\node_modules\@cypress\request\request.js:183:22)
    at Request.emit (node:events:518:28)
    at Request.onRequestError (C:\web\chat-bot-telegram\node_modules\@cypress\request\request.js:869:8)
    at ClientRequest.emit (node:events:518:28)
    at _destroy (node:_http_client:880:13)
    at onSocketNT (node:_http_client:900:5)
    at process.processTicksAndRejections (node:internal/process/task_queues:83:21)
[22.08.2024, 16:07:24] RequestError: Error: connect ECONNREFUSED 169.254.0.51:3274
    at new RequestError (C:\web\chat-bot-telegram\node_modules\request-promise-core\lib\errors.js:14:15)
    at plumbing.callback (C:\web\chat-bot-telegram\node_modules\request-promise-core\lib\plumbing.js:87:29)
    at Request.RP$callback [as _callback] (C:\web\chat-bot-telegram\node_modules\request-promise-core\lib\plumbing.js:46:31)
    at self.callback (C:\web\chat-bot-telegram\node_modules\@cypress\request\request.js:183:22)
    at Request.emit (node:events:518:28)
    at Request.onRequestError (C:\web\chat-bot-telegram\node_modules\@cypress\request\request.js:869:8)
    at ClientRequest.emit (node:events:518:28)
    at _destroy (node:_http_client:880:13)
    at onSocketNT (node:_http_client:900:5)
    at process.processTicksAndRejections (node:internal/process/task_queues:83:21)
[22.08.2024, 16:07:26] RequestError: Error: connect ECONNREFUSED 169.254.0.51:3274
    at new RequestError (C:\web\chat-bot-telegram\node_modules\request-promise-core\lib\errors.js:14:15)
    at plumbing.callback (C:\web\chat-bot-telegram\node_modules\request-promise-core\lib\plumbing.js:87:29)
    at Request.RP$callback [as _callback] (C:\web\chat-bot-telegram\node_modules\request-promise-core\lib\plumbing.js:46:31)
    at self.callback (C:\web\chat-bot-telegram\node_modules\@cypress\request\request.js:183:22)
    at Request.emit (node:events:518:28)
    at Request.onRequestError (C:\web\chat-bot-telegram\node_modules\@cypress\request\request.js:869:8)
    at ClientRequest.emit (node:events:518:28)
    at _destroy (node:_http_client:880:13)
    at onSocketNT (node:_http_client:900:5)
    at process.processTicksAndRejections (node:internal/process/task_queues:83:21)
[22.08.2024, 16:07:49] RequestError: Error: connect ECONNREFUSED 169.254.0.51:3274
    at new RequestError (C:\web\chat-bot-telegram\node_modules\request-promise-core\lib\errors.js:14:15)
    at plumbing.callback (C:\web\chat-bot-telegram\node_modules\request-promise-core\lib\plumbing.js:87:29)
    at Request.RP$callback [as _callback] (C:\web\chat-bot-telegram\node_modules\request-promise-core\lib\plumbing.js:46:31)
    at self.callback (C:\web\chat-bot-telegram\node_modules\@cypress\request\request.js:183:22)
    at Request.emit (node:events:518:28)
    at Request.onRequestError (C:\web\chat-bot-telegram\node_modules\@cypress\request\request.js:869:8)
    at ClientRequest.emit (node:events:518:28)
    at _destroy (node:_http_client:880:13)
    at onSocketNT (node:_http_client:900:5)
    at process.processTicksAndRejections (node:internal/process/task_queues:83:21)
[22.08.2024, 16:07:52] RequestError: Error: connect ECONNREFUSED 169.254.0.51:3274
    at new RequestError (C:\web\chat-bot-telegram\node_modules\request-promise-core\lib\errors.js:14:15)
    at plumbing.callback (C:\web\chat-bot-telegram\node_modules\request-promise-core\lib\plumbing.js:87:29)
    at Request.RP$callback [as _callback] (C:\web\chat-bot-telegram\node_modules\request-promise-core\lib\plumbing.js:46:31)
    at self.callback (C:\web\chat-bot-telegram\node_modules\@cypress\request\request.js:183:22)
    at Request.emit (node:events:518:28)
    at Request.onRequestError (C:\web\chat-bot-telegram\node_modules\@cypress\request\request.js:869:8)
    at ClientRequest.emit (node:events:518:28)
    at _destroy (node:_http_client:880:13)
    at onSocketNT (node:_http_client:900:5)
    at process.processTicksAndRejections (node:internal/process/task_queues:83:21)
[22.08.2024, 16:07:54] RequestError: Error: connect ECONNREFUSED 169.254.0.51:3274
    at new RequestError (C:\web\chat-bot-telegram\node_modules\request-promise-core\lib\errors.js:14:15)
    at plumbing.callback (C:\web\chat-bot-telegram\node_modules\request-promise-core\lib\plumbing.js:87:29)
    at Request.RP$callback [as _callback] (C:\web\chat-bot-telegram\node_modules\request-promise-core\lib\plumbing.js:46:31)
    at self.callback (C:\web\chat-bot-telegram\node_modules\@cypress\request\request.js:183:22)
    at Request.emit (node:events:518:28)
    at Request.onRequestError (C:\web\chat-bot-telegram\node_modules\@cypress\request\request.js:869:8)
    at ClientRequest.emit (node:events:518:28)
    at _destroy (node:_http_client:880:13)
    at onSocketNT (node:_http_client:900:5)
    at process.processTicksAndRejections (node:internal/process/task_queues:83:21)
[22.08.2024, 16:42:26] RequestError: Error: connect ECONNREFUSED 169.254.0.51:3274
    at new RequestError (C:\web\chat-bot-telegram\node_modules\request-promise-core\lib\errors.js:14:15)
    at plumbing.callback (C:\web\chat-bot-telegram\node_modules\request-promise-core\lib\plumbing.js:87:29)
    at Request.RP$callback [as _callback] (C:\web\chat-bot-telegram\node_modules\request-promise-core\lib\plumbing.js:46:31)
    at self.callback (C:\web\chat-bot-telegram\node_modules\@cypress\request\request.js:183:22)
    at Request.emit (node:events:518:28)
    at Request.onRequestError (C:\web\chat-bot-telegram\node_modules\@cypress\request\request.js:869:8)
    at ClientRequest.emit (node:events:518:28)
    at _destroy (node:_http_client:880:13)
    at onSocketNT (node:_http_client:900:5)
    at process.processTicksAndRejections (node:internal/process/task_queues:83:21)
[22.08.2024, 16:42:28] RequestError: Error: connect ECONNREFUSED 169.254.0.51:3274
    at new RequestError (C:\web\chat-bot-telegram\node_modules\request-promise-core\lib\errors.js:14:15)
    at plumbing.callback (C:\web\chat-bot-telegram\node_modules\request-promise-core\lib\plumbing.js:87:29)
    at Request.RP$callback [as _callback] (C:\web\chat-bot-telegram\node_modules\request-promise-core\lib\plumbing.js:46:31)
    at self.callback (C:\web\chat-bot-telegram\node_modules\@cypress\request\request.js:183:22)
    at Request.emit (node:events:518:28)
    at Request.onRequestError (C:\web\chat-bot-telegram\node_modules\@cypress\request\request.js:869:8)
    at ClientRequest.emit (node:events:518:28)
    at _destroy (node:_http_client:880:13)
    at onSocketNT (node:_http_client:900:5)
    at process.processTicksAndRejections (node:internal/process/task_queues:83:21)
[22.08.2024, 16:42:30] RequestError: Error: connect ECONNREFUSED 169.254.0.51:3274
    at new RequestError (C:\web\chat-bot-telegram\node_modules\request-promise-core\lib\errors.js:14:15)
    at plumbing.callback (C:\web\chat-bot-telegram\node_modules\request-promise-core\lib\plumbing.js:87:29)
    at Request.RP$callback [as _callback] (C:\web\chat-bot-telegram\node_modules\request-promise-core\lib\plumbing.js:46:31)
    at self.callback (C:\web\chat-bot-telegram\node_modules\@cypress\request\request.js:183:22)
    at Request.emit (node:events:518:28)
    at Request.onRequestError (C:\web\chat-bot-telegram\node_modules\@cypress\request\request.js:869:8)
    at ClientRequest.emit (node:events:518:28)
    at _destroy (node:_http_client:880:13)
    at onSocketNT (node:_http_client:900:5)
    at process.processTicksAndRejections (node:internal/process/task_queues:83:21)
[22.08.2024, 16:42:33] RequestError: Error: connect ECONNREFUSED 169.254.0.51:3274
    at new RequestError (C:\web\chat-bot-telegram\node_modules\request-promise-core\lib\errors.js:14:15)
    at plumbing.callback (C:\web\chat-bot-telegram\node_modules\request-promise-core\lib\plumbing.js:87:29)
    at Request.RP$callback [as _callback] (C:\web\chat-bot-telegram\node_modules\request-promise-core\lib\plumbing.js:46:31)
    at self.callback (C:\web\chat-bot-telegram\node_modules\@cypress\request\request.js:183:22)
    at Request.emit (node:events:518:28)
    at Request.onRequestError (C:\web\chat-bot-telegram\node_modules\@cypress\request\request.js:869:8)
    at ClientRequest.emit (node:events:518:28)
    at _destroy (node:_http_client:880:13)
    at onSocketNT (node:_http_client:900:5)
    at process.processTicksAndRejections (node:internal/process/task_queues:83:21)
[22.08.2024, 16:42:35] RequestError: Error: connect ECONNREFUSED 169.254.0.51:3274
    at new RequestError (C:\web\chat-bot-telegram\node_modules\request-promise-core\lib\errors.js:14:15)
    at plumbing.callback (C:\web\chat-bot-telegram\node_modules\request-promise-core\lib\plumbing.js:87:29)
    at Request.RP$callback [as _callback] (C:\web\chat-bot-telegram\node_modules\request-promise-core\lib\plumbing.js:46:31)
    at self.callback (C:\web\chat-bot-telegram\node_modules\@cypress\request\request.js:183:22)
    at Request.emit (node:events:518:28)
    at Request.onRequestError (C:\web\chat-bot-telegram\node_modules\@cypress\request\request.js:869:8)
    at ClientRequest.emit (node:events:518:28)
    at _destroy (node:_http_client:880:13)
    at onSocketNT (node:_http_client:900:5)
    at process.processTicksAndRejections (node:internal/process/task_queues:83:21)
[22.08.2024, 16:42:49] RequestError: Error: connect ECONNREFUSED 169.254.0.51:3274
    at new RequestError (C:\web\chat-bot-telegram\node_modules\request-promise-core\lib\errors.js:14:15)
    at plumbing.callback (C:\web\chat-bot-telegram\node_modules\request-promise-core\lib\plumbing.js:87:29)
    at Request.RP$callback [as _callback] (C:\web\chat-bot-telegram\node_modules\request-promise-core\lib\plumbing.js:46:31)
    at self.callback (C:\web\chat-bot-telegram\node_modules\@cypress\request\request.js:183:22)
    at Request.emit (node:events:518:28)
    at Request.onRequestError (C:\web\chat-bot-telegram\node_modules\@cypress\request\request.js:869:8)
    at ClientRequest.emit (node:events:518:28)
    at _destroy (node:_http_client:880:13)
    at onSocketNT (node:_http_client:900:5)
    at process.processTicksAndRejections (node:internal/process/task_queues:83:21)
[22.08.2024, 16:42:51] RequestError: Error: connect ECONNREFUSED 169.254.0.51:3274
    at new RequestError (C:\web\chat-bot-telegram\node_modules\request-promise-core\lib\errors.js:14:15)
    at plumbing.callback (C:\web\chat-bot-telegram\node_modules\request-promise-core\lib\plumbing.js:87:29)
    at Request.RP$callback [as _callback] (C:\web\chat-bot-telegram\node_modules\request-promise-core\lib\plumbing.js:46:31)
    at self.callback (C:\web\chat-bot-telegram\node_modules\@cypress\request\request.js:183:22)
    at Request.emit (node:events:518:28)
    at Request.onRequestError (C:\web\chat-bot-telegram\node_modules\@cypress\request\request.js:869:8)
    at ClientRequest.emit (node:events:518:28)
    at _destroy (node:_http_client:880:13)
    at onSocketNT (node:_http_client:900:5)
    at process.processTicksAndRejections (node:internal/process/task_queues:83:21)
[22.08.2024, 16:42:54] RequestError: Error: connect ECONNREFUSED 169.254.0.51:3274
    at new RequestError (C:\web\chat-bot-telegram\node_modules\request-promise-core\lib\errors.js:14:15)
    at plumbing.callback (C:\web\chat-bot-telegram\node_modules\request-promise-core\lib\plumbing.js:87:29)
    at Request.RP$callback [as _callback] (C:\web\chat-bot-telegram\node_modules\request-promise-core\lib\plumbing.js:46:31)
    at self.callback (C:\web\chat-bot-telegram\node_modules\@cypress\request\request.js:183:22)
    at Request.emit (node:events:518:28)
    at Request.onRequestError (C:\web\chat-bot-telegram\node_modules\@cypress\request\request.js:869:8)
    at ClientRequest.emit (node:events:518:28)
    at _destroy (node:_http_client:880:13)
    at onSocketNT (node:_http_client:900:5)
    at process.processTicksAndRejections (node:internal/process/task_queues:83:21)
[22.08.2024, 16:42:56] RequestError: Error: connect ECONNREFUSED 169.254.0.51:3274
    at new RequestError (C:\web\chat-bot-telegram\node_modules\request-promise-core\lib\errors.js:14:15)
    at plumbing.callback (C:\web\chat-bot-telegram\node_modules\request-promise-core\lib\plumbing.js:87:29)
    at Request.RP$callback [as _callback] (C:\web\chat-bot-telegram\node_modules\request-promise-core\lib\plumbing.js:46:31)
    at self.callback (C:\web\chat-bot-telegram\node_modules\@cypress\request\request.js:183:22)
    at Request.emit (node:events:518:28)
    at Request.onRequestError (C:\web\chat-bot-telegram\node_modules\@cypress\request\request.js:869:8)
    at ClientRequest.emit (node:events:518:28)
    at _destroy (node:_http_client:880:13)
    at onSocketNT (node:_http_client:900:5)
    at process.processTicksAndRejections (node:internal/process/task_queues:83:21)
[22.08.2024, 16:49:46] RequestError: Error: connect ECONNREFUSED 169.254.0.51:3274
    at new RequestError (C:\web\chat-bot-telegram\node_modules\request-promise-core\lib\errors.js:14:15)
    at plumbing.callback (C:\web\chat-bot-telegram\node_modules\request-promise-core\lib\plumbing.js:87:29)
    at Request.RP$callback [as _callback] (C:\web\chat-bot-telegram\node_modules\request-promise-core\lib\plumbing.js:46:31)
    at self.callback (C:\web\chat-bot-telegram\node_modules\@cypress\request\request.js:183:22)
    at Request.emit (node:events:518:28)
    at Request.onRequestError (C:\web\chat-bot-telegram\node_modules\@cypress\request\request.js:869:8)
    at ClientRequest.emit (node:events:518:28)
    at _destroy (node:_http_client:880:13)
    at onSocketNT (node:_http_client:900:5)
    at process.processTicksAndRejections (node:internal/process/task_queues:83:21)
[22.08.2024, 16:49:48] RequestError: Error: connect ECONNREFUSED 169.254.0.51:3274
    at new RequestError (C:\web\chat-bot-telegram\node_modules\request-promise-core\lib\errors.js:14:15)
    at plumbing.callback (C:\web\chat-bot-telegram\node_modules\request-promise-core\lib\plumbing.js:87:29)
    at Request.RP$callback [as _callback] (C:\web\chat-bot-telegram\node_modules\request-promise-core\lib\plumbing.js:46:31)
    at self.callback (C:\web\chat-bot-telegram\node_modules\@cypress\request\request.js:183:22)
    at Request.emit (node:events:518:28)
    at Request.onRequestError (C:\web\chat-bot-telegram\node_modules\@cypress\request\request.js:869:8)
    at ClientRequest.emit (node:events:518:28)
    at _destroy (node:_http_client:880:13)
    at onSocketNT (node:_http_client:900:5)
    at process.processTicksAndRejections (node:internal/process/task_queues:83:21)
[22.08.2024, 16:49:51] RequestError: Error: connect ECONNREFUSED 169.254.0.51:3274
    at new RequestError (C:\web\chat-bot-telegram\node_modules\request-promise-core\lib\errors.js:14:15)
    at plumbing.callback (C:\web\chat-bot-telegram\node_modules\request-promise-core\lib\plumbing.js:87:29)
    at Request.RP$callback [as _callback] (C:\web\chat-bot-telegram\node_modules\request-promise-core\lib\plumbing.js:46:31)
    at self.callback (C:\web\chat-bot-telegram\node_modules\@cypress\request\request.js:183:22)
    at Request.emit (node:events:518:28)
    at Request.onRequestError (C:\web\chat-bot-telegram\node_modules\@cypress\request\request.js:869:8)
    at ClientRequest.emit (node:events:518:28)
    at _destroy (node:_http_client:880:13)
    at onSocketNT (node:_http_client:900:5)
    at process.processTicksAndRejections (node:internal/process/task_queues:83:21)
[22.08.2024, 16:49:53] RequestError: Error: connect ECONNREFUSED 169.254.0.51:3274
    at new RequestError (C:\web\chat-bot-telegram\node_modules\request-promise-core\lib\errors.js:14:15)
    at plumbing.callback (C:\web\chat-bot-telegram\node_modules\request-promise-core\lib\plumbing.js:87:29)
    at Request.RP$callback [as _callback] (C:\web\chat-bot-telegram\node_modules\request-promise-core\lib\plumbing.js:46:31)
    at self.callback (C:\web\chat-bot-telegram\node_modules\@cypress\request\request.js:183:22)
    at Request.emit (node:events:518:28)
    at Request.onRequestError (C:\web\chat-bot-telegram\node_modules\@cypress\request\request.js:869:8)
    at ClientRequest.emit (node:events:518:28)
    at _destroy (node:_http_client:880:13)
    at onSocketNT (node:_http_client:900:5)
    at process.processTicksAndRejections (node:internal/process/task_queues:83:21)
[22.08.2024, 16:49:55] RequestError: Error: connect ECONNREFUSED 169.254.0.51:3274
    at new RequestError (C:\web\chat-bot-telegram\node_modules\request-promise-core\lib\errors.js:14:15)
    at plumbing.callback (C:\web\chat-bot-telegram\node_modules\request-promise-core\lib\plumbing.js:87:29)
    at Request.RP$callback [as _callback] (C:\web\chat-bot-telegram\node_modules\request-promise-core\lib\plumbing.js:46:31)
    at self.callback (C:\web\chat-bot-telegram\node_modules\@cypress\request\request.js:183:22)
    at Request.emit (node:events:518:28)
    at Request.onRequestError (C:\web\chat-bot-telegram\node_modules\@cypress\request\request.js:869:8)
    at ClientRequest.emit (node:events:518:28)
    at _destroy (node:_http_client:880:13)
    at onSocketNT (node:_http_client:900:5)
    at process.processTicksAndRejections (node:internal/process/task_queues:83:21)
[22.08.2024, 16:49:58] RequestError: Error: connect ECONNREFUSED 169.254.0.51:3274
    at new RequestError (C:\web\chat-bot-telegram\node_modules\request-promise-core\lib\errors.js:14:15)
    at plumbing.callback (C:\web\chat-bot-telegram\node_modules\request-promise-core\lib\plumbing.js:87:29)
    at Request.RP$callback [as _callback] (C:\web\chat-bot-telegram\node_modules\request-promise-core\lib\plumbing.js:46:31)
    at self.callback (C:\web\chat-bot-telegram\node_modules\@cypress\request\request.js:183:22)
    at Request.emit (node:events:518:28)
    at Request.onRequestError (C:\web\chat-bot-telegram\node_modules\@cypress\request\request.js:869:8)
    at ClientRequest.emit (node:events:518:28)
    at _destroy (node:_http_client:880:13)
    at onSocketNT (node:_http_client:900:5)
    at process.processTicksAndRejections (node:internal/process/task_queues:83:21)
[22.08.2024, 16:50:00] RequestError: Error: connect ECONNREFUSED 169.254.0.51:3274
    at new RequestError (C:\web\chat-bot-telegram\node_modules\request-promise-core\lib\errors.js:14:15)
    at plumbing.callback (C:\web\chat-bot-telegram\node_modules\request-promise-core\lib\plumbing.js:87:29)
    at Request.RP$callback [as _callback] (C:\web\chat-bot-telegram\node_modules\request-promise-core\lib\plumbing.js:46:31)
    at self.callback (C:\web\chat-bot-telegram\node_modules\@cypress\request\request.js:183:22)
    at Request.emit (node:events:518:28)
    at Request.onRequestError (C:\web\chat-bot-telegram\node_modules\@cypress\request\request.js:869:8)
    at ClientRequest.emit (node:events:518:28)
    at _destroy (node:_http_client:880:13)
    at onSocketNT (node:_http_client:900:5)
    at process.processTicksAndRejections (node:internal/process/task_queues:83:21)
[22.08.2024, 17:07:58] RequestError: Error: connect ECONNREFUSED 169.254.0.51:3274
    at new RequestError (C:\web\chat-bot-telegram\node_modules\request-promise-core\lib\errors.js:14:15)
    at plumbing.callback (C:\web\chat-bot-telegram\node_modules\request-promise-core\lib\plumbing.js:87:29)
    at Request.RP$callback [as _callback] (C:\web\chat-bot-telegram\node_modules\request-promise-core\lib\plumbing.js:46:31)
    at self.callback (C:\web\chat-bot-telegram\node_modules\@cypress\request\request.js:183:22)
    at Request.emit (node:events:518:28)
    at Request.onRequestError (C:\web\chat-bot-telegram\node_modules\@cypress\request\request.js:869:8)
    at ClientRequest.emit (node:events:518:28)
    at _destroy (node:_http_client:880:13)
    at onSocketNT (node:_http_client:900:5)
    at process.processTicksAndRejections (node:internal/process/task_queues:83:21)
[22.08.2024, 18:36:38] RequestError: Error: Proxy connection ended before receiving CONNECT response
    at new RequestError (C:\web\chat-bot-telegram\node_modules\request-promise-core\lib\errors.js:14:15)
    at plumbing.callback (C:\web\chat-bot-telegram\node_modules\request-promise-core\lib\plumbing.js:87:29)
    at Request.RP$callback [as _callback] (C:\web\chat-bot-telegram\node_modules\request-promise-core\lib\plumbing.js:46:31)
    at self.callback (C:\web\chat-bot-telegram\node_modules\@cypress\request\request.js:183:22)
    at Request.emit (node:events:518:28)
    at Request.onRequestError (C:\web\chat-bot-telegram\node_modules\@cypress\request\request.js:869:8)
    at ClientRequest.emit (node:events:518:28)
    at _destroy (node:_http_client:880:13)
    at onSocketNT (node:_http_client:900:5)
    at process.processTicksAndRejections (node:internal/process/task_queues:83:21)
[22.08.2024, 23:56:16] RequestError: Error: Proxy connection ended before receiving CONNECT response
    at new RequestError (C:\web\chat-bot-telegram\node_modules\request-promise-core\lib\errors.js:14:15)
    at plumbing.callback (C:\web\chat-bot-telegram\node_modules\request-promise-core\lib\plumbing.js:87:29)
    at Request.RP$callback [as _callback] (C:\web\chat-bot-telegram\node_modules\request-promise-core\lib\plumbing.js:46:31)
    at self.callback (C:\web\chat-bot-telegram\node_modules\@cypress\request\request.js:183:22)
    at Request.emit (node:events:518:28)
    at Request.onRequestError (C:\web\chat-bot-telegram\node_modules\@cypress\request\request.js:869:8)
    at ClientRequest.emit (node:events:518:28)
    at _destroy (node:_http_client:880:13)
    at onSocketNT (node:_http_client:900:5)
    at process.processTicksAndRejections (node:internal/process/task_queues:83:21)
[23.08.2024, 06:13:40] Error: ETELEGRAM: 502 Bad Gateway
    at C:\web\chat-bot-telegram\node_modules\node-telegram-bot-api\src\telegram.js:316:15
    at tryCatcher (C:\web\chat-bot-telegram\node_modules\bluebird\js\release\util.js:16:23)
    at Promise._settlePromiseFromHandler (C:\web\chat-bot-telegram\node_modules\bluebird\js\release\promise.js:547:31)
    at Promise._settlePromise (C:\web\chat-bot-telegram\node_modules\bluebird\js\release\promise.js:604:18)
    at Promise._settlePromise0 (C:\web\chat-bot-telegram\node_modules\bluebird\js\release\promise.js:649:10)
    at Promise._settlePromises (C:\web\chat-bot-telegram\node_modules\bluebird\js\release\promise.js:729:18)
    at _drainQueueStep (C:\web\chat-bot-telegram\node_modules\bluebird\js\release\async.js:93:12)
    at _drainQueue (C:\web\chat-bot-telegram\node_modules\bluebird\js\release\async.js:86:9)
    at Async._drainQueues (C:\web\chat-bot-telegram\node_modules\bluebird\js\release\async.js:102:5)
    at Async.drainQueues [as _onImmediate] (C:\web\chat-bot-telegram\node_modules\bluebird\js\release\async.js:15:14)
    at process.processImmediate (node:internal/timers:478:21)
[23.08.2024, 06:13:40] Error: ETELEGRAM: 502 Bad Gateway
    at C:\web\chat-bot-telegram\node_modules\node-telegram-bot-api\src\telegram.js:316:15
    at tryCatcher (C:\web\chat-bot-telegram\node_modules\bluebird\js\release\util.js:16:23)
    at Promise._settlePromiseFromHandler (C:\web\chat-bot-telegram\node_modules\bluebird\js\release\promise.js:547:31)
    at Promise._settlePromise (C:\web\chat-bot-telegram\node_modules\bluebird\js\release\promise.js:604:18)
    at Promise._settlePromise0 (C:\web\chat-bot-telegram\node_modules\bluebird\js\release\promise.js:649:10)
    at Promise._settlePromises (C:\web\chat-bot-telegram\node_modules\bluebird\js\release\promise.js:729:18)
    at _drainQueueStep (C:\web\chat-bot-telegram\node_modules\bluebird\js\release\async.js:93:12)
    at _drainQueue (C:\web\chat-bot-telegram\node_modules\bluebird\js\release\async.js:86:9)
    at Async._drainQueues (C:\web\chat-bot-telegram\node_modules\bluebird\js\release\async.js:102:5)
    at Async.drainQueues [as _onImmediate] (C:\web\chat-bot-telegram\node_modules\bluebird\js\release\async.js:15:14)
    at process.processImmediate (node:internal/timers:478:21)
[23.08.2024, 06:13:41] Error: ETELEGRAM: 502 Bad Gateway
    at C:\web\chat-bot-telegram\node_modules\node-telegram-bot-api\src\telegram.js:316:15
    at tryCatcher (C:\web\chat-bot-telegram\node_modules\bluebird\js\release\util.js:16:23)
    at Promise._settlePromiseFromHandler (C:\web\chat-bot-telegram\node_modules\bluebird\js\release\promise.js:547:31)
    at Promise._settlePromise (C:\web\chat-bot-telegram\node_modules\bluebird\js\release\promise.js:604:18)
    at Promise._settlePromise0 (C:\web\chat-bot-telegram\node_modules\bluebird\js\release\promise.js:649:10)
    at Promise._settlePromises (C:\web\chat-bot-telegram\node_modules\bluebird\js\release\promise.js:729:18)
    at _drainQueueStep (C:\web\chat-bot-telegram\node_modules\bluebird\js\release\async.js:93:12)
    at _drainQueue (C:\web\chat-bot-telegram\node_modules\bluebird\js\release\async.js:86:9)
    at Async._drainQueues (C:\web\chat-bot-telegram\node_modules\bluebird\js\release\async.js:102:5)
    at Async.drainQueues [as _onImmediate] (C:\web\chat-bot-telegram\node_modules\bluebird\js\release\async.js:15:14)
    at process.processImmediate (node:internal/timers:478:21)
[23.08.2024, 06:13:42] Error: ETELEGRAM: 502 Bad Gateway
    at C:\web\chat-bot-telegram\node_modules\node-telegram-bot-api\src\telegram.js:316:15
    at tryCatcher (C:\web\chat-bot-telegram\node_modules\bluebird\js\release\util.js:16:23)
    at Promise._settlePromiseFromHandler (C:\web\chat-bot-telegram\node_modules\bluebird\js\release\promise.js:547:31)
    at Promise._settlePromise (C:\web\chat-bot-telegram\node_modules\bluebird\js\release\promise.js:604:18)
    at Promise._settlePromise0 (C:\web\chat-bot-telegram\node_modules\bluebird\js\release\promise.js:649:10)
    at Promise._settlePromises (C:\web\chat-bot-telegram\node_modules\bluebird\js\release\promise.js:729:18)
    at _drainQueueStep (C:\web\chat-bot-telegram\node_modules\bluebird\js\release\async.js:93:12)
    at _drainQueue (C:\web\chat-bot-telegram\node_modules\bluebird\js\release\async.js:86:9)
    at Async._drainQueues (C:\web\chat-bot-telegram\node_modules\bluebird\js\release\async.js:102:5)
    at Async.drainQueues [as _onImmediate] (C:\web\chat-bot-telegram\node_modules\bluebird\js\release\async.js:15:14)
    at process.processImmediate (node:internal/timers:478:21)
[23.08.2024, 06:13:42] Error: ETELEGRAM: 502 Bad Gateway
    at C:\web\chat-bot-telegram\node_modules\node-telegram-bot-api\src\telegram.js:316:15
    at tryCatcher (C:\web\chat-bot-telegram\node_modules\bluebird\js\release\util.js:16:23)
    at Promise._settlePromiseFromHandler (C:\web\chat-bot-telegram\node_modules\bluebird\js\release\promise.js:547:31)
    at Promise._settlePromise (C:\web\chat-bot-telegram\node_modules\bluebird\js\release\promise.js:604:18)
    at Promise._settlePromise0 (C:\web\chat-bot-telegram\node_modules\bluebird\js\release\promise.js:649:10)
    at Promise._settlePromises (C:\web\chat-bot-telegram\node_modules\bluebird\js\release\promise.js:729:18)
    at _drainQueueStep (C:\web\chat-bot-telegram\node_modules\bluebird\js\release\async.js:93:12)
    at _drainQueue (C:\web\chat-bot-telegram\node_modules\bluebird\js\release\async.js:86:9)
    at Async._drainQueues (C:\web\chat-bot-telegram\node_modules\bluebird\js\release\async.js:102:5)
    at Async.drainQueues [as _onImmediate] (C:\web\chat-bot-telegram\node_modules\bluebird\js\release\async.js:15:14)
    at process.processImmediate (node:internal/timers:478:21)
[23.08.2024, 06:13:43] Error: ETELEGRAM: 502 Bad Gateway
    at C:\web\chat-bot-telegram\node_modules\node-telegram-bot-api\src\telegram.js:316:15
    at tryCatcher (C:\web\chat-bot-telegram\node_modules\bluebird\js\release\util.js:16:23)
    at Promise._settlePromiseFromHandler (C:\web\chat-bot-telegram\node_modules\bluebird\js\release\promise.js:547:31)
    at Promise._settlePromise (C:\web\chat-bot-telegram\node_modules\bluebird\js\release\promise.js:604:18)
    at Promise._settlePromise0 (C:\web\chat-bot-telegram\node_modules\bluebird\js\release\promise.js:649:10)
    at Promise._settlePromises (C:\web\chat-bot-telegram\node_modules\bluebird\js\release\promise.js:729:18)
    at _drainQueueStep (C:\web\chat-bot-telegram\node_modules\bluebird\js\release\async.js:93:12)
    at _drainQueue (C:\web\chat-bot-telegram\node_modules\bluebird\js\release\async.js:86:9)
    at Async._drainQueues (C:\web\chat-bot-telegram\node_modules\bluebird\js\release\async.js:102:5)
    at Async.drainQueues [as _onImmediate] (C:\web\chat-bot-telegram\node_modules\bluebird\js\release\async.js:15:14)
    at process.processImmediate (node:internal/timers:478:21)
[23.08.2024, 06:13:43] Error: ETELEGRAM: 502 Bad Gateway
    at C:\web\chat-bot-telegram\node_modules\node-telegram-bot-api\src\telegram.js:316:15
    at tryCatcher (C:\web\chat-bot-telegram\node_modules\bluebird\js\release\util.js:16:23)
    at Promise._settlePromiseFromHandler (C:\web\chat-bot-telegram\node_modules\bluebird\js\release\promise.js:547:31)
    at Promise._settlePromise (C:\web\chat-bot-telegram\node_modules\bluebird\js\release\promise.js:604:18)
    at Promise._settlePromise0 (C:\web\chat-bot-telegram\node_modules\bluebird\js\release\promise.js:649:10)
    at Promise._settlePromises (C:\web\chat-bot-telegram\node_modules\bluebird\js\release\promise.js:729:18)
    at _drainQueueStep (C:\web\chat-bot-telegram\node_modules\bluebird\js\release\async.js:93:12)
    at _drainQueue (C:\web\chat-bot-telegram\node_modules\bluebird\js\release\async.js:86:9)
    at Async._drainQueues (C:\web\chat-bot-telegram\node_modules\bluebird\js\release\async.js:102:5)
    at Async.drainQueues [as _onImmediate] (C:\web\chat-bot-telegram\node_modules\bluebird\js\release\async.js:15:14)
    at process.processImmediate (node:internal/timers:478:21)
[23.08.2024, 06:13:44] Error: ETELEGRAM: 502 Bad Gateway
    at C:\web\chat-bot-telegram\node_modules\node-telegram-bot-api\src\telegram.js:316:15
    at tryCatcher (C:\web\chat-bot-telegram\node_modules\bluebird\js\release\util.js:16:23)
    at Promise._settlePromiseFromHandler (C:\web\chat-bot-telegram\node_modules\bluebird\js\release\promise.js:547:31)
    at Promise._settlePromise (C:\web\chat-bot-telegram\node_modules\bluebird\js\release\promise.js:604:18)
    at Promise._settlePromise0 (C:\web\chat-bot-telegram\node_modules\bluebird\js\release\promise.js:649:10)
    at Promise._settlePromises (C:\web\chat-bot-telegram\node_modules\bluebird\js\release\promise.js:729:18)
    at _drainQueueStep (C:\web\chat-bot-telegram\node_modules\bluebird\js\release\async.js:93:12)
    at _drainQueue (C:\web\chat-bot-telegram\node_modules\bluebird\js\release\async.js:86:9)
    at Async._drainQueues (C:\web\chat-bot-telegram\node_modules\bluebird\js\release\async.js:102:5)
    at Async.drainQueues [as _onImmediate] (C:\web\chat-bot-telegram\node_modules\bluebird\js\release\async.js:15:14)
    at process.processImmediate (node:internal/timers:478:21)
[23.08.2024, 06:13:44] Error: ETELEGRAM: 502 Bad Gateway
    at C:\web\chat-bot-telegram\node_modules\node-telegram-bot-api\src\telegram.js:316:15
    at tryCatcher (C:\web\chat-bot-telegram\node_modules\bluebird\js\release\util.js:16:23)
    at Promise._settlePromiseFromHandler (C:\web\chat-bot-telegram\node_modules\bluebird\js\release\promise.js:547:31)
    at Promise._settlePromise (C:\web\chat-bot-telegram\node_modules\bluebird\js\release\promise.js:604:18)
    at Promise._settlePromise0 (C:\web\chat-bot-telegram\node_modules\bluebird\js\release\promise.js:649:10)
    at Promise._settlePromises (C:\web\chat-bot-telegram\node_modules\bluebird\js\release\promise.js:729:18)
    at _drainQueueStep (C:\web\chat-bot-telegram\node_modules\bluebird\js\release\async.js:93:12)
    at _drainQueue (C:\web\chat-bot-telegram\node_modules\bluebird\js\release\async.js:86:9)
    at Async._drainQueues (C:\web\chat-bot-telegram\node_modules\bluebird\js\release\async.js:102:5)
    at Async.drainQueues [as _onImmediate] (C:\web\chat-bot-telegram\node_modules\bluebird\js\release\async.js:15:14)
    at process.processImmediate (node:internal/timers:478:21)
[23.08.2024, 06:30:54] RequestError: Error: socket hang up
    at new RequestError (C:\web\chat-bot-telegram\node_modules\request-promise-core\lib\errors.js:14:15)
    at plumbing.callback (C:\web\chat-bot-telegram\node_modules\request-promise-core\lib\plumbing.js:87:29)
    at Request.RP$callback [as _callback] (C:\web\chat-bot-telegram\node_modules\request-promise-core\lib\plumbing.js:46:31)
    at self.callback (C:\web\chat-bot-telegram\node_modules\@cypress\request\request.js:183:22)
    at Request.emit (node:events:518:28)
    at Request.onRequestError (C:\web\chat-bot-telegram\node_modules\@cypress\request\request.js:869:8)
    at ClientRequest.emit (node:events:518:28)
    at TLSSocket.socketOnEnd (node:_http_client:524:9)
    at TLSSocket.emit (node:events:530:35)
    at endReadableNT (node:internal/streams/readable:1696:12)
    at process.processTicksAndRejections (node:internal/process/task_queues:82:21)
[23.08.2024, 06:30:55] RequestError: Error: read ECONNRESET
    at new RequestError (C:\web\chat-bot-telegram\node_modules\request-promise-core\lib\errors.js:14:15)
    at plumbing.callback (C:\web\chat-bot-telegram\node_modules\request-promise-core\lib\plumbing.js:87:29)
    at Request.RP$callback [as _callback] (C:\web\chat-bot-telegram\node_modules\request-promise-core\lib\plumbing.js:46:31)
    at self.callback (C:\web\chat-bot-telegram\node_modules\@cypress\request\request.js:183:22)
    at Request.emit (node:events:518:28)
    at Request.onRequestError (C:\web\chat-bot-telegram\node_modules\@cypress\request\request.js:869:8)
    at ClientRequest.emit (node:events:518:28)
    at _destroy (node:_http_client:880:13)
    at onSocketNT (node:_http_client:900:5)
    at process.processTicksAndRejections (node:internal/process/task_queues:83:21)
[23.08.2024, 06:31:16] RequestError: Error: connect ETIMEDOUT 169.254.0.51:3274
    at new RequestError (C:\web\chat-bot-telegram\node_modules\request-promise-core\lib\errors.js:14:15)
    at plumbing.callback (C:\web\chat-bot-telegram\node_modules\request-promise-core\lib\plumbing.js:87:29)
    at Request.RP$callback [as _callback] (C:\web\chat-bot-telegram\node_modules\request-promise-core\lib\plumbing.js:46:31)
    at self.callback (C:\web\chat-bot-telegram\node_modules\@cypress\request\request.js:183:22)
    at Request.emit (node:events:518:28)
    at Request.onRequestError (C:\web\chat-bot-telegram\node_modules\@cypress\request\request.js:869:8)
    at ClientRequest.emit (node:events:518:28)
    at _destroy (node:_http_client:880:13)
    at onSocketNT (node:_http_client:900:5)
    at process.processTicksAndRejections (node:internal/process/task_queues:83:21)
[23.08.2024, 06:31:38] RequestError: Error: connect ETIMEDOUT 169.254.0.51:3274
    at new RequestError (C:\web\chat-bot-telegram\node_modules\request-promise-core\lib\errors.js:14:15)
    at plumbing.callback (C:\web\chat-bot-telegram\node_modules\request-promise-core\lib\plumbing.js:87:29)
    at Request.RP$callback [as _callback] (C:\web\chat-bot-telegram\node_modules\request-promise-core\lib\plumbing.js:46:31)
    at self.callback (C:\web\chat-bot-telegram\node_modules\@cypress\request\request.js:183:22)
    at Request.emit (node:events:518:28)
    at Request.onRequestError (C:\web\chat-bot-telegram\node_modules\@cypress\request\request.js:869:8)
    at ClientRequest.emit (node:events:518:28)
    at _destroy (node:_http_client:880:13)
    at onSocketNT (node:_http_client:900:5)
    at process.processTicksAndRejections (node:internal/process/task_queues:83:21)
[23.08.2024, 06:31:59] RequestError: Error: connect ETIMEDOUT 169.254.0.51:3274
    at new RequestError (C:\web\chat-bot-telegram\node_modules\request-promise-core\lib\errors.js:14:15)
    at plumbing.callback (C:\web\chat-bot-telegram\node_modules\request-promise-core\lib\plumbing.js:87:29)
    at Request.RP$callback [as _callback] (C:\web\chat-bot-telegram\node_modules\request-promise-core\lib\plumbing.js:46:31)
    at self.callback (C:\web\chat-bot-telegram\node_modules\@cypress\request\request.js:183:22)
    at Request.emit (node:events:518:28)
    at Request.onRequestError (C:\web\chat-bot-telegram\node_modules\@cypress\request\request.js:869:8)
    at ClientRequest.emit (node:events:518:28)
    at _destroy (node:_http_client:880:13)
    at onSocketNT (node:_http_client:900:5)
    at process.processTicksAndRejections (node:internal/process/task_queues:83:21)
[23.08.2024, 06:32:21] RequestError: Error: connect ETIMEDOUT 169.254.0.51:3274
    at new RequestError (C:\web\chat-bot-telegram\node_modules\request-promise-core\lib\errors.js:14:15)
    at plumbing.callback (C:\web\chat-bot-telegram\node_modules\request-promise-core\lib\plumbing.js:87:29)
    at Request.RP$callback [as _callback] (C:\web\chat-bot-telegram\node_modules\request-promise-core\lib\plumbing.js:46:31)
    at self.callback (C:\web\chat-bot-telegram\node_modules\@cypress\request\request.js:183:22)
    at Request.emit (node:events:518:28)
    at Request.onRequestError (C:\web\chat-bot-telegram\node_modules\@cypress\request\request.js:869:8)
    at ClientRequest.emit (node:events:518:28)
    at _destroy (node:_http_client:880:13)
    at onSocketNT (node:_http_client:900:5)
    at process.processTicksAndRejections (node:internal/process/task_queues:83:21)
[23.08.2024, 08:31:58] RequestError: Error: connect ECONNREFUSED 169.254.0.51:3274
    at new RequestError (C:\web\chat-bot-telegram\node_modules\request-promise-core\lib\errors.js:14:15)
    at plumbing.callback (C:\web\chat-bot-telegram\node_modules\request-promise-core\lib\plumbing.js:87:29)
    at Request.RP$callback [as _callback] (C:\web\chat-bot-telegram\node_modules\request-promise-core\lib\plumbing.js:46:31)
    at self.callback (C:\web\chat-bot-telegram\node_modules\@cypress\request\request.js:183:22)
    at Request.emit (node:events:518:28)
    at Request.onRequestError (C:\web\chat-bot-telegram\node_modules\@cypress\request\request.js:869:8)
    at ClientRequest.emit (node:events:518:28)
    at _destroy (node:_http_client:880:13)
    at onSocketNT (node:_http_client:900:5)
    at process.processTicksAndRejections (node:internal/process/task_queues:83:21)
[23.08.2024, 08:32:22] RequestError: Error: connect ECONNREFUSED 169.254.0.51:3274
    at new RequestError (C:\web\chat-bot-telegram\node_modules\request-promise-core\lib\errors.js:14:15)
    at plumbing.callback (C:\web\chat-bot-telegram\node_modules\request-promise-core\lib\plumbing.js:87:29)
    at Request.RP$callback [as _callback] (C:\web\chat-bot-telegram\node_modules\request-promise-core\lib\plumbing.js:46:31)
    at self.callback (C:\web\chat-bot-telegram\node_modules\@cypress\request\request.js:183:22)
    at Request.emit (node:events:518:28)
    at Request.onRequestError (C:\web\chat-bot-telegram\node_modules\@cypress\request\request.js:869:8)
    at ClientRequest.emit (node:events:518:28)
    at _destroy (node:_http_client:880:13)
    at onSocketNT (node:_http_client:900:5)
    at process.processTicksAndRejections (node:internal/process/task_queues:83:21)
[23.08.2024, 08:32:24] RequestError: Error: connect ECONNREFUSED 169.254.0.51:3274
    at new RequestError (C:\web\chat-bot-telegram\node_modules\request-promise-core\lib\errors.js:14:15)
    at plumbing.callback (C:\web\chat-bot-telegram\node_modules\request-promise-core\lib\plumbing.js:87:29)
    at Request.RP$callback [as _callback] (C:\web\chat-bot-telegram\node_modules\request-promise-core\lib\plumbing.js:46:31)
    at self.callback (C:\web\chat-bot-telegram\node_modules\@cypress\request\request.js:183:22)
    at Request.emit (node:events:518:28)
    at Request.onRequestError (C:\web\chat-bot-telegram\node_modules\@cypress\request\request.js:869:8)
    at ClientRequest.emit (node:events:518:28)
    at _destroy (node:_http_client:880:13)
    at onSocketNT (node:_http_client:900:5)
    at process.processTicksAndRejections (node:internal/process/task_queues:83:21)
[23.08.2024, 08:32:27] RequestError: Error: connect ECONNREFUSED 169.254.0.51:3274
    at new RequestError (C:\web\chat-bot-telegram\node_modules\request-promise-core\lib\errors.js:14:15)
    at plumbing.callback (C:\web\chat-bot-telegram\node_modules\request-promise-core\lib\plumbing.js:87:29)
    at Request.RP$callback [as _callback] (C:\web\chat-bot-telegram\node_modules\request-promise-core\lib\plumbing.js:46:31)
    at self.callback (C:\web\chat-bot-telegram\node_modules\@cypress\request\request.js:183:22)
    at Request.emit (node:events:518:28)
    at Request.onRequestError (C:\web\chat-bot-telegram\node_modules\@cypress\request\request.js:869:8)
    at ClientRequest.emit (node:events:518:28)
    at _destroy (node:_http_client:880:13)
    at onSocketNT (node:_http_client:900:5)
    at process.processTicksAndRejections (node:internal/process/task_queues:83:21)
[23.08.2024, 08:32:29] RequestError: Error: connect ECONNREFUSED 169.254.0.51:3274
    at new RequestError (C:\web\chat-bot-telegram\node_modules\request-promise-core\lib\errors.js:14:15)
    at plumbing.callback (C:\web\chat-bot-telegram\node_modules\request-promise-core\lib\plumbing.js:87:29)
    at Request.RP$callback [as _callback] (C:\web\chat-bot-telegram\node_modules\request-promise-core\lib\plumbing.js:46:31)
    at self.callback (C:\web\chat-bot-telegram\node_modules\@cypress\request\request.js:183:22)
    at Request.emit (node:events:518:28)
    at Request.onRequestError (C:\web\chat-bot-telegram\node_modules\@cypress\request\request.js:869:8)
    at ClientRequest.emit (node:events:518:28)
    at _destroy (node:_http_client:880:13)
    at onSocketNT (node:_http_client:900:5)
    at process.processTicksAndRejections (node:internal/process/task_queues:83:21)
[23.08.2024, 09:24:14] RequestError: Error: connect ECONNREFUSED 169.254.0.51:3274
    at new RequestError (C:\web\chat-bot-telegram\node_modules\request-promise-core\lib\errors.js:14:15)
    at plumbing.callback (C:\web\chat-bot-telegram\node_modules\request-promise-core\lib\plumbing.js:87:29)
    at Request.RP$callback [as _callback] (C:\web\chat-bot-telegram\node_modules\request-promise-core\lib\plumbing.js:46:31)
    at self.callback (C:\web\chat-bot-telegram\node_modules\@cypress\request\request.js:183:22)
    at Request.emit (node:events:518:28)
    at Request.onRequestError (C:\web\chat-bot-telegram\node_modules\@cypress\request\request.js:869:8)
    at ClientRequest.emit (node:events:518:28)
    at _destroy (node:_http_client:880:13)
    at onSocketNT (node:_http_client:900:5)
    at process.processTicksAndRejections (node:internal/process/task_queues:83:21)
[23.08.2024, 09:24:17] RequestError: Error: connect ECONNREFUSED 169.254.0.51:3274
    at new RequestError (C:\web\chat-bot-telegram\node_modules\request-promise-core\lib\errors.js:14:15)
    at plumbing.callback (C:\web\chat-bot-telegram\node_modules\request-promise-core\lib\plumbing.js:87:29)
    at Request.RP$callback [as _callback] (C:\web\chat-bot-telegram\node_modules\request-promise-core\lib\plumbing.js:46:31)
    at self.callback (C:\web\chat-bot-telegram\node_modules\@cypress\request\request.js:183:22)
    at Request.emit (node:events:518:28)
    at Request.onRequestError (C:\web\chat-bot-telegram\node_modules\@cypress\request\request.js:869:8)
    at ClientRequest.emit (node:events:518:28)
    at _destroy (node:_http_client:880:13)
    at onSocketNT (node:_http_client:900:5)
    at process.processTicksAndRejections (node:internal/process/task_queues:83:21)
[23.08.2024, 09:24:19] RequestError: Error: connect ECONNREFUSED 169.254.0.51:3274
    at new RequestError (C:\web\chat-bot-telegram\node_modules\request-promise-core\lib\errors.js:14:15)
    at plumbing.callback (C:\web\chat-bot-telegram\node_modules\request-promise-core\lib\plumbing.js:87:29)
    at Request.RP$callback [as _callback] (C:\web\chat-bot-telegram\node_modules\request-promise-core\lib\plumbing.js:46:31)
    at self.callback (C:\web\chat-bot-telegram\node_modules\@cypress\request\request.js:183:22)
    at Request.emit (node:events:518:28)
    at Request.onRequestError (C:\web\chat-bot-telegram\node_modules\@cypress\request\request.js:869:8)
    at ClientRequest.emit (node:events:518:28)
    at _destroy (node:_http_client:880:13)
    at onSocketNT (node:_http_client:900:5)
    at process.processTicksAndRejections (node:internal/process/task_queues:83:21)
[23.08.2024, 09:24:21] RequestError: Error: connect ECONNREFUSED 169.254.0.51:3274
    at new RequestError (C:\web\chat-bot-telegram\node_modules\request-promise-core\lib\errors.js:14:15)
    at plumbing.callback (C:\web\chat-bot-telegram\node_modules\request-promise-core\lib\plumbing.js:87:29)
    at Request.RP$callback [as _callback] (C:\web\chat-bot-telegram\node_modules\request-promise-core\lib\plumbing.js:46:31)
    at self.callback (C:\web\chat-bot-telegram\node_modules\@cypress\request\request.js:183:22)
    at Request.emit (node:events:518:28)
    at Request.onRequestError (C:\web\chat-bot-telegram\node_modules\@cypress\request\request.js:869:8)
    at ClientRequest.emit (node:events:518:28)
    at _destroy (node:_http_client:880:13)
    at onSocketNT (node:_http_client:900:5)
    at process.processTicksAndRejections (node:internal/process/task_queues:83:21)
[23.08.2024, 09:24:24] RequestError: Error: connect ECONNREFUSED 169.254.0.51:3274
    at new RequestError (C:\web\chat-bot-telegram\node_modules\request-promise-core\lib\errors.js:14:15)
    at plumbing.callback (C:\web\chat-bot-telegram\node_modules\request-promise-core\lib\plumbing.js:87:29)
    at Request.RP$callback [as _callback] (C:\web\chat-bot-telegram\node_modules\request-promise-core\lib\plumbing.js:46:31)
    at self.callback (C:\web\chat-bot-telegram\node_modules\@cypress\request\request.js:183:22)
    at Request.emit (node:events:518:28)
    at Request.onRequestError (C:\web\chat-bot-telegram\node_modules\@cypress\request\request.js:869:8)
    at ClientRequest.emit (node:events:518:28)
    at _destroy (node:_http_client:880:13)
    at onSocketNT (node:_http_client:900:5)
    at process.processTicksAndRejections (node:internal/process/task_queues:83:21)
[23.08.2024, 09:26:37] RequestError: Error: connect ECONNREFUSED 169.254.0.51:3274
    at new RequestError (C:\web\chat-bot-telegram\node_modules\request-promise-core\lib\errors.js:14:15)
    at plumbing.callback (C:\web\chat-bot-telegram\node_modules\request-promise-core\lib\plumbing.js:87:29)
    at Request.RP$callback [as _callback] (C:\web\chat-bot-telegram\node_modules\request-promise-core\lib\plumbing.js:46:31)
    at self.callback (C:\web\chat-bot-telegram\node_modules\@cypress\request\request.js:183:22)
    at Request.emit (node:events:518:28)
    at Request.onRequestError (C:\web\chat-bot-telegram\node_modules\@cypress\request\request.js:869:8)
    at ClientRequest.emit (node:events:518:28)
    at _destroy (node:_http_client:880:13)
    at onSocketNT (node:_http_client:900:5)
    at process.processTicksAndRejections (node:internal/process/task_queues:83:21)
[23.08.2024, 09:29:53] RequestError: Error: connect ECONNREFUSED 169.254.0.51:3274
    at new RequestError (C:\web\chat-bot-telegram\node_modules\request-promise-core\lib\errors.js:14:15)
    at plumbing.callback (C:\web\chat-bot-telegram\node_modules\request-promise-core\lib\plumbing.js:87:29)
    at Request.RP$callback [as _callback] (C:\web\chat-bot-telegram\node_modules\request-promise-core\lib\plumbing.js:46:31)
    at self.callback (C:\web\chat-bot-telegram\node_modules\@cypress\request\request.js:183:22)
    at Request.emit (node:events:518:28)
    at Request.onRequestError (C:\web\chat-bot-telegram\node_modules\@cypress\request\request.js:869:8)
    at ClientRequest.emit (node:events:518:28)
    at _destroy (node:_http_client:880:13)
    at onSocketNT (node:_http_client:900:5)
    at process.processTicksAndRejections (node:internal/process/task_queues:83:21)
[23.08.2024, 09:29:56] RequestError: Error: connect ECONNREFUSED 169.254.0.51:3274
    at new RequestError (C:\web\chat-bot-telegram\node_modules\request-promise-core\lib\errors.js:14:15)
    at plumbing.callback (C:\web\chat-bot-telegram\node_modules\request-promise-core\lib\plumbing.js:87:29)
    at Request.RP$callback [as _callback] (C:\web\chat-bot-telegram\node_modules\request-promise-core\lib\plumbing.js:46:31)
    at self.callback (C:\web\chat-bot-telegram\node_modules\@cypress\request\request.js:183:22)
    at Request.emit (node:events:518:28)
    at Request.onRequestError (C:\web\chat-bot-telegram\node_modules\@cypress\request\request.js:869:8)
    at ClientRequest.emit (node:events:518:28)
    at _destroy (node:_http_client:880:13)
    at onSocketNT (node:_http_client:900:5)
    at process.processTicksAndRejections (node:internal/process/task_queues:83:21)
[23.08.2024, 09:29:58] RequestError: Error: connect ECONNREFUSED 169.254.0.51:3274
    at new RequestError (C:\web\chat-bot-telegram\node_modules\request-promise-core\lib\errors.js:14:15)
    at plumbing.callback (C:\web\chat-bot-telegram\node_modules\request-promise-core\lib\plumbing.js:87:29)
    at Request.RP$callback [as _callback] (C:\web\chat-bot-telegram\node_modules\request-promise-core\lib\plumbing.js:46:31)
    at self.callback (C:\web\chat-bot-telegram\node_modules\@cypress\request\request.js:183:22)
    at Request.emit (node:events:518:28)
    at Request.onRequestError (C:\web\chat-bot-telegram\node_modules\@cypress\request\request.js:869:8)
    at ClientRequest.emit (node:events:518:28)
    at _destroy (node:_http_client:880:13)
    at onSocketNT (node:_http_client:900:5)
    at process.processTicksAndRejections (node:internal/process/task_queues:83:21)
[23.08.2024, 09:30:00] RequestError: Error: connect ECONNREFUSED 169.254.0.51:3274
    at new RequestError (C:\web\chat-bot-telegram\node_modules\request-promise-core\lib\errors.js:14:15)
    at plumbing.callback (C:\web\chat-bot-telegram\node_modules\request-promise-core\lib\plumbing.js:87:29)
    at Request.RP$callback [as _callback] (C:\web\chat-bot-telegram\node_modules\request-promise-core\lib\plumbing.js:46:31)
    at self.callback (C:\web\chat-bot-telegram\node_modules\@cypress\request\request.js:183:22)
    at Request.emit (node:events:518:28)
    at Request.onRequestError (C:\web\chat-bot-telegram\node_modules\@cypress\request\request.js:869:8)
    at ClientRequest.emit (node:events:518:28)
    at _destroy (node:_http_client:880:13)
    at onSocketNT (node:_http_client:900:5)
    at process.processTicksAndRejections (node:internal/process/task_queues:83:21)
[23.08.2024, 09:30:03] RequestError: Error: connect ECONNREFUSED 169.254.0.51:3274
    at new RequestError (C:\web\chat-bot-telegram\node_modules\request-promise-core\lib\errors.js:14:15)
    at plumbing.callback (C:\web\chat-bot-telegram\node_modules\request-promise-core\lib\plumbing.js:87:29)
    at Request.RP$callback [as _callback] (C:\web\chat-bot-telegram\node_modules\request-promise-core\lib\plumbing.js:46:31)
    at self.callback (C:\web\chat-bot-telegram\node_modules\@cypress\request\request.js:183:22)
    at Request.emit (node:events:518:28)
    at Request.onRequestError (C:\web\chat-bot-telegram\node_modules\@cypress\request\request.js:869:8)
    at ClientRequest.emit (node:events:518:28)
    at _destroy (node:_http_client:880:13)
    at onSocketNT (node:_http_client:900:5)
    at process.processTicksAndRejections (node:internal/process/task_queues:83:21)
[23.08.2024, 09:30:05] RequestError: Error: connect ECONNREFUSED 169.254.0.51:3274
    at new RequestError (C:\web\chat-bot-telegram\node_modules\request-promise-core\lib\errors.js:14:15)
    at plumbing.callback (C:\web\chat-bot-telegram\node_modules\request-promise-core\lib\plumbing.js:87:29)
    at Request.RP$callback [as _callback] (C:\web\chat-bot-telegram\node_modules\request-promise-core\lib\plumbing.js:46:31)
    at self.callback (C:\web\chat-bot-telegram\node_modules\@cypress\request\request.js:183:22)
    at Request.emit (node:events:518:28)
    at Request.onRequestError (C:\web\chat-bot-telegram\node_modules\@cypress\request\request.js:869:8)
    at ClientRequest.emit (node:events:518:28)
    at _destroy (node:_http_client:880:13)
    at onSocketNT (node:_http_client:900:5)
    at process.processTicksAndRejections (node:internal/process/task_queues:83:21)
[23.08.2024, 09:30:07] RequestError: Error: connect ECONNREFUSED 169.254.0.51:3274
    at new RequestError (C:\web\chat-bot-telegram\node_modules\request-promise-core\lib\errors.js:14:15)
    at plumbing.callback (C:\web\chat-bot-telegram\node_modules\request-promise-core\lib\plumbing.js:87:29)
    at Request.RP$callback [as _callback] (C:\web\chat-bot-telegram\node_modules\request-promise-core\lib\plumbing.js:46:31)
    at self.callback (C:\web\chat-bot-telegram\node_modules\@cypress\request\request.js:183:22)
    at Request.emit (node:events:518:28)
    at Request.onRequestError (C:\web\chat-bot-telegram\node_modules\@cypress\request\request.js:869:8)
    at ClientRequest.emit (node:events:518:28)
    at _destroy (node:_http_client:880:13)
    at onSocketNT (node:_http_client:900:5)
    at process.processTicksAndRejections (node:internal/process/task_queues:83:21)
[23.08.2024, 14:59:12] RequestError: Error: connect ECONNREFUSED 169.254.0.51:3274
    at new RequestError (C:\web\chat-bot-telegram\node_modules\request-promise-core\lib\errors.js:14:15)
    at plumbing.callback (C:\web\chat-bot-telegram\node_modules\request-promise-core\lib\plumbing.js:87:29)
    at Request.RP$callback [as _callback] (C:\web\chat-bot-telegram\node_modules\request-promise-core\lib\plumbing.js:46:31)
    at self.callback (C:\web\chat-bot-telegram\node_modules\@cypress\request\request.js:183:22)
    at Request.emit (node:events:518:28)
    at Request.onRequestError (C:\web\chat-bot-telegram\node_modules\@cypress\request\request.js:869:8)
    at ClientRequest.emit (node:events:518:28)
    at _destroy (node:_http_client:880:13)
    at onSocketNT (node:_http_client:900:5)
    at process.processTicksAndRejections (node:internal/process/task_queues:83:21)
[23.08.2024, 14:59:14] RequestError: Error: connect ECONNREFUSED 169.254.0.51:3274
    at new RequestError (C:\web\chat-bot-telegram\node_modules\request-promise-core\lib\errors.js:14:15)
    at plumbing.callback (C:\web\chat-bot-telegram\node_modules\request-promise-core\lib\plumbing.js:87:29)
    at Request.RP$callback [as _callback] (C:\web\chat-bot-telegram\node_modules\request-promise-core\lib\plumbing.js:46:31)
    at self.callback (C:\web\chat-bot-telegram\node_modules\@cypress\request\request.js:183:22)
    at Request.emit (node:events:518:28)
    at Request.onRequestError (C:\web\chat-bot-telegram\node_modules\@cypress\request\request.js:869:8)
    at ClientRequest.emit (node:events:518:28)
    at _destroy (node:_http_client:880:13)
    at onSocketNT (node:_http_client:900:5)
    at process.processTicksAndRejections (node:internal/process/task_queues:83:21)
[23.08.2024, 14:59:16] RequestError: Error: connect ECONNREFUSED 169.254.0.51:3274
    at new RequestError (C:\web\chat-bot-telegram\node_modules\request-promise-core\lib\errors.js:14:15)
    at plumbing.callback (C:\web\chat-bot-telegram\node_modules\request-promise-core\lib\plumbing.js:87:29)
    at Request.RP$callback [as _callback] (C:\web\chat-bot-telegram\node_modules\request-promise-core\lib\plumbing.js:46:31)
    at self.callback (C:\web\chat-bot-telegram\node_modules\@cypress\request\request.js:183:22)
    at Request.emit (node:events:518:28)
    at Request.onRequestError (C:\web\chat-bot-telegram\node_modules\@cypress\request\request.js:869:8)
    at ClientRequest.emit (node:events:518:28)
    at _destroy (node:_http_client:880:13)
    at onSocketNT (node:_http_client:900:5)
    at process.processTicksAndRejections (node:internal/process/task_queues:83:21)
[23.08.2024, 14:59:19] RequestError: Error: connect ECONNREFUSED 169.254.0.51:3274
    at new RequestError (C:\web\chat-bot-telegram\node_modules\request-promise-core\lib\errors.js:14:15)
    at plumbing.callback (C:\web\chat-bot-telegram\node_modules\request-promise-core\lib\plumbing.js:87:29)
    at Request.RP$callback [as _callback] (C:\web\chat-bot-telegram\node_modules\request-promise-core\lib\plumbing.js:46:31)
    at self.callback (C:\web\chat-bot-telegram\node_modules\@cypress\request\request.js:183:22)
    at Request.emit (node:events:518:28)
    at Request.onRequestError (C:\web\chat-bot-telegram\node_modules\@cypress\request\request.js:869:8)
    at ClientRequest.emit (node:events:518:28)
    at _destroy (node:_http_client:880:13)
    at onSocketNT (node:_http_client:900:5)
    at process.processTicksAndRejections (node:internal/process/task_queues:83:21)
[23.08.2024, 14:59:21] RequestError: Error: connect ECONNREFUSED 169.254.0.51:3274
    at new RequestError (C:\web\chat-bot-telegram\node_modules\request-promise-core\lib\errors.js:14:15)
    at plumbing.callback (C:\web\chat-bot-telegram\node_modules\request-promise-core\lib\plumbing.js:87:29)
    at Request.RP$callback [as _callback] (C:\web\chat-bot-telegram\node_modules\request-promise-core\lib\plumbing.js:46:31)
    at self.callback (C:\web\chat-bot-telegram\node_modules\@cypress\request\request.js:183:22)
    at Request.emit (node:events:518:28)
    at Request.onRequestError (C:\web\chat-bot-telegram\node_modules\@cypress\request\request.js:869:8)
    at ClientRequest.emit (node:events:518:28)
    at _destroy (node:_http_client:880:13)
    at onSocketNT (node:_http_client:900:5)
    at process.processTicksAndRejections (node:internal/process/task_queues:83:21)
[23.08.2024, 14:59:27] RequestError: Error: connect ECONNREFUSED 169.254.0.51:3274
    at new RequestError (C:\web\chat-bot-telegram\node_modules\request-promise-core\lib\errors.js:14:15)
    at plumbing.callback (C:\web\chat-bot-telegram\node_modules\request-promise-core\lib\plumbing.js:87:29)
    at Request.RP$callback [as _callback] (C:\web\chat-bot-telegram\node_modules\request-promise-core\lib\plumbing.js:46:31)
    at self.callback (C:\web\chat-bot-telegram\node_modules\@cypress\request\request.js:183:22)
    at Request.emit (node:events:518:28)
    at Request.onRequestError (C:\web\chat-bot-telegram\node_modules\@cypress\request\request.js:869:8)
    at ClientRequest.emit (node:events:518:28)
    at _destroy (node:_http_client:880:13)
    at onSocketNT (node:_http_client:900:5)
    at process.processTicksAndRejections (node:internal/process/task_queues:83:21)
[23.08.2024, 15:01:40] RequestError: Error: connect ECONNREFUSED 169.254.0.51:3274
    at new RequestError (C:\web\chat-bot-telegram\node_modules\request-promise-core\lib\errors.js:14:15)
    at plumbing.callback (C:\web\chat-bot-telegram\node_modules\request-promise-core\lib\plumbing.js:87:29)
    at Request.RP$callback [as _callback] (C:\web\chat-bot-telegram\node_modules\request-promise-core\lib\plumbing.js:46:31)
    at self.callback (C:\web\chat-bot-telegram\node_modules\@cypress\request\request.js:183:22)
    at Request.emit (node:events:518:28)
    at Request.onRequestError (C:\web\chat-bot-telegram\node_modules\@cypress\request\request.js:869:8)
    at ClientRequest.emit (node:events:518:28)
    at _destroy (node:_http_client:880:13)
    at onSocketNT (node:_http_client:900:5)
    at process.processTicksAndRejections (node:internal/process/task_queues:83:21)
[23.08.2024, 15:01:42] RequestError: Error: connect ECONNREFUSED 169.254.0.51:3274
    at new RequestError (C:\web\chat-bot-telegram\node_modules\request-promise-core\lib\errors.js:14:15)
    at plumbing.callback (C:\web\chat-bot-telegram\node_modules\request-promise-core\lib\plumbing.js:87:29)
    at Request.RP$callback [as _callback] (C:\web\chat-bot-telegram\node_modules\request-promise-core\lib\plumbing.js:46:31)
    at self.callback (C:\web\chat-bot-telegram\node_modules\@cypress\request\request.js:183:22)
    at Request.emit (node:events:518:28)
    at Request.onRequestError (C:\web\chat-bot-telegram\node_modules\@cypress\request\request.js:869:8)
    at ClientRequest.emit (node:events:518:28)
    at _destroy (node:_http_client:880:13)
    at onSocketNT (node:_http_client:900:5)
    at process.processTicksAndRejections (node:internal/process/task_queues:83:21)
[23.08.2024, 15:01:45] RequestError: Error: connect ECONNREFUSED 169.254.0.51:3274
    at new RequestError (C:\web\chat-bot-telegram\node_modules\request-promise-core\lib\errors.js:14:15)
    at plumbing.callback (C:\web\chat-bot-telegram\node_modules\request-promise-core\lib\plumbing.js:87:29)
    at Request.RP$callback [as _callback] (C:\web\chat-bot-telegram\node_modules\request-promise-core\lib\plumbing.js:46:31)
    at self.callback (C:\web\chat-bot-telegram\node_modules\@cypress\request\request.js:183:22)
    at Request.emit (node:events:518:28)
    at Request.onRequestError (C:\web\chat-bot-telegram\node_modules\@cypress\request\request.js:869:8)
    at ClientRequest.emit (node:events:518:28)
    at _destroy (node:_http_client:880:13)
    at onSocketNT (node:_http_client:900:5)
    at process.processTicksAndRejections (node:internal/process/task_queues:83:21)
[23.08.2024, 15:01:47] RequestError: Error: connect ECONNREFUSED 169.254.0.51:3274
    at new RequestError (C:\web\chat-bot-telegram\node_modules\request-promise-core\lib\errors.js:14:15)
    at plumbing.callback (C:\web\chat-bot-telegram\node_modules\request-promise-core\lib\plumbing.js:87:29)
    at Request.RP$callback [as _callback] (C:\web\chat-bot-telegram\node_modules\request-promise-core\lib\plumbing.js:46:31)
    at self.callback (C:\web\chat-bot-telegram\node_modules\@cypress\request\request.js:183:22)
    at Request.emit (node:events:518:28)
    at Request.onRequestError (C:\web\chat-bot-telegram\node_modules\@cypress\request\request.js:869:8)
    at ClientRequest.emit (node:events:518:28)
    at _destroy (node:_http_client:880:13)
    at onSocketNT (node:_http_client:900:5)
    at process.processTicksAndRejections (node:internal/process/task_queues:83:21)
[23.08.2024, 15:01:49] RequestError: Error: connect ECONNREFUSED 169.254.0.51:3274
    at new RequestError (C:\web\chat-bot-telegram\node_modules\request-promise-core\lib\errors.js:14:15)
    at plumbing.callback (C:\web\chat-bot-telegram\node_modules\request-promise-core\lib\plumbing.js:87:29)
    at Request.RP$callback [as _callback] (C:\web\chat-bot-telegram\node_modules\request-promise-core\lib\plumbing.js:46:31)
    at self.callback (C:\web\chat-bot-telegram\node_modules\@cypress\request\request.js:183:22)
    at Request.emit (node:events:518:28)
    at Request.onRequestError (C:\web\chat-bot-telegram\node_modules\@cypress\request\request.js:869:8)
    at ClientRequest.emit (node:events:518:28)
    at _destroy (node:_http_client:880:13)
    at onSocketNT (node:_http_client:900:5)
    at process.processTicksAndRejections (node:internal/process/task_queues:83:21)
[23.08.2024, 15:01:52] RequestError: Error: connect ECONNREFUSED 169.254.0.51:3274
    at new RequestError (C:\web\chat-bot-telegram\node_modules\request-promise-core\lib\errors.js:14:15)
    at plumbing.callback (C:\web\chat-bot-telegram\node_modules\request-promise-core\lib\plumbing.js:87:29)
    at Request.RP$callback [as _callback] (C:\web\chat-bot-telegram\node_modules\request-promise-core\lib\plumbing.js:46:31)
    at self.callback (C:\web\chat-bot-telegram\node_modules\@cypress\request\request.js:183:22)
    at Request.emit (node:events:518:28)
    at Request.onRequestError (C:\web\chat-bot-telegram\node_modules\@cypress\request\request.js:869:8)
    at ClientRequest.emit (node:events:518:28)
    at _destroy (node:_http_client:880:13)
    at onSocketNT (node:_http_client:900:5)
    at process.processTicksAndRejections (node:internal/process/task_queues:83:21)
[23.08.2024, 15:01:54] RequestError: Error: connect ECONNREFUSED 169.254.0.51:3274
    at new RequestError (C:\web\chat-bot-telegram\node_modules\request-promise-core\lib\errors.js:14:15)
    at plumbing.callback (C:\web\chat-bot-telegram\node_modules\request-promise-core\lib\plumbing.js:87:29)
    at Request.RP$callback [as _callback] (C:\web\chat-bot-telegram\node_modules\request-promise-core\lib\plumbing.js:46:31)
    at self.callback (C:\web\chat-bot-telegram\node_modules\@cypress\request\request.js:183:22)
    at Request.emit (node:events:518:28)
    at Request.onRequestError (C:\web\chat-bot-telegram\node_modules\@cypress\request\request.js:869:8)
    at ClientRequest.emit (node:events:518:28)
    at _destroy (node:_http_client:880:13)
    at onSocketNT (node:_http_client:900:5)
    at process.processTicksAndRejections (node:internal/process/task_queues:83:21)
[23.08.2024, 15:01:56] RequestError: Error: connect ECONNREFUSED 169.254.0.51:3274
    at new RequestError (C:\web\chat-bot-telegram\node_modules\request-promise-core\lib\errors.js:14:15)
    at plumbing.callback (C:\web\chat-bot-telegram\node_modules\request-promise-core\lib\plumbing.js:87:29)
    at Request.RP$callback [as _callback] (C:\web\chat-bot-telegram\node_modules\request-promise-core\lib\plumbing.js:46:31)
    at self.callback (C:\web\chat-bot-telegram\node_modules\@cypress\request\request.js:183:22)
    at Request.emit (node:events:518:28)
    at Request.onRequestError (C:\web\chat-bot-telegram\node_modules\@cypress\request\request.js:869:8)
    at ClientRequest.emit (node:events:518:28)
    at _destroy (node:_http_client:880:13)
    at onSocketNT (node:_http_client:900:5)
    at process.processTicksAndRejections (node:internal/process/task_queues:83:21)
[23.08.2024, 15:17:03] RequestError: Error: connect ECONNREFUSED 169.254.0.51:3274
    at new RequestError (C:\web\chat-bot-telegram\node_modules\request-promise-core\lib\errors.js:14:15)
    at plumbing.callback (C:\web\chat-bot-telegram\node_modules\request-promise-core\lib\plumbing.js:87:29)
    at Request.RP$callback [as _callback] (C:\web\chat-bot-telegram\node_modules\request-promise-core\lib\plumbing.js:46:31)
    at self.callback (C:\web\chat-bot-telegram\node_modules\@cypress\request\request.js:183:22)
    at Request.emit (node:events:518:28)
    at Request.onRequestError (C:\web\chat-bot-telegram\node_modules\@cypress\request\request.js:869:8)
    at ClientRequest.emit (node:events:518:28)
    at _destroy (node:_http_client:880:13)
    at onSocketNT (node:_http_client:900:5)
    at process.processTicksAndRejections (node:internal/process/task_queues:83:21)
[23.08.2024, 15:17:05] RequestError: Error: connect ECONNREFUSED 169.254.0.51:3274
    at new RequestError (C:\web\chat-bot-telegram\node_modules\request-promise-core\lib\errors.js:14:15)
    at plumbing.callback (C:\web\chat-bot-telegram\node_modules\request-promise-core\lib\plumbing.js:87:29)
    at Request.RP$callback [as _callback] (C:\web\chat-bot-telegram\node_modules\request-promise-core\lib\plumbing.js:46:31)
    at self.callback (C:\web\chat-bot-telegram\node_modules\@cypress\request\request.js:183:22)
    at Request.emit (node:events:518:28)
    at Request.onRequestError (C:\web\chat-bot-telegram\node_modules\@cypress\request\request.js:869:8)
    at ClientRequest.emit (node:events:518:28)
    at _destroy (node:_http_client:880:13)
    at onSocketNT (node:_http_client:900:5)
    at process.processTicksAndRejections (node:internal/process/task_queues:83:21)
[23.08.2024, 15:17:07] RequestError: Error: connect ECONNREFUSED 169.254.0.51:3274
    at new RequestError (C:\web\chat-bot-telegram\node_modules\request-promise-core\lib\errors.js:14:15)
    at plumbing.callback (C:\web\chat-bot-telegram\node_modules\request-promise-core\lib\plumbing.js:87:29)
    at Request.RP$callback [as _callback] (C:\web\chat-bot-telegram\node_modules\request-promise-core\lib\plumbing.js:46:31)
    at self.callback (C:\web\chat-bot-telegram\node_modules\@cypress\request\request.js:183:22)
    at Request.emit (node:events:518:28)
    at Request.onRequestError (C:\web\chat-bot-telegram\node_modules\@cypress\request\request.js:869:8)
    at ClientRequest.emit (node:events:518:28)
    at _destroy (node:_http_client:880:13)
    at onSocketNT (node:_http_client:900:5)
    at process.processTicksAndRejections (node:internal/process/task_queues:83:21)
[23.08.2024, 15:17:10] RequestError: Error: connect ECONNREFUSED 169.254.0.51:3274
    at new RequestError (C:\web\chat-bot-telegram\node_modules\request-promise-core\lib\errors.js:14:15)
    at plumbing.callback (C:\web\chat-bot-telegram\node_modules\request-promise-core\lib\plumbing.js:87:29)
    at Request.RP$callback [as _callback] (C:\web\chat-bot-telegram\node_modules\request-promise-core\lib\plumbing.js:46:31)
    at self.callback (C:\web\chat-bot-telegram\node_modules\@cypress\request\request.js:183:22)
    at Request.emit (node:events:518:28)
    at Request.onRequestError (C:\web\chat-bot-telegram\node_modules\@cypress\request\request.js:869:8)
    at ClientRequest.emit (node:events:518:28)
    at _destroy (node:_http_client:880:13)
    at onSocketNT (node:_http_client:900:5)
    at process.processTicksAndRejections (node:internal/process/task_queues:83:21)
[23.08.2024, 15:17:12] RequestError: Error: connect ECONNREFUSED 169.254.0.51:3274
    at new RequestError (C:\web\chat-bot-telegram\node_modules\request-promise-core\lib\errors.js:14:15)
    at plumbing.callback (C:\web\chat-bot-telegram\node_modules\request-promise-core\lib\plumbing.js:87:29)
    at Request.RP$callback [as _callback] (C:\web\chat-bot-telegram\node_modules\request-promise-core\lib\plumbing.js:46:31)
    at self.callback (C:\web\chat-bot-telegram\node_modules\@cypress\request\request.js:183:22)
    at Request.emit (node:events:518:28)
    at Request.onRequestError (C:\web\chat-bot-telegram\node_modules\@cypress\request\request.js:869:8)
    at ClientRequest.emit (node:events:518:28)
    at _destroy (node:_http_client:880:13)
    at onSocketNT (node:_http_client:900:5)
    at process.processTicksAndRejections (node:internal/process/task_queues:83:21)
[23.08.2024, 16:04:32] Error: ETELEGRAM: 400 Bad Request: query is too old and response timeout expired or query ID is invalid
    at C:\web\chat-bot-telegram\node_modules\node-telegram-bot-api\src\telegram.js:316:15
    at tryCatcher (C:\web\chat-bot-telegram\node_modules\bluebird\js\release\util.js:16:23)
    at Promise._settlePromiseFromHandler (C:\web\chat-bot-telegram\node_modules\bluebird\js\release\promise.js:547:31)
    at Promise._settlePromise (C:\web\chat-bot-telegram\node_modules\bluebird\js\release\promise.js:604:18)
    at Promise._settlePromise0 (C:\web\chat-bot-telegram\node_modules\bluebird\js\release\promise.js:649:10)
    at Promise._settlePromises (C:\web\chat-bot-telegram\node_modules\bluebird\js\release\promise.js:729:18)
    at _drainQueueStep (C:\web\chat-bot-telegram\node_modules\bluebird\js\release\async.js:93:12)
    at _drainQueue (C:\web\chat-bot-telegram\node_modules\bluebird\js\release\async.js:86:9)
    at Async._drainQueues (C:\web\chat-bot-telegram\node_modules\bluebird\js\release\async.js:102:5)
    at Async.drainQueues [as _onImmediate] (C:\web\chat-bot-telegram\node_modules\bluebird\js\release\async.js:15:14)
    at process.processImmediate (node:internal/timers:478:21)
[23.08.2024, 16:05:55] Error: ETELEGRAM: 400 Bad Request: query is too old and response timeout expired or query ID is invalid
    at C:\web\chat-bot-telegram\node_modules\node-telegram-bot-api\src\telegram.js:316:15
    at tryCatcher (C:\web\chat-bot-telegram\node_modules\bluebird\js\release\util.js:16:23)
    at Promise._settlePromiseFromHandler (C:\web\chat-bot-telegram\node_modules\bluebird\js\release\promise.js:547:31)
    at Promise._settlePromise (C:\web\chat-bot-telegram\node_modules\bluebird\js\release\promise.js:604:18)
    at Promise._settlePromise0 (C:\web\chat-bot-telegram\node_modules\bluebird\js\release\promise.js:649:10)
    at Promise._settlePromises (C:\web\chat-bot-telegram\node_modules\bluebird\js\release\promise.js:729:18)
    at _drainQueueStep (C:\web\chat-bot-telegram\node_modules\bluebird\js\release\async.js:93:12)
    at _drainQueue (C:\web\chat-bot-telegram\node_modules\bluebird\js\release\async.js:86:9)
    at Async._drainQueues (C:\web\chat-bot-telegram\node_modules\bluebird\js\release\async.js:102:5)
    at Async.drainQueues [as _onImmediate] (C:\web\chat-bot-telegram\node_modules\bluebird\js\release\async.js:15:14)
    at process.processImmediate (node:internal/timers:478:21)
[23.08.2024, 16:09:34] Error: ETELEGRAM: 400 Bad Request: query is too old and response timeout expired or query ID is invalid
    at C:\web\chat-bot-telegram\node_modules\node-telegram-bot-api\src\telegram.js:316:15
    at tryCatcher (C:\web\chat-bot-telegram\node_modules\bluebird\js\release\util.js:16:23)
    at Promise._settlePromiseFromHandler (C:\web\chat-bot-telegram\node_modules\bluebird\js\release\promise.js:547:31)
    at Promise._settlePromise (C:\web\chat-bot-telegram\node_modules\bluebird\js\release\promise.js:604:18)
    at Promise._settlePromise0 (C:\web\chat-bot-telegram\node_modules\bluebird\js\release\promise.js:649:10)
    at Promise._settlePromises (C:\web\chat-bot-telegram\node_modules\bluebird\js\release\promise.js:729:18)
    at _drainQueueStep (C:\web\chat-bot-telegram\node_modules\bluebird\js\release\async.js:93:12)
    at _drainQueue (C:\web\chat-bot-telegram\node_modules\bluebird\js\release\async.js:86:9)
    at Async._drainQueues (C:\web\chat-bot-telegram\node_modules\bluebird\js\release\async.js:102:5)
    at Async.drainQueues [as _onImmediate] (C:\web\chat-bot-telegram\node_modules\bluebird\js\release\async.js:15:14)
    at process.processImmediate (node:internal/timers:478:21)
[24.08.2024, 06:12:46] Error: ETELEGRAM: 429 Too Many Requests: retry after 5
    at C:\web\chat-bot-telegram\node_modules\node-telegram-bot-api\src\telegram.js:316:15
    at tryCatcher (C:\web\chat-bot-telegram\node_modules\bluebird\js\release\util.js:16:23)
    at Promise._settlePromiseFromHandler (C:\web\chat-bot-telegram\node_modules\bluebird\js\release\promise.js:547:31)
    at Promise._settlePromise (C:\web\chat-bot-telegram\node_modules\bluebird\js\release\promise.js:604:18)
    at Promise._settlePromise0 (C:\web\chat-bot-telegram\node_modules\bluebird\js\release\promise.js:649:10)
    at Promise._settlePromises (C:\web\chat-bot-telegram\node_modules\bluebird\js\release\promise.js:729:18)
    at _drainQueueStep (C:\web\chat-bot-telegram\node_modules\bluebird\js\release\async.js:93:12)
    at _drainQueue (C:\web\chat-bot-telegram\node_modules\bluebird\js\release\async.js:86:9)
    at Async._drainQueues (C:\web\chat-bot-telegram\node_modules\bluebird\js\release\async.js:102:5)
    at Async.drainQueues [as _onImmediate] (C:\web\chat-bot-telegram\node_modules\bluebird\js\release\async.js:15:14)
    at process.processImmediate (node:internal/timers:478:21)
[24.08.2024, 06:12:46] Error: ETELEGRAM: 429 Too Many Requests: retry after 5
    at C:\web\chat-bot-telegram\node_modules\node-telegram-bot-api\src\telegram.js:316:15
    at tryCatcher (C:\web\chat-bot-telegram\node_modules\bluebird\js\release\util.js:16:23)
    at Promise._settlePromiseFromHandler (C:\web\chat-bot-telegram\node_modules\bluebird\js\release\promise.js:547:31)
    at Promise._settlePromise (C:\web\chat-bot-telegram\node_modules\bluebird\js\release\promise.js:604:18)
    at Promise._settlePromise0 (C:\web\chat-bot-telegram\node_modules\bluebird\js\release\promise.js:649:10)
    at Promise._settlePromises (C:\web\chat-bot-telegram\node_modules\bluebird\js\release\promise.js:729:18)
    at _drainQueueStep (C:\web\chat-bot-telegram\node_modules\bluebird\js\release\async.js:93:12)
    at _drainQueue (C:\web\chat-bot-telegram\node_modules\bluebird\js\release\async.js:86:9)
    at Async._drainQueues (C:\web\chat-bot-telegram\node_modules\bluebird\js\release\async.js:102:5)
    at Async.drainQueues [as _onImmediate] (C:\web\chat-bot-telegram\node_modules\bluebird\js\release\async.js:15:14)
    at process.processImmediate (node:internal/timers:478:21)
[24.08.2024, 06:12:47] Error: ETELEGRAM: 429 Too Many Requests: retry after 5
    at C:\web\chat-bot-telegram\node_modules\node-telegram-bot-api\src\telegram.js:316:15
    at tryCatcher (C:\web\chat-bot-telegram\node_modules\bluebird\js\release\util.js:16:23)
    at Promise._settlePromiseFromHandler (C:\web\chat-bot-telegram\node_modules\bluebird\js\release\promise.js:547:31)
    at Promise._settlePromise (C:\web\chat-bot-telegram\node_modules\bluebird\js\release\promise.js:604:18)
    at Promise._settlePromise0 (C:\web\chat-bot-telegram\node_modules\bluebird\js\release\promise.js:649:10)
    at Promise._settlePromises (C:\web\chat-bot-telegram\node_modules\bluebird\js\release\promise.js:729:18)
    at _drainQueueStep (C:\web\chat-bot-telegram\node_modules\bluebird\js\release\async.js:93:12)
    at _drainQueue (C:\web\chat-bot-telegram\node_modules\bluebird\js\release\async.js:86:9)
    at Async._drainQueues (C:\web\chat-bot-telegram\node_modules\bluebird\js\release\async.js:102:5)
    at Async.drainQueues [as _onImmediate] (C:\web\chat-bot-telegram\node_modules\bluebird\js\release\async.js:15:14)
    at process.processImmediate (node:internal/timers:478:21)
[24.08.2024, 06:12:49] Error: ETELEGRAM: 502 Bad Gateway
    at C:\web\chat-bot-telegram\node_modules\node-telegram-bot-api\src\telegram.js:316:15
    at tryCatcher (C:\web\chat-bot-telegram\node_modules\bluebird\js\release\util.js:16:23)
    at Promise._settlePromiseFromHandler (C:\web\chat-bot-telegram\node_modules\bluebird\js\release\promise.js:547:31)
    at Promise._settlePromise (C:\web\chat-bot-telegram\node_modules\bluebird\js\release\promise.js:604:18)
    at Promise._settlePromise0 (C:\web\chat-bot-telegram\node_modules\bluebird\js\release\promise.js:649:10)
    at Promise._settlePromises (C:\web\chat-bot-telegram\node_modules\bluebird\js\release\promise.js:729:18)
    at _drainQueueStep (C:\web\chat-bot-telegram\node_modules\bluebird\js\release\async.js:93:12)
    at _drainQueue (C:\web\chat-bot-telegram\node_modules\bluebird\js\release\async.js:86:9)
    at Async._drainQueues (C:\web\chat-bot-telegram\node_modules\bluebird\js\release\async.js:102:5)
    at Async.drainQueues [as _onImmediate] (C:\web\chat-bot-telegram\node_modules\bluebird\js\release\async.js:15:14)
    at process.processImmediate (node:internal/timers:478:21)
[24.08.2024, 06:12:49] Error: ETELEGRAM: 502 Bad Gateway
    at C:\web\chat-bot-telegram\node_modules\node-telegram-bot-api\src\telegram.js:316:15
    at tryCatcher (C:\web\chat-bot-telegram\node_modules\bluebird\js\release\util.js:16:23)
    at Promise._settlePromiseFromHandler (C:\web\chat-bot-telegram\node_modules\bluebird\js\release\promise.js:547:31)
    at Promise._settlePromise (C:\web\chat-bot-telegram\node_modules\bluebird\js\release\promise.js:604:18)
    at Promise._settlePromise0 (C:\web\chat-bot-telegram\node_modules\bluebird\js\release\promise.js:649:10)
    at Promise._settlePromises (C:\web\chat-bot-telegram\node_modules\bluebird\js\release\promise.js:729:18)
    at _drainQueueStep (C:\web\chat-bot-telegram\node_modules\bluebird\js\release\async.js:93:12)
    at _drainQueue (C:\web\chat-bot-telegram\node_modules\bluebird\js\release\async.js:86:9)
    at Async._drainQueues (C:\web\chat-bot-telegram\node_modules\bluebird\js\release\async.js:102:5)
    at Async.drainQueues [as _onImmediate] (C:\web\chat-bot-telegram\node_modules\bluebird\js\release\async.js:15:14)
    at process.processImmediate (node:internal/timers:478:21)
[24.08.2024, 06:12:50] Error: ETELEGRAM: 502 Bad Gateway
    at C:\web\chat-bot-telegram\node_modules\node-telegram-bot-api\src\telegram.js:316:15
    at tryCatcher (C:\web\chat-bot-telegram\node_modules\bluebird\js\release\util.js:16:23)
    at Promise._settlePromiseFromHandler (C:\web\chat-bot-telegram\node_modules\bluebird\js\release\promise.js:547:31)
    at Promise._settlePromise (C:\web\chat-bot-telegram\node_modules\bluebird\js\release\promise.js:604:18)
    at Promise._settlePromise0 (C:\web\chat-bot-telegram\node_modules\bluebird\js\release\promise.js:649:10)
    at Promise._settlePromises (C:\web\chat-bot-telegram\node_modules\bluebird\js\release\promise.js:729:18)
    at _drainQueueStep (C:\web\chat-bot-telegram\node_modules\bluebird\js\release\async.js:93:12)
    at _drainQueue (C:\web\chat-bot-telegram\node_modules\bluebird\js\release\async.js:86:9)
    at Async._drainQueues (C:\web\chat-bot-telegram\node_modules\bluebird\js\release\async.js:102:5)
    at Async.drainQueues [as _onImmediate] (C:\web\chat-bot-telegram\node_modules\bluebird\js\release\async.js:15:14)
    at process.processImmediate (node:internal/timers:478:21)
[24.08.2024, 06:12:50] Error: ETELEGRAM: 502 Bad Gateway
    at C:\web\chat-bot-telegram\node_modules\node-telegram-bot-api\src\telegram.js:316:15
    at tryCatcher (C:\web\chat-bot-telegram\node_modules\bluebird\js\release\util.js:16:23)
    at Promise._settlePromiseFromHandler (C:\web\chat-bot-telegram\node_modules\bluebird\js\release\promise.js:547:31)
    at Promise._settlePromise (C:\web\chat-bot-telegram\node_modules\bluebird\js\release\promise.js:604:18)
    at Promise._settlePromise0 (C:\web\chat-bot-telegram\node_modules\bluebird\js\release\promise.js:649:10)
    at Promise._settlePromises (C:\web\chat-bot-telegram\node_modules\bluebird\js\release\promise.js:729:18)
    at _drainQueueStep (C:\web\chat-bot-telegram\node_modules\bluebird\js\release\async.js:93:12)
    at _drainQueue (C:\web\chat-bot-telegram\node_modules\bluebird\js\release\async.js:86:9)
    at Async._drainQueues (C:\web\chat-bot-telegram\node_modules\bluebird\js\release\async.js:102:5)
    at Async.drainQueues [as _onImmediate] (C:\web\chat-bot-telegram\node_modules\bluebird\js\release\async.js:15:14)
    at process.processImmediate (node:internal/timers:478:21)
[24.08.2024, 06:12:51] Error: ETELEGRAM: 502 Bad Gateway
    at C:\web\chat-bot-telegram\node_modules\node-telegram-bot-api\src\telegram.js:316:15
    at tryCatcher (C:\web\chat-bot-telegram\node_modules\bluebird\js\release\util.js:16:23)
    at Promise._settlePromiseFromHandler (C:\web\chat-bot-telegram\node_modules\bluebird\js\release\promise.js:547:31)
    at Promise._settlePromise (C:\web\chat-bot-telegram\node_modules\bluebird\js\release\promise.js:604:18)
    at Promise._settlePromise0 (C:\web\chat-bot-telegram\node_modules\bluebird\js\release\promise.js:649:10)
    at Promise._settlePromises (C:\web\chat-bot-telegram\node_modules\bluebird\js\release\promise.js:729:18)
    at _drainQueueStep (C:\web\chat-bot-telegram\node_modules\bluebird\js\release\async.js:93:12)
    at _drainQueue (C:\web\chat-bot-telegram\node_modules\bluebird\js\release\async.js:86:9)
    at Async._drainQueues (C:\web\chat-bot-telegram\node_modules\bluebird\js\release\async.js:102:5)
    at Async.drainQueues [as _onImmediate] (C:\web\chat-bot-telegram\node_modules\bluebird\js\release\async.js:15:14)
    at process.processImmediate (node:internal/timers:478:21)
[24.08.2024, 06:30:59] RequestError: Error: socket hang up
    at new RequestError (C:\web\chat-bot-telegram\node_modules\request-promise-core\lib\errors.js:14:15)
    at plumbing.callback (C:\web\chat-bot-telegram\node_modules\request-promise-core\lib\plumbing.js:87:29)
    at Request.RP$callback [as _callback] (C:\web\chat-bot-telegram\node_modules\request-promise-core\lib\plumbing.js:46:31)
    at self.callback (C:\web\chat-bot-telegram\node_modules\@cypress\request\request.js:183:22)
    at Request.emit (node:events:518:28)
    at Request.onRequestError (C:\web\chat-bot-telegram\node_modules\@cypress\request\request.js:869:8)
    at ClientRequest.emit (node:events:518:28)
    at TLSSocket.socketOnEnd (node:_http_client:524:9)
    at TLSSocket.emit (node:events:530:35)
    at endReadableNT (node:internal/streams/readable:1696:12)
    at process.processTicksAndRejections (node:internal/process/task_queues:82:21)
[24.08.2024, 06:31:00] RequestError: Error: read ECONNRESET
    at new RequestError (C:\web\chat-bot-telegram\node_modules\request-promise-core\lib\errors.js:14:15)
    at plumbing.callback (C:\web\chat-bot-telegram\node_modules\request-promise-core\lib\plumbing.js:87:29)
    at Request.RP$callback [as _callback] (C:\web\chat-bot-telegram\node_modules\request-promise-core\lib\plumbing.js:46:31)
    at self.callback (C:\web\chat-bot-telegram\node_modules\@cypress\request\request.js:183:22)
    at Request.emit (node:events:518:28)
    at Request.onRequestError (C:\web\chat-bot-telegram\node_modules\@cypress\request\request.js:869:8)
    at ClientRequest.emit (node:events:518:28)
    at _destroy (node:_http_client:880:13)
    at onSocketNT (node:_http_client:900:5)
    at process.processTicksAndRejections (node:internal/process/task_queues:83:21)
[24.08.2024, 06:31:22] RequestError: Error: connect ETIMEDOUT 169.254.0.51:3274
    at new RequestError (C:\web\chat-bot-telegram\node_modules\request-promise-core\lib\errors.js:14:15)
    at plumbing.callback (C:\web\chat-bot-telegram\node_modules\request-promise-core\lib\plumbing.js:87:29)
    at Request.RP$callback [as _callback] (C:\web\chat-bot-telegram\node_modules\request-promise-core\lib\plumbing.js:46:31)
    at self.callback (C:\web\chat-bot-telegram\node_modules\@cypress\request\request.js:183:22)
    at Request.emit (node:events:518:28)
    at Request.onRequestError (C:\web\chat-bot-telegram\node_modules\@cypress\request\request.js:869:8)
    at ClientRequest.emit (node:events:518:28)
    at _destroy (node:_http_client:880:13)
    at onSocketNT (node:_http_client:900:5)
    at process.processTicksAndRejections (node:internal/process/task_queues:83:21)
[24.08.2024, 06:31:43] RequestError: Error: connect ETIMEDOUT 169.254.0.51:3274
    at new RequestError (C:\web\chat-bot-telegram\node_modules\request-promise-core\lib\errors.js:14:15)
    at plumbing.callback (C:\web\chat-bot-telegram\node_modules\request-promise-core\lib\plumbing.js:87:29)
    at Request.RP$callback [as _callback] (C:\web\chat-bot-telegram\node_modules\request-promise-core\lib\plumbing.js:46:31)
    at self.callback (C:\web\chat-bot-telegram\node_modules\@cypress\request\request.js:183:22)
    at Request.emit (node:events:518:28)
    at Request.onRequestError (C:\web\chat-bot-telegram\node_modules\@cypress\request\request.js:869:8)
    at ClientRequest.emit (node:events:518:28)
    at _destroy (node:_http_client:880:13)
    at onSocketNT (node:_http_client:900:5)
    at process.processTicksAndRejections (node:internal/process/task_queues:83:21)
[24.08.2024, 06:32:04] RequestError: Error: connect ETIMEDOUT 169.254.0.51:3274
    at new RequestError (C:\web\chat-bot-telegram\node_modules\request-promise-core\lib\errors.js:14:15)
    at plumbing.callback (C:\web\chat-bot-telegram\node_modules\request-promise-core\lib\plumbing.js:87:29)
    at Request.RP$callback [as _callback] (C:\web\chat-bot-telegram\node_modules\request-promise-core\lib\plumbing.js:46:31)
    at self.callback (C:\web\chat-bot-telegram\node_modules\@cypress\request\request.js:183:22)
    at Request.emit (node:events:518:28)
    at Request.onRequestError (C:\web\chat-bot-telegram\node_modules\@cypress\request\request.js:869:8)
    at ClientRequest.emit (node:events:518:28)
    at _destroy (node:_http_client:880:13)
    at onSocketNT (node:_http_client:900:5)
    at process.processTicksAndRejections (node:internal/process/task_queues:83:21)
[24.08.2024, 12:09:30] RequestError: Error: Proxy connection ended before receiving CONNECT response
    at new RequestError (C:\web\chat-bot-telegram\node_modules\request-promise-core\lib\errors.js:14:15)
    at plumbing.callback (C:\web\chat-bot-telegram\node_modules\request-promise-core\lib\plumbing.js:87:29)
    at Request.RP$callback [as _callback] (C:\web\chat-bot-telegram\node_modules\request-promise-core\lib\plumbing.js:46:31)
    at self.callback (C:\web\chat-bot-telegram\node_modules\@cypress\request\request.js:183:22)
    at Request.emit (node:events:518:28)
    at Request.onRequestError (C:\web\chat-bot-telegram\node_modules\@cypress\request\request.js:869:8)
    at ClientRequest.emit (node:events:518:28)
    at _destroy (node:_http_client:880:13)
    at onSocketNT (node:_http_client:900:5)
    at process.processTicksAndRejections (node:internal/process/task_queues:83:21)
